<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>5 Chromosomal Disorders | Textbook For SCI200 Genetics at Roxbury Community College</title>
  <meta name="description" content="This is a free online texbook for Genetics." />
  <meta name="generator" content="bookdown 0.14 and GitBook 2.6.7" />

  <meta property="og:title" content="5 Chromosomal Disorders | Textbook For SCI200 Genetics at Roxbury Community College" />
  <meta property="og:type" content="book" />
  
  <meta property="og:image" content="cover.jpg" />
  <meta property="og:description" content="This is a free online texbook for Genetics." />
  <meta name="github-repo" content="nikolaussuchert/genetics" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="5 Chromosomal Disorders | Textbook For SCI200 Genetics at Roxbury Community College" />
  <meta name="twitter:site" content="@NikolausSucher" />
  <meta name="twitter:description" content="This is a free online texbook for Genetics." />
  <meta name="twitter:image" content="cover.jpg" />

<meta name="author" content="Assembled by Nikolaus Sucher" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="chromosomes-the-vectors-of-heredity.html"/>
<link rel="next" href="the-structure-and-function-of-dna-and-rna.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />












<link rel="stylesheet" href="css/style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><strong><a href="./">Genetics</a></strong></li>

<li class="divider"></li>
<li><a href="index.html#section"></a></li>
<li class="chapter" data-level="" data-path="acknowledgements.html"><a href="acknowledgements.html"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="1" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><i class="fa fa-check"></i><b>1</b> Genetics: The Science of Genes, Genetic Variation and Heredity</a><ul>
<li class="chapter" data-level="1.1" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#mendelian-classical-genetics"><i class="fa fa-check"></i><b>1.1</b> Mendelian (classical) genetics</a></li>
<li class="chapter" data-level="1.2" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#molecular-genetics"><i class="fa fa-check"></i><b>1.2</b> Molecular genetics</a></li>
<li class="chapter" data-level="1.3" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#discrete-inheritance-and-mendels-laws"><i class="fa fa-check"></i><b>1.3</b> Discrete inheritance and Mendel’s laws</a></li>
<li class="chapter" data-level="1.4" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#notation-and-diagrams"><i class="fa fa-check"></i><b>1.4</b> Notation and diagrams</a></li>
<li class="chapter" data-level="1.5" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#multiple-gene-interactions"><i class="fa fa-check"></i><b>1.5</b> Multiple gene interactions</a></li>
<li class="chapter" data-level="1.6" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#molecular-basis-for-inheritance"><i class="fa fa-check"></i><b>1.6</b> Molecular basis for inheritance</a><ul>
<li class="chapter" data-level="1.6.1" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#dna-and-chromosomes"><i class="fa fa-check"></i><b>1.6.1</b> DNA and chromosomes</a></li>
<li class="chapter" data-level="1.6.2" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#reproduction"><i class="fa fa-check"></i><b>1.6.2</b> Reproduction</a></li>
<li class="chapter" data-level="1.6.3" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#recombination-and-genetic-linkage"><i class="fa fa-check"></i><b>1.6.3</b> Recombination and genetic linkage</a></li>
</ul></li>
<li class="chapter" data-level="1.7" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#gene-expression"><i class="fa fa-check"></i><b>1.7</b> Gene expression</a><ul>
<li class="chapter" data-level="1.7.1" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#genetic-code"><i class="fa fa-check"></i><b>1.7.1</b> Genetic code</a></li>
</ul></li>
<li class="chapter" data-level="1.8" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#nature-and-nurture"><i class="fa fa-check"></i><b>1.8</b> Nature and nurture</a></li>
<li class="chapter" data-level="1.9" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#gene-regulation"><i class="fa fa-check"></i><b>1.9</b> Gene regulation</a></li>
<li class="chapter" data-level="1.10" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#genetic-change"><i class="fa fa-check"></i><b>1.10</b> Genetic change</a><ul>
<li class="chapter" data-level="1.10.1" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#mutations"><i class="fa fa-check"></i><b>1.10.1</b> Mutations</a></li>
</ul></li>
<li class="chapter" data-level="1.11" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#natural-selection-and-evolution"><i class="fa fa-check"></i><b>1.11</b> Natural selection and evolution</a></li>
<li class="chapter" data-level="1.12" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#model-organisms"><i class="fa fa-check"></i><b>1.12</b> Model organisms</a></li>
<li class="chapter" data-level="1.13" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#medical-genetics"><i class="fa fa-check"></i><b>1.13</b> Medical Genetics</a></li>
<li class="chapter" data-level="1.14" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#research-methods"><i class="fa fa-check"></i><b>1.14</b> Research methods</a><ul>
<li class="chapter" data-level="1.14.1" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#dna-sequencing-and-genomics"><i class="fa fa-check"></i><b>1.14.1</b> DNA sequencing and genomics</a></li>
</ul></li>
<li class="chapter" data-level="1.15" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#society-and-culture"><i class="fa fa-check"></i><b>1.15</b> Society and culture</a></li>
<li class="chapter" data-level="1.16" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#genetic-counseling"><i class="fa fa-check"></i><b>1.16</b> Genetic counseling</a></li>
<li class="chapter" data-level="1.17" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#human-genetics"><i class="fa fa-check"></i><b>1.17</b> Human genetics</a><ul>
<li class="chapter" data-level="1.17.1" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#genetic-differences-and-inheritance-patterns"><i class="fa fa-check"></i><b>1.17.1</b> Genetic differences and inheritance patterns</a></li>
<li class="chapter" data-level="1.17.2" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#autosomal-dominant-inheritance"><i class="fa fa-check"></i><b>1.17.2</b> Autosomal dominant inheritance</a></li>
<li class="chapter" data-level="1.17.3" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#autosomal-recessive-inheritance"><i class="fa fa-check"></i><b>1.17.3</b> Autosomal recessive inheritance</a></li>
<li class="chapter" data-level="1.17.4" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#x-linked-and-y-linked-inheritance"><i class="fa fa-check"></i><b>1.17.4</b> X-linked and Y-linked inheritance</a></li>
<li class="chapter" data-level="1.17.5" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#pedigrees-analysis"><i class="fa fa-check"></i><b>1.17.5</b> Pedigrees analysis</a></li>
<li class="chapter" data-level="1.17.6" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#karyotype"><i class="fa fa-check"></i><b>1.17.6</b> Karyotype</a></li>
</ul></li>
<li class="chapter" data-level="1.18" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#genomics"><i class="fa fa-check"></i><b>1.18</b> Genomics</a></li>
<li class="chapter" data-level="1.19" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#population-genetics"><i class="fa fa-check"></i><b>1.19</b> Population genetics</a></li>
<li class="chapter" data-level="1.20" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#eugenics"><i class="fa fa-check"></i><b>1.20</b> Eugenics</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html"><i class="fa fa-check"></i><b>2</b> Reproduction of Cells and Organisms</a><ul>
<li class="chapter" data-level="2.1" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#cell"><i class="fa fa-check"></i><b>2.1</b> Cell</a></li>
<li class="chapter" data-level="2.2" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#prokaryotic-cells"><i class="fa fa-check"></i><b>2.2</b> Prokaryotic cells</a></li>
<li class="chapter" data-level="2.3" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#eukaryotic-cells"><i class="fa fa-check"></i><b>2.3</b> Eukaryotic cells</a></li>
<li class="chapter" data-level="2.4" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#asexual"><i class="fa fa-check"></i><b>2.4</b> Asexual</a></li>
<li class="chapter" data-level="2.5" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#sexual"><i class="fa fa-check"></i><b>2.5</b> Sexual</a></li>
<li class="chapter" data-level="2.6" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#allogamy"><i class="fa fa-check"></i><b>2.6</b> Allogamy</a></li>
<li class="chapter" data-level="2.7" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#autogamy"><i class="fa fa-check"></i><b>2.7</b> Autogamy</a></li>
<li class="chapter" data-level="2.8" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#mitosis-and-meiosis"><i class="fa fa-check"></i><b>2.8</b> Mitosis and meiosis</a></li>
<li class="chapter" data-level="2.9" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#same-sex"><i class="fa fa-check"></i><b>2.9</b> Same-sex</a></li>
<li class="chapter" data-level="2.10" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#strategies"><i class="fa fa-check"></i><b>2.10</b> Strategies</a></li>
<li class="chapter" data-level="2.11" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#other-types"><i class="fa fa-check"></i><b>2.11</b> Other types</a></li>
<li class="chapter" data-level="2.12" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#asexual-vs.sexual-reproduction"><i class="fa fa-check"></i><b>2.12</b> Asexual vs. sexual reproduction</a></li>
<li class="chapter" data-level="2.13" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#life-without"><i class="fa fa-check"></i><b>2.13</b> Life without</a></li>
<li class="chapter" data-level="2.14" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#binary-fission"><i class="fa fa-check"></i><b>2.14</b> Binary fission</a></li>
<li class="chapter" data-level="2.15" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#fission-of-prokaryotes"><i class="fa fa-check"></i><b>2.15</b> Fission of prokaryotes</a></li>
<li class="chapter" data-level="2.16" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#types-of-binary-fission"><i class="fa fa-check"></i><b>2.16</b> Types of binary fission</a></li>
<li class="chapter" data-level="2.17" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#mitosis"><i class="fa fa-check"></i><b>2.17</b> Mitosis</a></li>
<li class="chapter" data-level="2.18" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#meiosis"><i class="fa fa-check"></i><b>2.18</b> Meiosis</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html"><i class="fa fa-check"></i><b>3</b> Mendelian Inheritance</a><ul>
<li class="chapter" data-level="3.1" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#mendels-genetic-discoveries"><i class="fa fa-check"></i><b>3.1</b> Mendel’s genetic discoveries</a></li>
<li class="chapter" data-level="3.2" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#law-of-dominance-and-uniformity"><i class="fa fa-check"></i><b>3.2</b> Law of Dominance and Uniformity</a></li>
<li class="chapter" data-level="3.3" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#law-of-segregation-of-genes"><i class="fa fa-check"></i><b>3.3</b> Law of Segregation of genes</a></li>
<li class="chapter" data-level="3.4" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#law-of-independent-assortment"><i class="fa fa-check"></i><b>3.4</b> Law of Independent Assortment</a></li>
<li class="chapter" data-level="3.5" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#mendelian-trait"><i class="fa fa-check"></i><b>3.5</b> Mendelian trait</a></li>
<li class="chapter" data-level="3.6" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#zygosity"><i class="fa fa-check"></i><b>3.6</b> Zygosity</a></li>
<li class="chapter" data-level="3.7" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#monohybrid-cross"><i class="fa fa-check"></i><b>3.7</b> Monohybrid cross</a></li>
<li class="chapter" data-level="3.8" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#punnett-square"><i class="fa fa-check"></i><b>3.8</b> Punnett square</a></li>
<li class="chapter" data-level="3.9" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#dihybrid-cross"><i class="fa fa-check"></i><b>3.9</b> Dihybrid cross</a></li>
<li class="chapter" data-level="3.10" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#forked-line-method"><i class="fa fa-check"></i><b>3.10</b> Forked-line method</a></li>
<li class="chapter" data-level="3.11" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#mendelian-traits-in-humans"><i class="fa fa-check"></i><b>3.11</b> Mendelian traits in humans</a></li>
<li class="chapter" data-level="3.12" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#omim---online-mendelian-inheritance-in-man"><i class="fa fa-check"></i><b>3.12</b> OMIM - Online Mendelian Inheritance in Man</a></li>
<li class="chapter" data-level="3.13" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#non-mendelian-inheritance"><i class="fa fa-check"></i><b>3.13</b> Non-Mendelian inheritance</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="chromosomes-the-vectors-of-heredity.html"><a href="chromosomes-the-vectors-of-heredity.html"><i class="fa fa-check"></i><b>4</b> Chromosomes: The Vectors of Heredity</a><ul>
<li class="chapter" data-level="4.1" data-path="chromosomes-the-vectors-of-heredity.html"><a href="chromosomes-the-vectors-of-heredity.html#prokaryotes"><i class="fa fa-check"></i><b>4.1</b> Prokaryotes</a><ul>
<li class="chapter" data-level="4.1.1" data-path="chromosomes-the-vectors-of-heredity.html"><a href="chromosomes-the-vectors-of-heredity.html#structure-in-sequences"><i class="fa fa-check"></i><b>4.1.1</b> Structure in sequences</a></li>
<li class="chapter" data-level="4.1.2" data-path="chromosomes-the-vectors-of-heredity.html"><a href="chromosomes-the-vectors-of-heredity.html#dna-packaging"><i class="fa fa-check"></i><b>4.1.2</b> DNA packaging</a></li>
</ul></li>
<li class="chapter" data-level="4.2" data-path="chromosomes-the-vectors-of-heredity.html"><a href="chromosomes-the-vectors-of-heredity.html#eukaryotes"><i class="fa fa-check"></i><b>4.2</b> Eukaryotes</a><ul>
<li class="chapter" data-level="4.2.1" data-path="chromosomes-the-vectors-of-heredity.html"><a href="chromosomes-the-vectors-of-heredity.html#interphase-chromatin"><i class="fa fa-check"></i><b>4.2.1</b> Interphase chromatin</a></li>
<li class="chapter" data-level="4.2.2" data-path="chromosomes-the-vectors-of-heredity.html"><a href="chromosomes-the-vectors-of-heredity.html#metaphase-chromatin-and-division"><i class="fa fa-check"></i><b>4.2.2</b> Metaphase chromatin and division</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="chromosomes-the-vectors-of-heredity.html"><a href="chromosomes-the-vectors-of-heredity.html#human-chromosomes"><i class="fa fa-check"></i><b>4.3</b> Human chromosomes</a></li>
<li class="chapter" data-level="4.4" data-path="chromosomes-the-vectors-of-heredity.html"><a href="chromosomes-the-vectors-of-heredity.html#number-in-various-organisms"><i class="fa fa-check"></i><b>4.4</b> Number in various organisms</a><ul>
<li class="chapter" data-level="4.4.1" data-path="chromosomes-the-vectors-of-heredity.html"><a href="chromosomes-the-vectors-of-heredity.html#in-eukaryotes"><i class="fa fa-check"></i><b>4.4.1</b> In eukaryotes</a></li>
<li class="chapter" data-level="4.4.2" data-path="chromosomes-the-vectors-of-heredity.html"><a href="chromosomes-the-vectors-of-heredity.html#in-prokaryotes"><i class="fa fa-check"></i><b>4.4.2</b> In prokaryotes</a></li>
</ul></li>
<li class="chapter" data-level="4.5" data-path="chromosomes-the-vectors-of-heredity.html"><a href="chromosomes-the-vectors-of-heredity.html#karyotype-1"><i class="fa fa-check"></i><b>4.5</b> Karyotype</a></li>
<li class="chapter" data-level="4.6" data-path="chromosomes-the-vectors-of-heredity.html"><a href="chromosomes-the-vectors-of-heredity.html#aberrations"><i class="fa fa-check"></i><b>4.6</b> Aberrations</a></li>
<li class="chapter" data-level="4.7" data-path="chromosomes-the-vectors-of-heredity.html"><a href="chromosomes-the-vectors-of-heredity.html#sperm-aneuploidy"><i class="fa fa-check"></i><b>4.7</b> Sperm aneuploidy</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="chromosomal-disorders.html"><a href="chromosomal-disorders.html"><i class="fa fa-check"></i><b>5</b> Chromosomal Disorders</a><ul>
<li class="chapter" data-level="5.1" data-path="chromosomal-disorders.html"><a href="chromosomal-disorders.html#numerical-disorders"><i class="fa fa-check"></i><b>5.1</b> Numerical disorders</a></li>
<li class="chapter" data-level="5.2" data-path="chromosomal-disorders.html"><a href="chromosomal-disorders.html#structural-abnormalities"><i class="fa fa-check"></i><b>5.2</b> Structural abnormalities</a></li>
<li class="chapter" data-level="5.3" data-path="chromosomal-disorders.html"><a href="chromosomal-disorders.html#inheritance"><i class="fa fa-check"></i><b>5.3</b> Inheritance</a></li>
<li class="chapter" data-level="5.4" data-path="chromosomal-disorders.html"><a href="chromosomal-disorders.html#acquired-chromosome-abnormalities"><i class="fa fa-check"></i><b>5.4</b> Acquired chromosome abnormalities</a></li>
<li class="chapter" data-level="5.5" data-path="chromosomal-disorders.html"><a href="chromosomal-disorders.html#aneuploidy"><i class="fa fa-check"></i><b>5.5</b> Aneuploidy</a></li>
<li class="chapter" data-level="5.6" data-path="chromosomal-disorders.html"><a href="chromosomal-disorders.html#mechanisms"><i class="fa fa-check"></i><b>5.6</b> Mechanisms</a></li>
<li class="chapter" data-level="5.7" data-path="chromosomal-disorders.html"><a href="chromosomal-disorders.html#somatic-mosaicism-in-the-nervous-system"><i class="fa fa-check"></i><b>5.7</b> Somatic mosaicism in the nervous system</a></li>
<li class="chapter" data-level="5.8" data-path="chromosomal-disorders.html"><a href="chromosomal-disorders.html#somatic-mosaicism-in-cancer"><i class="fa fa-check"></i><b>5.8</b> Somatic mosaicism in cancer</a></li>
<li class="chapter" data-level="5.9" data-path="chromosomal-disorders.html"><a href="chromosomal-disorders.html#partial-aneuploidy"><i class="fa fa-check"></i><b>5.9</b> Partial aneuploidy</a></li>
<li class="chapter" data-level="5.10" data-path="chromosomal-disorders.html"><a href="chromosomal-disorders.html#aneuploidogens"><i class="fa fa-check"></i><b>5.10</b> Aneuploidogens</a></li>
<li class="chapter" data-level="5.11" data-path="chromosomal-disorders.html"><a href="chromosomal-disorders.html#diagnosis"><i class="fa fa-check"></i><b>5.11</b> Diagnosis</a></li>
<li class="chapter" data-level="5.12" data-path="chromosomal-disorders.html"><a href="chromosomal-disorders.html#chromosomal-crossover"><i class="fa fa-check"></i><b>5.12</b> Chromosomal crossover</a></li>
<li class="chapter" data-level="5.13" data-path="chromosomal-disorders.html"><a href="chromosomal-disorders.html#gene-mapping"><i class="fa fa-check"></i><b>5.13</b> Gene mapping</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="the-structure-and-function-of-dna-and-rna.html"><a href="the-structure-and-function-of-dna-and-rna.html"><i class="fa fa-check"></i><b>6</b> The Structure And Function of DNA And RNA</a><ul>
<li class="chapter" data-level="6.1" data-path="the-structure-and-function-of-dna-and-rna.html"><a href="the-structure-and-function-of-dna-and-rna.html#dna"><i class="fa fa-check"></i><b>6.1</b> DNA</a></li>
<li class="chapter" data-level="6.2" data-path="the-structure-and-function-of-dna-and-rna.html"><a href="the-structure-and-function-of-dna-and-rna.html#properties"><i class="fa fa-check"></i><b>6.2</b> Properties</a></li>
<li class="chapter" data-level="6.3" data-path="the-structure-and-function-of-dna-and-rna.html"><a href="the-structure-and-function-of-dna-and-rna.html#nucleobase-classification"><i class="fa fa-check"></i><b>6.3</b> Nucleobase classification</a></li>
<li class="chapter" data-level="6.4" data-path="the-structure-and-function-of-dna-and-rna.html"><a href="the-structure-and-function-of-dna-and-rna.html#grooves"><i class="fa fa-check"></i><b>6.4</b> Grooves</a></li>
<li class="chapter" data-level="6.5" data-path="the-structure-and-function-of-dna-and-rna.html"><a href="the-structure-and-function-of-dna-and-rna.html#base-pairing"><i class="fa fa-check"></i><b>6.5</b> Base pairing</a></li>
<li class="chapter" data-level="6.6" data-path="the-structure-and-function-of-dna-and-rna.html"><a href="the-structure-and-function-of-dna-and-rna.html#sense-and-antisense"><i class="fa fa-check"></i><b>6.6</b> Sense and antisense</a></li>
<li class="chapter" data-level="6.7" data-path="the-structure-and-function-of-dna-and-rna.html"><a href="the-structure-and-function-of-dna-and-rna.html#supercoiling"><i class="fa fa-check"></i><b>6.7</b> Supercoiling</a></li>
<li class="chapter" data-level="6.8" data-path="the-structure-and-function-of-dna-and-rna.html"><a href="the-structure-and-function-of-dna-and-rna.html#base-modifications-and-dna-packaging"><i class="fa fa-check"></i><b>6.8</b> Base modifications and DNA packaging</a></li>
<li class="chapter" data-level="6.9" data-path="the-structure-and-function-of-dna-and-rna.html"><a href="the-structure-and-function-of-dna-and-rna.html#damage"><i class="fa fa-check"></i><b>6.9</b> Damage</a></li>
<li class="chapter" data-level="6.10" data-path="the-structure-and-function-of-dna-and-rna.html"><a href="the-structure-and-function-of-dna-and-rna.html#biological-functions"><i class="fa fa-check"></i><b>6.10</b> Biological functions</a></li>
<li class="chapter" data-level="6.11" data-path="the-structure-and-function-of-dna-and-rna.html"><a href="the-structure-and-function-of-dna-and-rna.html#genes-and-genomes"><i class="fa fa-check"></i><b>6.11</b> Genes and genomes</a></li>
<li class="chapter" data-level="6.12" data-path="the-structure-and-function-of-dna-and-rna.html"><a href="the-structure-and-function-of-dna-and-rna.html#transcription-and-translation"><i class="fa fa-check"></i><b>6.12</b> Transcription and translation</a></li>
<li class="chapter" data-level="6.13" data-path="the-structure-and-function-of-dna-and-rna.html"><a href="the-structure-and-function-of-dna-and-rna.html#replication"><i class="fa fa-check"></i><b>6.13</b> Replication</a></li>
<li class="chapter" data-level="6.14" data-path="the-structure-and-function-of-dna-and-rna.html"><a href="the-structure-and-function-of-dna-and-rna.html#interactions-with-proteins"><i class="fa fa-check"></i><b>6.14</b> Interactions with proteins</a></li>
<li class="chapter" data-level="6.15" data-path="the-structure-and-function-of-dna-and-rna.html"><a href="the-structure-and-function-of-dna-and-rna.html#dna-binding-proteins"><i class="fa fa-check"></i><b>6.15</b> DNA-binding proteins</a></li>
<li class="chapter" data-level="6.16" data-path="the-structure-and-function-of-dna-and-rna.html"><a href="the-structure-and-function-of-dna-and-rna.html#dna-modifying-enzymes"><i class="fa fa-check"></i><b>6.16</b> DNA-modifying enzymes</a><ul>
<li class="chapter" data-level="6.16.1" data-path="the-structure-and-function-of-dna-and-rna.html"><a href="the-structure-and-function-of-dna-and-rna.html#nucleases-and-ligases"><i class="fa fa-check"></i><b>6.16.1</b> Nucleases and ligases</a></li>
</ul></li>
<li class="chapter" data-level="6.17" data-path="the-structure-and-function-of-dna-and-rna.html"><a href="the-structure-and-function-of-dna-and-rna.html#topoisomerases-and-helicases"><i class="fa fa-check"></i><b>6.17</b> Topoisomerases and helicases</a></li>
<li class="chapter" data-level="6.18" data-path="the-structure-and-function-of-dna-and-rna.html"><a href="the-structure-and-function-of-dna-and-rna.html#polymerases"><i class="fa fa-check"></i><b>6.18</b> Polymerases</a></li>
<li class="chapter" data-level="6.19" data-path="the-structure-and-function-of-dna-and-rna.html"><a href="the-structure-and-function-of-dna-and-rna.html#genetic-recombination"><i class="fa fa-check"></i><b>6.19</b> Genetic recombination</a></li>
<li class="chapter" data-level="6.20" data-path="the-structure-and-function-of-dna-and-rna.html"><a href="the-structure-and-function-of-dna-and-rna.html#evolution"><i class="fa fa-check"></i><b>6.20</b> Evolution</a></li>
<li class="chapter" data-level="6.21" data-path="the-structure-and-function-of-dna-and-rna.html"><a href="the-structure-and-function-of-dna-and-rna.html#genetic-engineering"><i class="fa fa-check"></i><b>6.21</b> Genetic engineering</a></li>
<li class="chapter" data-level="6.22" data-path="the-structure-and-function-of-dna-and-rna.html"><a href="the-structure-and-function-of-dna-and-rna.html#dna-profiling"><i class="fa fa-check"></i><b>6.22</b> DNA profiling</a></li>
<li class="chapter" data-level="6.23" data-path="the-structure-and-function-of-dna-and-rna.html"><a href="the-structure-and-function-of-dna-and-rna.html#rna"><i class="fa fa-check"></i><b>6.23</b> RNA</a></li>
<li class="chapter" data-level="6.24" data-path="the-structure-and-function-of-dna-and-rna.html"><a href="the-structure-and-function-of-dna-and-rna.html#comparison-with-dna"><i class="fa fa-check"></i><b>6.24</b> Comparison with DNA</a></li>
<li class="chapter" data-level="6.25" data-path="the-structure-and-function-of-dna-and-rna.html"><a href="the-structure-and-function-of-dna-and-rna.html#structure"><i class="fa fa-check"></i><b>6.25</b> Structure</a></li>
<li class="chapter" data-level="6.26" data-path="the-structure-and-function-of-dna-and-rna.html"><a href="the-structure-and-function-of-dna-and-rna.html#synthesis"><i class="fa fa-check"></i><b>6.26</b> Synthesis</a></li>
<li class="chapter" data-level="6.27" data-path="the-structure-and-function-of-dna-and-rna.html"><a href="the-structure-and-function-of-dna-and-rna.html#types-of-rna"><i class="fa fa-check"></i><b>6.27</b> Types of RNA</a></li>
<li class="chapter" data-level="6.28" data-path="the-structure-and-function-of-dna-and-rna.html"><a href="the-structure-and-function-of-dna-and-rna.html#in-length"><i class="fa fa-check"></i><b>6.28</b> In length</a></li>
<li class="chapter" data-level="6.29" data-path="the-structure-and-function-of-dna-and-rna.html"><a href="the-structure-and-function-of-dna-and-rna.html#in-translation"><i class="fa fa-check"></i><b>6.29</b> In translation</a></li>
<li class="chapter" data-level="6.30" data-path="the-structure-and-function-of-dna-and-rna.html"><a href="the-structure-and-function-of-dna-and-rna.html#regulatory-rna"><i class="fa fa-check"></i><b>6.30</b> Regulatory RNA</a></li>
<li class="chapter" data-level="6.31" data-path="the-structure-and-function-of-dna-and-rna.html"><a href="the-structure-and-function-of-dna-and-rna.html#rna-interference-by-mirnas"><i class="fa fa-check"></i><b>6.31</b> RNA interference by miRNAs</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="eukaryotic-chromosome-structure.html"><a href="eukaryotic-chromosome-structure.html"><i class="fa fa-check"></i><b>7</b> Eukaryotic chromosome structure</a><ul>
<li class="chapter" data-level="7.1" data-path="eukaryotic-chromosome-structure.html"><a href="eukaryotic-chromosome-structure.html#nucleosome"><i class="fa fa-check"></i><b>7.1</b> Nucleosome</a></li>
<li class="chapter" data-level="7.2" data-path="eukaryotic-chromosome-structure.html"><a href="eukaryotic-chromosome-structure.html#structure-1"><i class="fa fa-check"></i><b>7.2</b> Structure</a><ul>
<li class="chapter" data-level="7.2.1" data-path="eukaryotic-chromosome-structure.html"><a href="eukaryotic-chromosome-structure.html#structure-of-the-core-particle"><i class="fa fa-check"></i><b>7.2.1</b> Structure of the core particle</a></li>
<li class="chapter" data-level="7.2.2" data-path="eukaryotic-chromosome-structure.html"><a href="eukaryotic-chromosome-structure.html#protein-interactions-within-the-nucleosome"><i class="fa fa-check"></i><b>7.2.2</b> Protein interactions within the nucleosome</a></li>
</ul></li>
<li class="chapter" data-level="7.3" data-path="eukaryotic-chromosome-structure.html"><a href="eukaryotic-chromosome-structure.html#histone---dna-interactions"><i class="fa fa-check"></i><b>7.3</b> Histone - DNA interactions</a></li>
<li class="chapter" data-level="7.4" data-path="eukaryotic-chromosome-structure.html"><a href="eukaryotic-chromosome-structure.html#histone-tail-domains"><i class="fa fa-check"></i><b>7.4</b> Histone tail domains</a></li>
<li class="chapter" data-level="7.5" data-path="eukaryotic-chromosome-structure.html"><a href="eukaryotic-chromosome-structure.html#higher-order-structure"><i class="fa fa-check"></i><b>7.5</b> Higher order structure</a></li>
<li class="chapter" data-level="7.6" data-path="eukaryotic-chromosome-structure.html"><a href="eukaryotic-chromosome-structure.html#dynamics"><i class="fa fa-check"></i><b>7.6</b> Dynamics</a></li>
<li class="chapter" data-level="7.7" data-path="eukaryotic-chromosome-structure.html"><a href="eukaryotic-chromosome-structure.html#nucleosome-sliding"><i class="fa fa-check"></i><b>7.7</b> Nucleosome sliding</a></li>
<li class="chapter" data-level="7.8" data-path="eukaryotic-chromosome-structure.html"><a href="eukaryotic-chromosome-structure.html#dna-site-exposure"><i class="fa fa-check"></i><b>7.8</b> DNA site exposure</a></li>
<li class="chapter" data-level="7.9" data-path="eukaryotic-chromosome-structure.html"><a href="eukaryotic-chromosome-structure.html#nucleosome-free-region"><i class="fa fa-check"></i><b>7.9</b> Nucleosome free region</a></li>
<li class="chapter" data-level="7.10" data-path="eukaryotic-chromosome-structure.html"><a href="eukaryotic-chromosome-structure.html#modulating-nucleosome-structure"><i class="fa fa-check"></i><b>7.10</b> Modulating nucleosome structure</a></li>
<li class="chapter" data-level="7.11" data-path="eukaryotic-chromosome-structure.html"><a href="eukaryotic-chromosome-structure.html#histone-post-translational-modifications"><i class="fa fa-check"></i><b>7.11</b> Histone post-translational modifications</a></li>
<li class="chapter" data-level="7.12" data-path="eukaryotic-chromosome-structure.html"><a href="eukaryotic-chromosome-structure.html#histone-variants"><i class="fa fa-check"></i><b>7.12</b> Histone variants</a></li>
<li class="chapter" data-level="7.13" data-path="eukaryotic-chromosome-structure.html"><a href="eukaryotic-chromosome-structure.html#atp-dependent-nucleosome-remodeling"><i class="fa fa-check"></i><b>7.13</b> ATP-dependent nucleosome remodeling</a></li>
<li class="chapter" data-level="7.14" data-path="eukaryotic-chromosome-structure.html"><a href="eukaryotic-chromosome-structure.html#dynamic-nucleosome-remodelling-across-the-yeast-genome"><i class="fa fa-check"></i><b>7.14</b> Dynamic nucleosome remodelling across the Yeast genome</a></li>
<li class="chapter" data-level="7.15" data-path="eukaryotic-chromosome-structure.html"><a href="eukaryotic-chromosome-structure.html#nucleosome-assembly-in-vitro"><i class="fa fa-check"></i><b>7.15</b> Nucleosome assembly in vitro</a></li>
<li class="chapter" data-level="7.16" data-path="eukaryotic-chromosome-structure.html"><a href="eukaryotic-chromosome-structure.html#nucleosome-assembly-in-vivo"><i class="fa fa-check"></i><b>7.16</b> Nucleosome assembly in vivo</a></li>
<li class="chapter" data-level="7.17" data-path="eukaryotic-chromosome-structure.html"><a href="eukaryotic-chromosome-structure.html#histones"><i class="fa fa-check"></i><b>7.17</b> Histones</a><ul>
<li class="chapter" data-level="7.17.1" data-path="eukaryotic-chromosome-structure.html"><a href="eukaryotic-chromosome-structure.html#structure-2"><i class="fa fa-check"></i><b>7.17.1</b> Structure</a></li>
</ul></li>
<li class="chapter" data-level="7.18" data-path="eukaryotic-chromosome-structure.html"><a href="eukaryotic-chromosome-structure.html#functions-of-histone-modifications"><i class="fa fa-check"></i><b>7.18</b> Functions of histone modifications</a></li>
<li class="chapter" data-level="7.19" data-path="eukaryotic-chromosome-structure.html"><a href="eukaryotic-chromosome-structure.html#chemistry-of-histone-modifications"><i class="fa fa-check"></i><b>7.19</b> Chemistry of histone modifications</a></li>
<li class="chapter" data-level="7.20" data-path="eukaryotic-chromosome-structure.html"><a href="eukaryotic-chromosome-structure.html#functions-in-transcription"><i class="fa fa-check"></i><b>7.20</b> Functions in transcription</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="dna-replication.html"><a href="dna-replication.html"><i class="fa fa-check"></i><b>8</b> DNA replication</a><ul>
<li class="chapter" data-level="8.1" data-path="dna-replication.html"><a href="dna-replication.html#eukaryotic-dna-replication"><i class="fa fa-check"></i><b>8.1</b> Eukaryotic DNA replication</a></li>
</ul></li>
<li class="chapter" data-level="9" data-path="transcription.html"><a href="transcription.html"><i class="fa fa-check"></i><b>9</b> Transcription</a><ul>
<li class="chapter" data-level="9.1" data-path="transcription.html"><a href="transcription.html#reverse-transcription"><i class="fa fa-check"></i><b>9.1</b> Reverse transcription</a></li>
</ul></li>
<li class="chapter" data-level="10" data-path="translation.html"><a href="translation.html"><i class="fa fa-check"></i><b>10</b> Translation</a><ul>
<li class="chapter" data-level="10.1" data-path="translation.html"><a href="translation.html#genetic-code-1"><i class="fa fa-check"></i><b>10.1</b> Genetic code</a></li>
</ul></li>
<li class="chapter" data-level="11" data-path="mutation-dna-repair-and-recombination.html"><a href="mutation-dna-repair-and-recombination.html"><i class="fa fa-check"></i><b>11</b> Mutation, DNA Repair And Recombination</a></li>
<li class="chapter" data-level="12" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html"><i class="fa fa-check"></i><b>12</b> Techniques of Molecular Genetics</a><ul>
<li class="chapter" data-level="12.1" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#polymerase-chain-reaction"><i class="fa fa-check"></i><b>12.1</b> Polymerase chain reaction</a></li>
<li class="chapter" data-level="12.2" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#dna-sequencing"><i class="fa fa-check"></i><b>12.2</b> DNA Sequencing</a></li>
<li class="chapter" data-level="12.3" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#rna-seq"><i class="fa fa-check"></i><b>12.3</b> RNA-Seq</a></li>
<li class="chapter" data-level="12.4" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#dna-microarray"><i class="fa fa-check"></i><b>12.4</b> DNA microarray</a></li>
<li class="chapter" data-level="12.5" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#recombinant-dna"><i class="fa fa-check"></i><b>12.5</b> Recombinant DNA</a></li>
<li class="chapter" data-level="12.6" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#molecular-cloning"><i class="fa fa-check"></i><b>12.6</b> Molecular cloning</a></li>
<li class="chapter" data-level="12.7" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#mutagenesis"><i class="fa fa-check"></i><b>12.7</b> Mutagenesis</a></li>
<li class="chapter" data-level="12.8" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#gene-knockout"><i class="fa fa-check"></i><b>12.8</b> Gene knockout</a></li>
<li class="chapter" data-level="12.9" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#rna-interference"><i class="fa fa-check"></i><b>12.9</b> RNA interference</a></li>
<li class="chapter" data-level="12.10" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#cre-lox-recombination"><i class="fa fa-check"></i><b>12.10</b> Cre-Lox recombination</a></li>
<li class="chapter" data-level="12.11" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#crispr-gene-editing"><i class="fa fa-check"></i><b>12.11</b> CRISPR gene editing</a></li>
</ul></li>
<li class="chapter" data-level="13" data-path="genomics-1.html"><a href="genomics-1.html"><i class="fa fa-check"></i><b>13</b> Genomics</a><ul>
<li class="chapter" data-level="13.1" data-path="genomics-1.html"><a href="genomics-1.html#dna-sequencing-technology-developed"><i class="fa fa-check"></i><b>13.1</b> DNA-sequencing technology developed</a></li>
<li class="chapter" data-level="13.2" data-path="genomics-1.html"><a href="genomics-1.html#complete-genomes"><i class="fa fa-check"></i><b>13.2</b> Complete genomes</a></li>
<li class="chapter" data-level="13.3" data-path="genomics-1.html"><a href="genomics-1.html#the-omics-revolution"><i class="fa fa-check"></i><b>13.3</b> The “omics” revolution</a></li>
<li class="chapter" data-level="13.4" data-path="genomics-1.html"><a href="genomics-1.html#genome-analysis"><i class="fa fa-check"></i><b>13.4</b> Genome analysis</a><ul>
<li class="chapter" data-level="13.4.1" data-path="genomics-1.html"><a href="genomics-1.html#sequencing"><i class="fa fa-check"></i><b>13.4.1</b> Sequencing</a></li>
<li class="chapter" data-level="13.4.2" data-path="genomics-1.html"><a href="genomics-1.html#shotgun-sequencing"><i class="fa fa-check"></i><b>13.4.2</b> Shotgun sequencing</a></li>
<li class="chapter" data-level="13.4.3" data-path="genomics-1.html"><a href="genomics-1.html#high-throughput-sequencing"><i class="fa fa-check"></i><b>13.4.3</b> High-throughput sequencing</a></li>
</ul></li>
<li class="chapter" data-level="13.5" data-path="genomics-1.html"><a href="genomics-1.html#assembly"><i class="fa fa-check"></i><b>13.5</b> Assembly</a></li>
<li class="chapter" data-level="13.6" data-path="genomics-1.html"><a href="genomics-1.html#finishing"><i class="fa fa-check"></i><b>13.6</b> Finishing</a></li>
<li class="chapter" data-level="13.7" data-path="genomics-1.html"><a href="genomics-1.html#annotation"><i class="fa fa-check"></i><b>13.7</b> Annotation</a></li>
<li class="chapter" data-level="13.8" data-path="genomics-1.html"><a href="genomics-1.html#sequencing-pipelines-and-databases"><i class="fa fa-check"></i><b>13.8</b> Sequencing pipelines and databases</a></li>
<li class="chapter" data-level="13.9" data-path="genomics-1.html"><a href="genomics-1.html#functional-genomics"><i class="fa fa-check"></i><b>13.9</b> Functional genomics</a></li>
<li class="chapter" data-level="13.10" data-path="genomics-1.html"><a href="genomics-1.html#structural-genomics"><i class="fa fa-check"></i><b>13.10</b> Structural genomics</a></li>
<li class="chapter" data-level="13.11" data-path="genomics-1.html"><a href="genomics-1.html#epigenomics"><i class="fa fa-check"></i><b>13.11</b> Epigenomics</a></li>
<li class="chapter" data-level="13.12" data-path="genomics-1.html"><a href="genomics-1.html#metagenomics"><i class="fa fa-check"></i><b>13.12</b> Metagenomics</a></li>
<li class="chapter" data-level="13.13" data-path="genomics-1.html"><a href="genomics-1.html#applications-of-genomics"><i class="fa fa-check"></i><b>13.13</b> Applications of genomics</a><ul>
<li class="chapter" data-level="13.13.1" data-path="genomics-1.html"><a href="genomics-1.html#genomic-medicine"><i class="fa fa-check"></i><b>13.13.1</b> Genomic medicine</a></li>
<li class="chapter" data-level="13.13.2" data-path="genomics-1.html"><a href="genomics-1.html#synthetic-biology-and-bioengineering"><i class="fa fa-check"></i><b>13.13.2</b> Synthetic biology and bioengineering</a></li>
<li class="chapter" data-level="13.13.3" data-path="genomics-1.html"><a href="genomics-1.html#conservation-genomics"><i class="fa fa-check"></i><b>13.13.3</b> Conservation genomics</a></li>
</ul></li>
<li class="chapter" data-level="13.14" data-path="genomics-1.html"><a href="genomics-1.html#genome-wide-association-study"><i class="fa fa-check"></i><b>13.14</b> Genome-wide association study</a></li>
</ul></li>
<li class="chapter" data-level="14" data-path="applications-of-molecular-genetics.html"><a href="applications-of-molecular-genetics.html"><i class="fa fa-check"></i><b>14</b> Applications of Molecular Genetics</a><ul>
<li class="chapter" data-level="14.1" data-path="applications-of-molecular-genetics.html"><a href="applications-of-molecular-genetics.html#genetic-testing"><i class="fa fa-check"></i><b>14.1</b> Genetic testing</a></li>
<li class="chapter" data-level="14.2" data-path="applications-of-molecular-genetics.html"><a href="applications-of-molecular-genetics.html#genetically-modified-organisms"><i class="fa fa-check"></i><b>14.2</b> Genetically modified organisms</a></li>
<li class="chapter" data-level="14.3" data-path="applications-of-molecular-genetics.html"><a href="applications-of-molecular-genetics.html#gene-therapy"><i class="fa fa-check"></i><b>14.3</b> Gene therapy</a></li>
</ul></li>
<li class="chapter" data-level="15" data-path="regulation-of-gene-expression.html"><a href="regulation-of-gene-expression.html"><i class="fa fa-check"></i><b>15</b> Regulation of Gene Expression</a><ul>
<li class="chapter" data-level="15.1" data-path="regulation-of-gene-expression.html"><a href="regulation-of-gene-expression.html#regulated-stages-of-gene-expression"><i class="fa fa-check"></i><b>15.1</b> Regulated stages of gene expression</a><ul>
<li class="chapter" data-level="15.1.1" data-path="regulation-of-gene-expression.html"><a href="regulation-of-gene-expression.html#structural"><i class="fa fa-check"></i><b>15.1.1</b> Structural</a></li>
<li class="chapter" data-level="15.1.2" data-path="regulation-of-gene-expression.html"><a href="regulation-of-gene-expression.html#chemical"><i class="fa fa-check"></i><b>15.1.2</b> Chemical</a></li>
</ul></li>
<li class="chapter" data-level="15.2" data-path="regulation-of-gene-expression.html"><a href="regulation-of-gene-expression.html#regulation-of-transcription"><i class="fa fa-check"></i><b>15.2</b> Regulation of transcription</a></li>
<li class="chapter" data-level="15.3" data-path="regulation-of-gene-expression.html"><a href="regulation-of-gene-expression.html#post-transcriptional-regulation"><i class="fa fa-check"></i><b>15.3</b> Post-transcriptional regulation</a></li>
<li class="chapter" data-level="15.4" data-path="regulation-of-gene-expression.html"><a href="regulation-of-gene-expression.html#three-prime-untranslated-regions-and-micrornas"><i class="fa fa-check"></i><b>15.4</b> Three prime untranslated regions and microRNAs</a></li>
<li class="chapter" data-level="15.5" data-path="regulation-of-gene-expression.html"><a href="regulation-of-gene-expression.html#regulation-of-translation"><i class="fa fa-check"></i><b>15.5</b> Regulation of translation</a><ul>
<li class="chapter" data-level="15.5.1" data-path="regulation-of-gene-expression.html"><a href="regulation-of-gene-expression.html#examples-of-gene-regulation"><i class="fa fa-check"></i><b>15.5.1</b> Examples of gene regulation</a></li>
<li class="chapter" data-level="15.5.2" data-path="regulation-of-gene-expression.html"><a href="regulation-of-gene-expression.html#circuitry"><i class="fa fa-check"></i><b>15.5.2</b> Circuitry</a></li>
</ul></li>
<li class="chapter" data-level="15.6" data-path="regulation-of-gene-expression.html"><a href="regulation-of-gene-expression.html#theoretical-circuits"><i class="fa fa-check"></i><b>15.6</b> Theoretical circuits</a></li>
</ul></li>
<li class="chapter" data-level="16" data-path="inheritance-of-complex-traits.html"><a href="inheritance-of-complex-traits.html"><i class="fa fa-check"></i><b>16</b> Inheritance of Complex Traits</a><ul>
<li class="chapter" data-level="16.1" data-path="inheritance-of-complex-traits.html"><a href="inheritance-of-complex-traits.html#qtl-mapping"><i class="fa fa-check"></i><b>16.1</b> QTL mapping</a></li>
<li class="chapter" data-level="16.2" data-path="inheritance-of-complex-traits.html"><a href="inheritance-of-complex-traits.html#quantitative-traits"><i class="fa fa-check"></i><b>16.2</b> Quantitative traits</a></li>
<li class="chapter" data-level="16.3" data-path="inheritance-of-complex-traits.html"><a href="inheritance-of-complex-traits.html#expression-quantitative-trait-loci"><i class="fa fa-check"></i><b>16.3</b> Expression quantitative trait loci</a></li>
<li class="chapter" data-level="16.4" data-path="inheritance-of-complex-traits.html"><a href="inheritance-of-complex-traits.html#gwas"><i class="fa fa-check"></i><b>16.4</b> GWAS</a></li>
</ul></li>
<li class="chapter" data-level="17" data-path="genetic-control-of-development.html"><a href="genetic-control-of-development.html"><i class="fa fa-check"></i><b>17</b> Genetic Control of Development</a><ul>
<li class="chapter" data-level="17.1" data-path="genetic-control-of-development.html"><a href="genetic-control-of-development.html#the-control-of-body-structure"><i class="fa fa-check"></i><b>17.1</b> The control of body structure</a><ul>
<li class="chapter" data-level="17.1.1" data-path="genetic-control-of-development.html"><a href="genetic-control-of-development.html#deep-homology"><i class="fa fa-check"></i><b>17.1.1</b> Deep homology</a></li>
</ul></li>
<li class="chapter" data-level="17.2" data-path="genetic-control-of-development.html"><a href="genetic-control-of-development.html#gene-toolkit"><i class="fa fa-check"></i><b>17.2</b> Gene toolkit</a></li>
</ul></li>
<li class="chapter" data-level="18" data-path="evolutionary-genetics.html"><a href="evolutionary-genetics.html"><i class="fa fa-check"></i><b>18</b> Evolutionary Genetics</a><ul>
<li class="chapter" data-level="18.1" data-path="evolutionary-genetics.html"><a href="evolutionary-genetics.html#the-modern-synthesis-of-the-early-20th-century"><i class="fa fa-check"></i><b>18.1</b> The modern synthesis of the early 20th century</a></li>
<li class="chapter" data-level="18.2" data-path="evolutionary-genetics.html"><a href="evolutionary-genetics.html#four-processes"><i class="fa fa-check"></i><b>18.2</b> Four processes</a><ul>
<li class="chapter" data-level="18.2.1" data-path="evolutionary-genetics.html"><a href="evolutionary-genetics.html#selection"><i class="fa fa-check"></i><b>18.2.1</b> Selection</a></li>
<li class="chapter" data-level="18.2.2" data-path="evolutionary-genetics.html"><a href="evolutionary-genetics.html#dominance"><i class="fa fa-check"></i><b>18.2.2</b> Dominance</a></li>
<li class="chapter" data-level="18.2.3" data-path="evolutionary-genetics.html"><a href="evolutionary-genetics.html#epistasis"><i class="fa fa-check"></i><b>18.2.3</b> Epistasis</a></li>
<li class="chapter" data-level="18.2.4" data-path="evolutionary-genetics.html"><a href="evolutionary-genetics.html#mutation"><i class="fa fa-check"></i><b>18.2.4</b> Mutation</a></li>
</ul></li>
<li class="chapter" data-level="18.3" data-path="evolutionary-genetics.html"><a href="evolutionary-genetics.html#human-genetic-variation"><i class="fa fa-check"></i><b>18.3</b> Human genetic variation</a></li>
</ul></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Textbook For SCI200 Genetics at Roxbury Community College</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="chromosomal-disorders" class="section level1">
<h1><span class="header-section-number">5</span> Chromosomal Disorders</h1>
<p>A <a href="https://en.wikipedia.org/wiki/Chromosome_abnormality">chromosomal disorder</a>, anomaly, aberration, or mutation is a missing, extra, or irregular portion of chromosomal DNA. It can be from a typical number of chromosomes or a structural abnormality in one or more chromosomes. Chromosome mutation was formerly used in a strict sense to mean a change in a chromosomal segment, involving more than one gene. The term “karyotype” refers to the full set of chromosomes from an individual; this can be compared to a “normal” karyotype for the species via genetic testing. A chromosome anomaly may be detected or confirmed in this manner. Chromosome anomalies usually occur when there is an error in cell division followingmeiosis or mitosis. There are many types of chromosome anomalies. They can be organized into two basic groups, numerical and structural anomalies.</p>
<div id="numerical-disorders" class="section level2">
<h2><span class="header-section-number">5.1</span> Numerical disorders</h2>
<p>This is called aneuploidy (an abnormal number of chromosomes), and occurs when an individual either is missing a chromosome from a pair (monosomy) or has more than two chromosomes of a pair (trisomy, tetrasomy, etc.).</p>
<p>An example of trisomy in humans is <a href="https://en.wikipedia.org/wiki/Down_syndrome">Down syndrome</a>, which is a developmental disorder caused by an extra copy of chromosome 21; the disorder is therefore also called trisomy 21. Having an extra copy of this chromosome means that individuals have three copies of each of its genes instead of two, making it difficult for cells to properly control how much protein is made. Producing too much or too little protein can have serious consequences. Genes on chromosome 21 that specifically contribute to the various symptoms of Down syndrome are now being identified. The frequency of Trisomy 21 has been determined to be a function of advanced maternal age.</p>
<p>An example of monosomy is <a href="https://en.wikipedia.org/wiki/Turner_syndrome">Turner syndrome</a>, where the individual is born with only one sex chromosome, an X.</p>
</div>
<div id="structural-abnormalities" class="section level2">
<h2><span class="header-section-number">5.2</span> Structural abnormalities</h2>
<p>When the chromosome’s structure is altered, this can take several forms:</p>
<ul>
<li>Deletions: A portion of the chromosome is missing or deleted. Known disorders in humans include Wolf-Hirschhorn syndrome, which is caused by partial deletion of the short arm of chromosome 4; and Jacobsen syndrome, also called the terminal 11q deletion disorder.</li>
<li>Duplications: A portion of the chromosome is duplicated, resulting in extra genetic material. Known human disorders include Charcot-Marie-Tooth disease type 1A, which may be caused by duplication of the gene encoding peripheral myelin protein 22 (PMP22) on chromosome 17.</li>
<li>Translocations: A portion of one chromosome is transferred to another chromosome. There are two main types of translocations:</li>
<li>Reciprocal translocation: Segments from two different chromosomes have been exchanged.</li>
<li>Robertsonian translocation: An entire chromosome has attached to another at the centromere - in humans these only occur with chromosomes 13, 14, 15, 21, and 22.</li>
<li>Inversions: A portion of the chromosome has broken off, turned upside down, and reattached, therefore the genetic material is inverted.</li>
<li>Insertions: A portion of one chromosome has been deleted from its normal place and inserted into another chromosome.</li>
<li>Rings: A portion of a chromosome has broken off and formed a circle or ring. This can happen with or without loss of genetic material.</li>
<li>Isochromosome: Formed by the mirror image copy of a chromosome segment including the centromere.
Chromosome instability syndromes are a group of disorders characterized by chromosomal instability and breakage. They often lead to an increased tendency to develop certain types of malignancies.</li>
</ul>
</div>
<div id="inheritance" class="section level2">
<h2><span class="header-section-number">5.3</span> Inheritance</h2>
<p>Most chromosome abnormalities occur as an accident in the egg cell or sperm, and therefore the anomaly is present in every cell of the body. Some anomalies, however, can happen after conception, resulting in Mosaicism (where some cells have the anomaly and some do not). Chromosome anomalies can be inherited from a parent or be “de novo”. This is why chromosome studies are often performed on parents when a child is found to have an anomaly. If the parents do not possess the abnormality it was not initially inherited; however it may be transmitted to subsequent generations.</p>
</div>
<div id="acquired-chromosome-abnormalities" class="section level2">
<h2><span class="header-section-number">5.4</span> Acquired chromosome abnormalities</h2>
<p>Most cancers, if not all, could cause chromosome abnormalities, with either the formation of hybrid genes and fusion proteins, deregulation of genes and overexpression of proteins, or loss of tumor suppressor genes (see the “Mitelman Database” and the Atlas of Genetics and Cytogenetics in Oncology and Haematology,). Furthermore, certain consistent chromosomal abnormalities can turn normal cells into a leukemic cell such as the translocation of a gene, resulting in its inappropriate expression.</p>
</div>
<div id="aneuploidy" class="section level2">
<h2><span class="header-section-number">5.5</span> Aneuploidy</h2>
<p>Aneuploidy is the presence of an abnormal number of chromosomes in a cell, for example a human cell having 45 or 47 chromosomes instead of the usual 46. It does not include a difference of one or more complete sets of chromosomes. A cell with any number of complete chromosome sets is called a euploid cell. An extra or missing chromosome is a common cause of genetic disorders, including some human birth defects. Some cancer cells also have abnormal numbers of chromosomes. About 68% of human solid tumors are aneuploid. Aneuploidy originates during cell division when the chromosomes do not separate properly between the two cells. Most cases of aneuploidy in the germline result in miscarriage and the most common extra autosomal chromosomes among live births are 21, 18, and 13.</p>
<p>Chromosome abnormalities are detected in 1 of 160 live human births.</p>
<p>Most cells in the human body have 23 pairs of chromosomes, or a total of 46 chromosomes. (The sperm and egg, or gametes, each have 23 unpaired chromosomes, and red blood cells have no nucleus and no chromosomes).</p>
<p>One copy of each pair is inherited from the mother and the other copy is inherited from the father. The first 22 pairs of chromosomes (called autosomes) are numbered from 1 to 22, from largest to smallest. The 23rd pair of chromosomes are the sex chromosomes. Normal females have two X chromosomes, while normal males have one X chromosome and one Y chromosome. The characteristics of the chromosomes in a cell as they are seen under a light microscope are called the karyotype.</p>
<p>During meiosis, when germ cells divide to create sperm and egg (gametes), each half should have the same number of chromosomes. But sometimes, the whole pair of chromosomes will end up in one gamete, and the other gamete will not get that chromosome at all.</p>
<p>Most embryos cannot survive with a missing or extra autosome (numbered chromosome) and are spontaneously aborted. The most frequent aneuploidy in humans is trisomy 16, although fetuses affected with the full version of this chromosome abnormality do not survive to term (it is possible for surviving individuals to have the mosaic form, where trisomy 16 exists in some cells but not all). The most common aneuploidy that infants can survive with is trisomy 21, which is found in Down syndrome, affecting 1 in 800 births. Trisomy 18 (Edwards syndrome) affects 1 in 6,000 births, and trisomy 13 (Patau syndrome) affects 1 in 10,000 births. 10% of infants with trisomy 18 or 13 reach 1 year of age.</p>
<p>Changes in chromosome number may not necessarily be present in all cells in an individual. When aneuploidy is detected in a fraction of cells in an individual, it is called chromosomal mosaicism. In general, individuals who are mosaic for a chromosomal aneuploidy tend to have a less severe form of the syndrome compared to those with full trisomy. For many of the autosomal trisomies, only mosaic cases survive to term. However, mitotic aneuploidy may be more common than previously recognized in somatic tissues, and aneuploidy is a characteristic of many types of tumorigenesis (see below).</p>
</div>
<div id="mechanisms" class="section level2">
<h2><span class="header-section-number">5.6</span> Mechanisms</h2>
<p>Nondisjunction usually occurs as the result of a weakened mitotic checkpoint, as these checkpoints tend to arrest or delay cell division until all components of the cell are ready to enter the next phase. If a checkpoint is weakened, the cell may fail to ‘notice’ that a chromosome pair is not lined up on the mitotic plate, for example. In such a case, most chromosomes would separate normally (with one chromatid ending up in each cell), while others could fail to separate at all. This would generate a daughter cell lacking a copy and a daughter cell with an extra copy.</p>
<p>Completely inactive mitotic checkpoints may cause nondisjunction at multiple chromosomes, possibly all. Such a scenario could result in each daughter cell possessing a disjoint set of genetic material.</p>
<p>Merotelic attachment occurs when one kinetochore is attached to both mitotic spindle poles. One daughter cell would have a normal complement of chromosomes; the second would lack one. A third daughter cell may end up with the ‘missing’ chromosome.</p>
<p>Multipolar spindles: more than two spindle poles form. Such a mitotic division would result in one daughter cell for each spindle pole; each cell may possess an unpredictable complement of chromosomes.</p>
<p>Monopolar spindle: only a single spindle pole forms. This produces a single daughter cell with its copy number doubled.</p>
<p>A tetraploid intermediate may be produced as the end-result of the monopolar spindle mechanism. In such a case, the cell has double the copy number of a normal cell, and produces double the number of spindle poles as well. This results in four daughter cells with an unpredictable complement of chromosomes, but in the normal copy number.</p>
</div>
<div id="somatic-mosaicism-in-the-nervous-system" class="section level2">
<h2><span class="header-section-number">5.7</span> Somatic mosaicism in the nervous system</h2>
<p>Mosaicism for aneuploid chromosome content may be part of the constitutional make-up of the mammalian brain. In the normal human brain, brain samples from six individuals ranging from 2–86 years of age had mosaicism for chromosome 21 aneuploidy (average of 4% of neurons analyzed). This low-level aneuploidy appears to arise from chromosomal segregation defects during cell division in neuronal precursor cells, and neurons containing such aneuploid chromosome content reportedly integrate into normal circuits. However, recent research using single-cell sequencing has challenged these findings, and has suggested that aneuploidy in the brain is actually very rare.</p>
</div>
<div id="somatic-mosaicism-in-cancer" class="section level2">
<h2><span class="header-section-number">5.8</span> Somatic mosaicism in cancer</h2>
<p>Aneuploidy is consistently observed in virtually all cancers and understanding through what mechanisms it can affect tumor evolution is an important topic of current cancer research.</p>
<p>Somatic mosaicism occurs in virtually all cancer cells, including trisomy 12 in chronic lymphocytic leukemia (CLL) and trisomy 8 in acute myeloid leukemia (AML). However, these forms of mosaic aneuploidy occur through mechanisms distinct from those typically associated with genetic syndromes involving complete or mosaic aneuploidy, such as chromosomal instability (due to mitotic segregation defects in cancer cells). Therefore, the molecular processes that lead to aneuploidy are targets for the development of cancer drugs. Both resveratrol and aspirin have been found in vivo (in mice) to selectively destroy tetraploid cells that may be precursors of aneuploid cells, and activate AMPK, which may be involved in the process.</p>
<p>Alteration of normal mitotic checkpoints are also important tumorogenic events, and these may directly lead to aneuploidy. Loss of tumor suppressor p53 gene often results in genomic instability, which could lead to the aneuploidy genotype.</p>
<p>In addition, genetic syndromes in which an individual is predisposed to breakage of chromosomes (chromosome instability syndromes) are frequently associated with increased risk for various types of cancer, thus highlighting the role of somatic aneuploidy in carcinogenesis.[citation needed]</p>
<p>The ability to evade the immune system appears to be enhanced in tumoral cells with strong aneuploidy. This has therefore suggested that the presence of an abnormal number of chromosomes might be an effective predictive biomarker for response to precise immunotherapy. For example, in melanoma patients, high somatic copy number alterations are associated with less effective response to immune checkpoint blockade anti–CTLA4 (cytotoxic T lymphocyte–associated protein 4) therapy.</p>
</div>
<div id="partial-aneuploidy" class="section level2">
<h2><span class="header-section-number">5.9</span> Partial aneuploidy</h2>
<p>The terms “partial monosomy” and “partial trisomy” are used to describe an imbalance of genetic material caused by loss or gain of part of a chromosome. In particular, these terms would be used in the situation of an unbalanced translocation, where an individual carries a derivative chromosome formed through the breakage and fusion of two different chromosomes. In this situation, the individual would have three copies of part of one chromosome (two normal copies and the portion that exists on the derivative chromosome) and only one copy of part of the other chromosome involved in the derivative chromosome. The formation of one isochromosome results in partial trisomy of the genes present in the isochromosome and partial monosomy of the genes in the lost arm.</p>
</div>
<div id="aneuploidogens" class="section level2">
<h2><span class="header-section-number">5.10</span> Aneuploidogens</h2>
<p>Agents capable of causing aneuploidy are called aneuploidogens. Many mutagenic carcinogens are aneuploidogens. X-rays, for example, may cause aneuploidy by fragmenting the chromosome; it may also target the spindle apparatus. Other chemicals such as colchicine can also produce aneuploidy by affecting microtubule polymerization.</p>
<p>Exposure of males to lifestyle, environmental and/or occupational hazards may increase the risk of spermatozoa aneuploidy. Tobacco smoke contains chemicals that cause DNA damage.[citation needed] Smoking also can induce aneuploidy. For instance, smoking increases chromosome 13 disomy in spermatozoa by 3-fold, and YY disomy by 2-fold.</p>
<p>Occupational exposure to benzene is associated with a 2.8-fold increase of XX disomy and a 2.6-fold increase of YY disomy in spermatozoa.</p>
<p>Pesticides are released to the environment in large quantities so that most individuals have some degree of exposure. The insecticides fenvalerate and carbaryl have been reported to increase spermatozoa aneuploidy. Occupational exposure of pesticide factory workers to fenvalerate is associated with increased spermatozoa DNA damage. Exposure to fenvalerate raised sex chromosome disomy 1.9-fold and disomy of chromosome 18 by 2.6-fold. Exposure of male workers to carbaryl increased DNA fragmentation in spermatozoa, and also increased sex chromosome disomy by 1.7-fold and chromosome 18 disomy by 2.2-fold.</p>
<p>Humans are exposed to perfluorinated compounds (PFCs) in many commercial products. Men contaminated with PFCs in whole blood or seminal plasma have spermatozoa with increased levels of DNA fragmentation and chromosomal aneuploidies.</p>
</div>
<div id="diagnosis" class="section level2">
<h2><span class="header-section-number">5.11</span> Diagnosis</h2>
<p>Germline aneuploidy is typically detected through karyotyping, a process in which a sample of cells is fixed and stained to create the typical light and dark chromosomal banding pattern and a picture of the chromosomes is analyzed. Other techniques include fluorescence in situ hybridization (FISH), quantitative PCR of short tandem repeats, quantitative fluorescence PCR (QF-PCR), quantitative PCR dosage analysis, Quantitative Mass Spectrometry of Single Nucleotide Polymorphisms, and comparative genomic hybridization (CGH).</p>
<p>These tests can also be performed prenatally to detect aneuploidy in a pregnancy, through either amniocentesis or chorionic villus sampling Pregnant women of 35 years or older are offered prenatal testing because the chance of chromosomal aneuploidy increases as the mother’s age increases.</p>
<p>Recent advances have allowed for less invasive testing methods based on the presence of fetal genetic material in maternal blood. See Triple test.</p>

<p>a pedigree
Make a number of estimates of recombination frequency
Calculate a LOD score for each estimate
The estimate with the highest LOD score will be considered the best estimate
The LOD score is calculated as follows:</p>
<p><span class="math inline">\(LOD = Z = \log_{10}\frac{probability\ of\ birth\ sequence\ with\ a\ given\ linkage\ value}{probability\ of\ birth\ sequence\ with\ no\ linkage\ value} = \log_{10}\frac{1-\theta^{N\theta} \times \theta^R}{0.5^{NR+R}}\)</span></p>
<p>NR denotes the number of non-recombinant offspring, and R denotes the number of recombinant offspring. The reason 0.5 is used in the denominator is that any alleles that are completely unlinked (e.g. alleles on separate chromosomes) have a 50% chance of recombination, due to independent assortment. θ is the recombinant fraction, i.e. the fraction of births in which recombination has happened between the studied genetic marker and the putative gene associated with the disease. Thus, it is equal to R / (NR + R).</p>
<p>By convention, a LOD score greater than 3.0 is considered evidence for linkage, as it indicates 1000 to 1 odds that the linkage being observed did not occur by chance. On the other hand, a LOD score less than −2.0 is considered evidence to exclude linkage. Although it is very unlikely that a LOD score of 3 would be obtained from a single pedigree, the mathematical properties of the test allow data from a number of pedigrees to be combined by summing their LOD scores. A LOD score of 3 translates to a p-value of approximately 0.05, and no multiple testing correction (e.g. Bonferroni correction) is required.</p>
<p>Linkage analysis has a number of methodological and theoretical limitations that can significantly increase the type-1 error rate and reduce the power to map human quantitative trait loci (QTL). While linkage analysis was successfully used to identify genetic variants that contribute to rare disorders such as Huntington disease, it did not perform that well when applied to more common disorders such as heart disease or different forms of cancer. An explanation for this is that the genetic mechanisms affecting common disorders are different from those causing rare disorders.</p>
<p>Recombination frequency
Recombination frequency is a measure of genetic linkage and is used in the creation of a genetic linkage map. Recombination frequency (θ) is the frequency with which a single chromosomal crossover will take place between two genes during meiosis. A centimorgan (cM) is a unit that describes a recombination frequency of 1%. In this way we can measure the genetic distance between two loci, based upon their recombination frequency. This is a good estimate of the real distance. Double crossovers would turn into no recombination. In this case we cannot tell if crossovers took place. If the loci we’re analysing are very close (less than 7 cM) a double crossover is very unlikely. When distances become higher, the likelihood of a double crossover increases. As the likelihood of a double crossover increases we systematically underestimate the genetic distance between two loci.</p>
<p>During meiosis, chromosomes assort randomly into gametes, such that the segregation of alleles of one gene is independent of alleles of another gene. This is stated in Mendel’s Second Law and is known as the law of independent assortment. The law of independent assortment always holds true for genes that are located on different chromosomes, but for genes that are on the same chromosome, it does not always hold true.</p>
<p>As an example of independent assortment, consider the crossing of the pure-bred homozygote parental strain with genotype AABB with a different pure-bred strain with genotype aabb. A and a and B and b represent the alleles of genes A and B. Crossing these homozygous parental strains will result in F1 generation offspring that are double heterozygotes with genotype AaBb. The F1 offspring AaBb produces gametes that are AB, Ab, aB, and ab with equal frequencies (25%) because the alleles of gene A assort independently of the alleles for gene B during meiosis. Note that 2 of the 4 gametes (50%)—Ab and aB—were not present in the parental generation. These gametes represent recombinant gametes. Recombinant gametes are those gametes that differ from both of the haploid gametes that made up the original diploid cell. In this example, the recombination frequency is 50% since 2 of the 4 gametes were recombinant gametes.</p>
<p>The recombination frequency will be 50% when two genes are located on different chromosomes or when they are widely separated on the same chromosome. This is a consequence of independent assortment.</p>
<p>When two genes are close together on the same chromosome, they do not assort independently and are said to be linked. Whereas genes located on different chromosomes assort independently and have a recombination frequency of 50%, linked genes have a recombination frequency that is less than 50%.</p>
<p>As an example of linkage, consider the classic experiment by William Bateson and Reginald Punnett.[citation needed] They were interested in trait inheritance in the sweet pea and were studying two genes—the gene for flower colour (P, purple, and p, red) and the gene affecting the shape of pollen grains (L, long, and l, round). They crossed the pure lines PPLL and ppll and then self-crossed the resulting PpLl lines. According to Mendelian genetics, the expected phenotypes would occur in a 9:3:3:1 ratio of PL:Pl:pL:pl. To their surprise, they observed an increased frequency of PL and pl and a decreased frequency of Pl and pL (see table below).</p>
<p>Their experiment revealed linkage between the P and L alleles and the p and l alleles. The frequency of P occurring together with L and with p occurring together with l is greater than that of the recombinant Pl and pL. The recombination frequency is more difficult to compute in an F2 cross than a backcross, but the lack of fit between observed and expected numbers of progeny in the above table indicate it is less than 50%.</p>
<p>The progeny in this case received two dominant alleles linked on one chromosome (referred to as coupling or cis arrangement). However, after crossover, some progeny could have received one parental chromosome with a dominant allele for one trait (e.g. Purple) linked to a recessive allele for a second trait (e.g. round) with the opposite being true for the other parental chromosome (e.g. red and Long). This is referred to as repulsion or a trans arrangement. The phenotype here would still be purple and long but a test cross of this individual with the recessive parent would produce progeny with much greater proportion of the two crossover phenotypes. While such a problem may not seem likely from this example, unfavourable repulsion linkages do appear when breeding for disease resistance in some crops.</p>
<p>The two possible arrangements, cis and trans, of alleles in a double heterozygote are referred to as gametic phases, and phasing is the process of determining which of the two is present in a given individual.</p>
<p>When two genes are located on the same chromosome, the chance of a crossover producing recombination between the genes is related to the distance between the two genes. Thus, the use of recombination frequencies has been used to develop linkage maps or genetic maps.</p>
<p>However, it is important to note that recombination frequency tends to underestimate the distance between two linked genes. This is because as the two genes are located farther apart, the chance of double or even number of crossovers between them also increases. Double or even number of crossovers between the two genes results in them being cosegregated to the same gamete, yielding a parental progeny instead of the expected recombinant progeny. As mentioned above, the Kosambi and Haldane transformations attempt to correct for multiple crossovers.</p>
<p>Variation of recombination frequency
While recombination of chromosomes is an essential process during meiosis, there is a large range of frequency of cross overs across organisms and within species. Sexually dimorphic rates of recombination are termed heterochiasmy, and are observed more often than a common rate between male and females. In mammals, females often have a higher rate of recombination compared to males. It is theorised that there are unique selections acting or meiotic drivers which influence the difference in rates. The difference in rates may also reflect the vastly different environments and conditions of meiosis in oogenesis and spermatogenesis.[citation needed]</p>
<p>Meiosis indicators
With very large pedigrees or with very dense genetic marker data, such as from whole-genome sequencing, it is possible to precisely locate recombinations. With this type of genetic analysis, a meiosis indicator is assigned to each position of the genome for each meiosis in a pedigree. The indicator indicates which copy of the parental chromosome contributes to the transmitted gamete at that position. For example, if the allele from the ‘first’ copy of the parental chromosome is transmitted, a ‘0’ might be assigned to that meiosis. If the allele from the ‘second’ copy of the parental chromosome is transmitted, a ‘1’ would be assigned to that meiosis. The two alleles in the parent came, one each, from two grandparents. These indicators are then used to determine identical-by-descent (IBD) states or inheritance states, which are in turn used to identify genes responsible for diseases.</p>
</div>
<div id="chromosomal-crossover" class="section level2">
<h2><span class="header-section-number">5.12</span> Chromosomal crossover</h2>
<p>Chromosomal crossover, or crossing over, is the exchange of genetic material between two homologous chromosomes non-sister chromatids that results in recombinant chromosomes during sexual reproduction. It is one of the final phases of genetic recombination, which occurs in the pachytene stage of prophase I of meiosis during a process called synapsis. Synapsis begins before the synaptonemal complex develops and is not completed until near the end of prophase I. Crossover usually occurs when matching regions on matching chromosomes break and then reconnect to the other chromosome.</p>
<p>Crossing over was described, in theory, by Thomas Hunt Morgan. He relied on the discovery of Frans Alfons Janssens who described the phenomenon in 1909 and had called it “chiasmatypie”. The term chiasma is linked, if not identical, to chromosomal crossover. Morgan immediately saw the great importance of Janssens’ cytological interpretation of chiasmata to the experimental results of his research on the heredity of Drosophila. The physical basis of crossing over was first demonstrated by Harriet Creighton and Barbara McClintock in 1931.</p>
<p>The linked frequency of crossing over between two gene loci (markers) is the crossing-over value . For fixed set of genetic and environmental conditions, recombination in a particular region of a linkage structure (chromosome) tends to be constant and the same is then true for the crossing-over value which is used in the production of genetic maps.</p>
</div>
<div id="gene-mapping" class="section level2">
<h2><span class="header-section-number">5.13</span> Gene mapping</h2>
<p>Gene mapping describes the methods used to identify the locus of a gene and the distances between genes.</p>
<p>The essence of all genome mapping is to place a collection of molecular markers onto their respective positions on the genome. Molecular markers come in all forms. Genes can be viewed as one special type of genetic markers in the construction of genome maps, and mapped the same way as any other markers.</p>
<p>Genetic mapping vs physical mapping
There are two distinctive types of “Maps” used in the field of genome mapping: genetic maps and physical maps. While both maps are a collection of genetic markers and gene loci, genetic maps’ distances are based on the genetic linkage information, while physical maps use actual physical distances usually measured in number of base pairs. While the physical map could be a more “accurate” representation of the genome, genetic maps often offer insights into the nature of different regions of the chromosome, e.g. the genetic distance to physical distance ratio varies greatly at different genomic regions which reflects different recombination rates, and such rate is often indicative of euchromatic (usually gene-rich) vs heterochromatic (usually gene poor) regions of the genome.</p>
<p>Gene mapping
Researchers begin a genetic map by collecting samples of blood., saliva, or tissue from family members that carry a prominent disease or trait and family members that don’t. The most common sample used in gene mapping, especially in personal genomic tests is saliva. Scientists then isolate DNA from the samples and closely examine it, looking for unique patterns in the DNA of the family members who do carry the disease that the DNA of those who don’t carry the disease don’t have. These unique molecular patterns in the DNA are referred to as polymorphisms, or markers.</p>
<p>The first steps of building a genetic map are the development of genetic markers and a mapping population. The closer two markers are on the chromosome, the more likely they are to be passed on to the next generation together. Therefore, the “co-segregation” patterns of all markers can be used to reconstruct their order. With this in mind, the genotypes of each genetic marker are recorded for both parents and each individual in the following generations. The quality of the genetic maps is largely dependent upon these factors: the number of genetic markers on the map and the size of the mapping population. The two factors are interlinked, as a larger mapping population could increase the “resolution” of the map and prevent the map from being “saturated”.</p>
<p>In gene mapping, any sequence feature that can be faithfully distinguished from the two parents can be used as a genetic marker. Genes, in this regard, are represented by “traits” that can be faithfully distinguished between two parents. Their linkage with other genetic markers is calculated in the same way as if they are common markers and the actual gene loci are then bracketed in a region between the two nearest neighboring markers. The entire process is then repeated by looking at more markers that target that region to map the gene neighborhood to a higher resolution until a specific causative locus can be identified. This process is often referred to as “positional cloning”, and it is used extensively in the study of plant species. One plant species, in particular in which positional cloning is utilized is in maize. The great advantage of genetic mapping is that it can identify the relative position of genes based solely on their phenotypic effect.</p>
<p>Genetic mapping is a way to identify exactly which chromosome has which gene and exactly pinpointing where that gene lies on that particular chromosome. Mapping also acts as a method in determining which gene is most likely recombine based on the distance between two genes. The distance between two genes is measured in units known as centimorgan. A centimorgan is a distance between genes for which one product of meiosis in one hundred is recombinant. The further two genes are from each other, the more likely they are going to recombine. If it were closer, the opposite would occur.[citation needed]</p>
<p>Physical mapping
Since actual base-pair distances are generally hard or impossible to directly measure, physical maps are actually constructed by first shattering the genome into hierarchically smaller pieces. By characterizing each single piece and assembling back together, the overlapping path or “tiling path” of these small fragments would allow researchers to infer physical distances between genomic features. The fragmentation of the genome can be achieved by restriction enzyme cutting or by physically shattering the genome by processes like sonication. Once cut, the DNA fragments are separated by electrophoresis. The resulting pattern of DNA migration (i.e. its genetic fingerprint) is used to identify what stretch of DNA is in the clone. By analyzing the fingerprints, contigs are assembled by automated (FPC) or manual means (pathfinders) into overlapping DNA stretches. Now a good choice of clones can be made to efficiently sequence the clones to determine the DNA sequence of the organism under study.</p>
<p>In physical mapping, there are no direct ways of marking up a specific gene since the mapping does not include any information that concerns traits and functions. Genetic markers can be linked to a physical map by processes like in situ hybridization. By this approach, physical map contigs can be “anchored” onto a genetic map. The clones used in the physical map contigs can then be sequenced on a local scale to help new genetic marker design and identification of the causative loci.</p>
<p>Macrorestriction is a type of physical mapping wherein the high molecular weight DNA is digested with a restriction enzyme having a low number of restriction sites.</p>
<p>There are alternative ways to determine how DNA in a group of clones overlaps without completely sequencing the clones. Once the map is determined, the clones can be used as a resource to efficiently contain large stretches of the genome. This type of mapping is more accurate than genetic maps.</p>
<p>Genome sequencing
Genome sequencing is sometimes mistakenly referred to as “genome mapping” by non-biologists. The process of “shotgun sequencing” resembles the process of physical mapping: it shatters the genome into small fragments, characterizes each fragment, then puts them back together (more recent sequencing technologies are drastically different). While the scope, purpose and process are totally different, a genome assembly can be viewed as the “ultimate” form of physical map, in that it provides in a much better way all the information that a traditional physical map can offer.</p>
<p>Use
Identification of genes is usually the first step in understanding a genome of a species; mapping of the gene is usually the first step of identification of the gene. Gene mapping is usually the starting point of many important downstream studies.</p>
<p>Disease association
The process to identify a genetic element that is responsible for a disease is also referred to as “mapping”. If the locus in which the search is performed is already considerably constrained, the search is called the fine mapping of a gene. This information is derived from the investigation of disease manifestations in large families (genetic linkage) or from populations-based genetic association studies.</p>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="chromosomes-the-vectors-of-heredity.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="the-structure-and-function-of-dna-and-rna.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": false,
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/nikolaussucher/genetics/edit/master/chromosome_variation.rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"download": ["_main.pdf"],
"toc": {
"collapse": "section"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
