<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>8 DNA And The Structure of Eukaryotic Chromosomes | Textbook For SCI200 Genetics at Roxbury Community College</title>
  <meta name="description" content="This is a free online texbook for Genetics." />
  <meta name="generator" content="bookdown 0.16 and GitBook 2.6.7" />

  <meta property="og:title" content="8 DNA And The Structure of Eukaryotic Chromosomes | Textbook For SCI200 Genetics at Roxbury Community College" />
  <meta property="og:type" content="book" />
  
  <meta property="og:image" content="cover.jpg" />
  <meta property="og:description" content="This is a free online texbook for Genetics." />
  <meta name="github-repo" content="nikolaussuchert/genetics" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="8 DNA And The Structure of Eukaryotic Chromosomes | Textbook For SCI200 Genetics at Roxbury Community College" />
  <meta name="twitter:site" content="@NikolausSucher" />
  <meta name="twitter:description" content="This is a free online texbook for Genetics." />
  <meta name="twitter:image" content="cover.jpg" />

<meta name="author" content="Nikolaus Sucher" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="the-structure-and-function-of-dna-and-rna.html"/>
<link rel="next" href="dna-replication.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />












<link rel="stylesheet" href="css/style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><strong><a href="./">Genetics</a></strong></li>

<li class="divider"></li>
<li><a href="index.html#section"></a></li>
<li class="chapter" data-level="" data-path="acknowledgements.html"><a href="acknowledgements.html"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="1" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><i class="fa fa-check"></i><b>1</b> Genetics: The Science of Genes, Genetic Variation and Heredity</a><ul>
<li class="chapter" data-level="1.1" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#mendelian-classical-genetics"><i class="fa fa-check"></i><b>1.1</b> Mendelian (classical) genetics</a></li>
<li class="chapter" data-level="1.2" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#molecular-genetics"><i class="fa fa-check"></i><b>1.2</b> Molecular genetics</a></li>
<li class="chapter" data-level="1.3" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#discrete-inheritance-and-mendels-laws"><i class="fa fa-check"></i><b>1.3</b> Discrete inheritance and Mendel’s laws</a></li>
<li class="chapter" data-level="1.4" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#notation-and-diagrams"><i class="fa fa-check"></i><b>1.4</b> Notation and diagrams</a></li>
<li class="chapter" data-level="1.5" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#multiple-gene-interactions"><i class="fa fa-check"></i><b>1.5</b> Multiple gene interactions</a></li>
<li class="chapter" data-level="1.6" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#molecular-basis-for-inheritance"><i class="fa fa-check"></i><b>1.6</b> Molecular basis for inheritance</a><ul>
<li class="chapter" data-level="1.6.1" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#dna-and-chromosomes"><i class="fa fa-check"></i><b>1.6.1</b> DNA and chromosomes</a></li>
<li class="chapter" data-level="1.6.2" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#reproduction"><i class="fa fa-check"></i><b>1.6.2</b> Reproduction</a></li>
<li class="chapter" data-level="1.6.3" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#recombination-and-genetic-linkage"><i class="fa fa-check"></i><b>1.6.3</b> Recombination and genetic linkage</a></li>
</ul></li>
<li class="chapter" data-level="1.7" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#gene-expression"><i class="fa fa-check"></i><b>1.7</b> Gene expression</a><ul>
<li class="chapter" data-level="1.7.1" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#genetic-code"><i class="fa fa-check"></i><b>1.7.1</b> Genetic code</a></li>
</ul></li>
<li class="chapter" data-level="1.8" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#nature-and-nurture"><i class="fa fa-check"></i><b>1.8</b> Nature and nurture</a></li>
<li class="chapter" data-level="1.9" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#gene-regulation"><i class="fa fa-check"></i><b>1.9</b> Gene regulation</a></li>
<li class="chapter" data-level="1.10" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#genetic-change"><i class="fa fa-check"></i><b>1.10</b> Genetic change</a><ul>
<li class="chapter" data-level="1.10.1" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#mutations"><i class="fa fa-check"></i><b>1.10.1</b> Mutations</a></li>
</ul></li>
<li class="chapter" data-level="1.11" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#natural-selection-and-evolution"><i class="fa fa-check"></i><b>1.11</b> Natural selection and evolution</a></li>
<li class="chapter" data-level="1.12" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#model-organisms"><i class="fa fa-check"></i><b>1.12</b> Model organisms</a></li>
<li class="chapter" data-level="1.13" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#medical-genetics"><i class="fa fa-check"></i><b>1.13</b> Medical Genetics</a></li>
<li class="chapter" data-level="1.14" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#research-methods"><i class="fa fa-check"></i><b>1.14</b> Research methods</a><ul>
<li class="chapter" data-level="1.14.1" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#dna-sequencing-and-genomics"><i class="fa fa-check"></i><b>1.14.1</b> DNA sequencing and genomics</a></li>
</ul></li>
<li class="chapter" data-level="1.15" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#society-and-culture"><i class="fa fa-check"></i><b>1.15</b> Society and culture</a></li>
<li class="chapter" data-level="1.16" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#genetic-counseling"><i class="fa fa-check"></i><b>1.16</b> Genetic counseling</a></li>
<li class="chapter" data-level="1.17" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#human-genetics"><i class="fa fa-check"></i><b>1.17</b> Human genetics</a><ul>
<li class="chapter" data-level="1.17.1" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#genetic-differences-and-inheritance-patterns"><i class="fa fa-check"></i><b>1.17.1</b> Genetic differences and inheritance patterns</a></li>
<li class="chapter" data-level="1.17.2" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#autosomal-dominant-inheritance"><i class="fa fa-check"></i><b>1.17.2</b> Autosomal dominant inheritance</a></li>
<li class="chapter" data-level="1.17.3" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#autosomal-recessive-inheritance"><i class="fa fa-check"></i><b>1.17.3</b> Autosomal recessive inheritance</a></li>
<li class="chapter" data-level="1.17.4" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#x-linked-and-y-linked-inheritance"><i class="fa fa-check"></i><b>1.17.4</b> X-linked and Y-linked inheritance</a></li>
<li class="chapter" data-level="1.17.5" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#pedigrees-analysis"><i class="fa fa-check"></i><b>1.17.5</b> Pedigrees analysis</a></li>
<li class="chapter" data-level="1.17.6" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#karyotype"><i class="fa fa-check"></i><b>1.17.6</b> Karyotype</a></li>
</ul></li>
<li class="chapter" data-level="1.18" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#genomics"><i class="fa fa-check"></i><b>1.18</b> Genomics</a></li>
<li class="chapter" data-level="1.19" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#medical-genetics-1"><i class="fa fa-check"></i><b>1.19</b> Medical genetics</a></li>
<li class="chapter" data-level="1.20" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#population-genetics"><i class="fa fa-check"></i><b>1.20</b> Population genetics</a></li>
<li class="chapter" data-level="1.21" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#eugenics"><i class="fa fa-check"></i><b>1.21</b> Eugenics</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html"><i class="fa fa-check"></i><b>2</b> Reproduction of Cells and Organisms</a><ul>
<li class="chapter" data-level="2.1" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#asexual"><i class="fa fa-check"></i><b>2.1</b> Asexual</a></li>
<li class="chapter" data-level="2.2" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#sexual"><i class="fa fa-check"></i><b>2.2</b> Sexual</a></li>
<li class="chapter" data-level="2.3" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#allogamy"><i class="fa fa-check"></i><b>2.3</b> Allogamy</a></li>
<li class="chapter" data-level="2.4" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#autogamy"><i class="fa fa-check"></i><b>2.4</b> Autogamy</a></li>
<li class="chapter" data-level="2.5" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#mitosis-and-meiosis"><i class="fa fa-check"></i><b>2.5</b> Mitosis and meiosis</a></li>
<li class="chapter" data-level="2.6" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#same-sex"><i class="fa fa-check"></i><b>2.6</b> Same-sex</a></li>
<li class="chapter" data-level="2.7" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#strategies"><i class="fa fa-check"></i><b>2.7</b> Strategies</a></li>
<li class="chapter" data-level="2.8" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#other-types"><i class="fa fa-check"></i><b>2.8</b> Other types</a></li>
<li class="chapter" data-level="2.9" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#asexual-vs.sexual-reproduction"><i class="fa fa-check"></i><b>2.9</b> Asexual vs. sexual reproduction</a></li>
<li class="chapter" data-level="2.10" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#life-without"><i class="fa fa-check"></i><b>2.10</b> Life without</a></li>
<li class="chapter" data-level="2.11" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#binary-fission"><i class="fa fa-check"></i><b>2.11</b> Binary fission</a></li>
<li class="chapter" data-level="2.12" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#fission-of-prokaryotes"><i class="fa fa-check"></i><b>2.12</b> Fission of prokaryotes</a></li>
<li class="chapter" data-level="2.13" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#types-of-binary-fission"><i class="fa fa-check"></i><b>2.13</b> Types of binary fission</a></li>
<li class="chapter" data-level="2.14" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#mitosis"><i class="fa fa-check"></i><b>2.14</b> Mitosis</a></li>
<li class="chapter" data-level="2.15" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#meiosis"><i class="fa fa-check"></i><b>2.15</b> Meiosis</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html"><i class="fa fa-check"></i><b>3</b> Mendelian Inheritance</a><ul>
<li class="chapter" data-level="3.1" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#mendels-genetic-discoveries"><i class="fa fa-check"></i><b>3.1</b> Mendel’s genetic discoveries</a></li>
<li class="chapter" data-level="3.2" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#law-of-dominance-and-uniformity"><i class="fa fa-check"></i><b>3.2</b> Law of Dominance and Uniformity</a></li>
<li class="chapter" data-level="3.3" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#law-of-segregation-of-genes"><i class="fa fa-check"></i><b>3.3</b> Law of Segregation of genes</a></li>
<li class="chapter" data-level="3.4" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#law-of-independent-assortment"><i class="fa fa-check"></i><b>3.4</b> Law of Independent Assortment</a></li>
<li class="chapter" data-level="3.5" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#mendelian-trait"><i class="fa fa-check"></i><b>3.5</b> Mendelian trait</a></li>
<li class="chapter" data-level="3.6" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#non-mendelian-inheritance"><i class="fa fa-check"></i><b>3.6</b> Non-Mendelian inheritance</a></li>
<li class="chapter" data-level="3.7" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#punnett-square"><i class="fa fa-check"></i><b>3.7</b> Punnett square</a></li>
<li class="chapter" data-level="3.8" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#zygosity"><i class="fa fa-check"></i><b>3.8</b> Zygosity</a></li>
<li class="chapter" data-level="3.9" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#monohybrid-cross"><i class="fa fa-check"></i><b>3.9</b> Monohybrid cross</a></li>
<li class="chapter" data-level="3.10" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#dihybrid-cross"><i class="fa fa-check"></i><b>3.10</b> Dihybrid cross</a></li>
<li class="chapter" data-level="3.11" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#forked-line-method"><i class="fa fa-check"></i><b>3.11</b> Forked-line method</a></li>
<li class="chapter" data-level="3.12" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#mendelian-traits-in-humans"><i class="fa fa-check"></i><b>3.12</b> Mendelian traits in humans</a></li>
<li class="chapter" data-level="3.13" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#omim---online-mendelian-inheritance-in-man"><i class="fa fa-check"></i><b>3.13</b> OMIM - Online Mendelian Inheritance in Man</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="chromosomes-the-vectors-of-heredity.html"><a href="chromosomes-the-vectors-of-heredity.html"><i class="fa fa-check"></i><b>4</b> Chromosomes: The Vectors of Heredity</a><ul>
<li class="chapter" data-level="4.1" data-path="chromosomes-the-vectors-of-heredity.html"><a href="chromosomes-the-vectors-of-heredity.html#prokaryotes"><i class="fa fa-check"></i><b>4.1</b> Prokaryotes</a><ul>
<li class="chapter" data-level="4.1.1" data-path="chromosomes-the-vectors-of-heredity.html"><a href="chromosomes-the-vectors-of-heredity.html#structure-in-sequences"><i class="fa fa-check"></i><b>4.1.1</b> Structure in sequences</a></li>
<li class="chapter" data-level="4.1.2" data-path="chromosomes-the-vectors-of-heredity.html"><a href="chromosomes-the-vectors-of-heredity.html#dna-packaging"><i class="fa fa-check"></i><b>4.1.2</b> DNA packaging</a></li>
</ul></li>
<li class="chapter" data-level="4.2" data-path="chromosomes-the-vectors-of-heredity.html"><a href="chromosomes-the-vectors-of-heredity.html#eukaryotes"><i class="fa fa-check"></i><b>4.2</b> Eukaryotes</a><ul>
<li class="chapter" data-level="4.2.1" data-path="chromosomes-the-vectors-of-heredity.html"><a href="chromosomes-the-vectors-of-heredity.html#interphase-chromatin"><i class="fa fa-check"></i><b>4.2.1</b> Interphase chromatin</a></li>
<li class="chapter" data-level="4.2.2" data-path="chromosomes-the-vectors-of-heredity.html"><a href="chromosomes-the-vectors-of-heredity.html#metaphase-chromatin-and-division"><i class="fa fa-check"></i><b>4.2.2</b> Metaphase chromatin and division</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="chromosomes-the-vectors-of-heredity.html"><a href="chromosomes-the-vectors-of-heredity.html#human-chromosomes"><i class="fa fa-check"></i><b>4.3</b> Human chromosomes</a></li>
<li class="chapter" data-level="4.4" data-path="chromosomes-the-vectors-of-heredity.html"><a href="chromosomes-the-vectors-of-heredity.html#number-in-various-organisms"><i class="fa fa-check"></i><b>4.4</b> Number in various organisms</a><ul>
<li class="chapter" data-level="4.4.1" data-path="chromosomes-the-vectors-of-heredity.html"><a href="chromosomes-the-vectors-of-heredity.html#in-eukaryotes"><i class="fa fa-check"></i><b>4.4.1</b> In eukaryotes</a></li>
<li class="chapter" data-level="4.4.2" data-path="chromosomes-the-vectors-of-heredity.html"><a href="chromosomes-the-vectors-of-heredity.html#in-prokaryotes"><i class="fa fa-check"></i><b>4.4.2</b> In prokaryotes</a></li>
</ul></li>
<li class="chapter" data-level="4.5" data-path="chromosomes-the-vectors-of-heredity.html"><a href="chromosomes-the-vectors-of-heredity.html#karyotype-1"><i class="fa fa-check"></i><b>4.5</b> Karyotype</a></li>
<li class="chapter" data-level="4.6" data-path="chromosomes-the-vectors-of-heredity.html"><a href="chromosomes-the-vectors-of-heredity.html#aberrations"><i class="fa fa-check"></i><b>4.6</b> Aberrations</a></li>
<li class="chapter" data-level="4.7" data-path="chromosomes-the-vectors-of-heredity.html"><a href="chromosomes-the-vectors-of-heredity.html#sperm-aneuploidy"><i class="fa fa-check"></i><b>4.7</b> Sperm aneuploidy</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="chromosomal-disorders.html"><a href="chromosomal-disorders.html"><i class="fa fa-check"></i><b>5</b> Chromosomal Disorders</a><ul>
<li class="chapter" data-level="5.1" data-path="chromosomal-disorders.html"><a href="chromosomal-disorders.html#numerical-disorders"><i class="fa fa-check"></i><b>5.1</b> Numerical disorders</a></li>
<li class="chapter" data-level="5.2" data-path="chromosomal-disorders.html"><a href="chromosomal-disorders.html#structural-abnormalities"><i class="fa fa-check"></i><b>5.2</b> Structural abnormalities</a></li>
<li class="chapter" data-level="5.3" data-path="chromosomal-disorders.html"><a href="chromosomal-disorders.html#inheritance"><i class="fa fa-check"></i><b>5.3</b> Inheritance</a></li>
<li class="chapter" data-level="5.4" data-path="chromosomal-disorders.html"><a href="chromosomal-disorders.html#acquired-chromosome-abnormalities"><i class="fa fa-check"></i><b>5.4</b> Acquired chromosome abnormalities</a></li>
<li class="chapter" data-level="5.5" data-path="chromosomal-disorders.html"><a href="chromosomal-disorders.html#aneuploidy"><i class="fa fa-check"></i><b>5.5</b> Aneuploidy</a></li>
<li class="chapter" data-level="5.6" data-path="chromosomal-disorders.html"><a href="chromosomal-disorders.html#mechanisms"><i class="fa fa-check"></i><b>5.6</b> Mechanisms</a></li>
<li class="chapter" data-level="5.7" data-path="chromosomal-disorders.html"><a href="chromosomal-disorders.html#somatic-mosaicism-in-the-nervous-system"><i class="fa fa-check"></i><b>5.7</b> Somatic mosaicism in the nervous system</a></li>
<li class="chapter" data-level="5.8" data-path="chromosomal-disorders.html"><a href="chromosomal-disorders.html#somatic-mosaicism-in-cancer"><i class="fa fa-check"></i><b>5.8</b> Somatic mosaicism in cancer</a></li>
<li class="chapter" data-level="5.9" data-path="chromosomal-disorders.html"><a href="chromosomal-disorders.html#partial-aneuploidy"><i class="fa fa-check"></i><b>5.9</b> Partial aneuploidy</a></li>
<li class="chapter" data-level="5.10" data-path="chromosomal-disorders.html"><a href="chromosomal-disorders.html#aneuploidogens"><i class="fa fa-check"></i><b>5.10</b> Aneuploidogens</a></li>
<li class="chapter" data-level="5.11" data-path="chromosomal-disorders.html"><a href="chromosomal-disorders.html#diagnosis"><i class="fa fa-check"></i><b>5.11</b> Diagnosis</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="genetic-linkage-and-gene-mapping.html"><a href="genetic-linkage-and-gene-mapping.html"><i class="fa fa-check"></i><b>6</b> Genetic Linkage And Gene Mapping</a><ul>
<li class="chapter" data-level="6.1" data-path="genetic-linkage-and-gene-mapping.html"><a href="genetic-linkage-and-gene-mapping.html#linkage-map"><i class="fa fa-check"></i><b>6.1</b> Linkage map</a></li>
<li class="chapter" data-level="6.2" data-path="genetic-linkage-and-gene-mapping.html"><a href="genetic-linkage-and-gene-mapping.html#chromosomal-crossover"><i class="fa fa-check"></i><b>6.2</b> Chromosomal crossover</a></li>
<li class="chapter" data-level="6.3" data-path="genetic-linkage-and-gene-mapping.html"><a href="genetic-linkage-and-gene-mapping.html#gene-mapping"><i class="fa fa-check"></i><b>6.3</b> Gene mapping</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="the-structure-and-function-of-dna-and-rna.html"><a href="the-structure-and-function-of-dna-and-rna.html"><i class="fa fa-check"></i><b>7</b> The Structure And Function of DNA and RNA</a><ul>
<li class="chapter" data-level="7.1" data-path="the-structure-and-function-of-dna-and-rna.html"><a href="the-structure-and-function-of-dna-and-rna.html#rna"><i class="fa fa-check"></i><b>7.1</b> RNA</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="dna-and-the-structure-of-eukaryotic-chromosomes.html"><a href="dna-and-the-structure-of-eukaryotic-chromosomes.html"><i class="fa fa-check"></i><b>8</b> DNA And The Structure of Eukaryotic Chromosomes</a><ul>
<li class="chapter" data-level="8.1" data-path="dna-and-the-structure-of-eukaryotic-chromosomes.html"><a href="dna-and-the-structure-of-eukaryotic-chromosomes.html#dna"><i class="fa fa-check"></i><b>8.1</b> DNA</a></li>
<li class="chapter" data-level="8.2" data-path="dna-and-the-structure-of-eukaryotic-chromosomes.html"><a href="dna-and-the-structure-of-eukaryotic-chromosomes.html#properties"><i class="fa fa-check"></i><b>8.2</b> Properties</a></li>
<li class="chapter" data-level="8.3" data-path="dna-and-the-structure-of-eukaryotic-chromosomes.html"><a href="dna-and-the-structure-of-eukaryotic-chromosomes.html#nucleobase-classification"><i class="fa fa-check"></i><b>8.3</b> Nucleobase classification</a></li>
<li class="chapter" data-level="8.4" data-path="dna-and-the-structure-of-eukaryotic-chromosomes.html"><a href="dna-and-the-structure-of-eukaryotic-chromosomes.html#grooves"><i class="fa fa-check"></i><b>8.4</b> Grooves</a></li>
<li class="chapter" data-level="8.5" data-path="dna-and-the-structure-of-eukaryotic-chromosomes.html"><a href="dna-and-the-structure-of-eukaryotic-chromosomes.html#base-pairing"><i class="fa fa-check"></i><b>8.5</b> Base pairing</a></li>
<li class="chapter" data-level="8.6" data-path="dna-and-the-structure-of-eukaryotic-chromosomes.html"><a href="dna-and-the-structure-of-eukaryotic-chromosomes.html#sense-and-antisense"><i class="fa fa-check"></i><b>8.6</b> Sense and antisense</a></li>
<li class="chapter" data-level="8.7" data-path="dna-and-the-structure-of-eukaryotic-chromosomes.html"><a href="dna-and-the-structure-of-eukaryotic-chromosomes.html#supercoiling"><i class="fa fa-check"></i><b>8.7</b> Supercoiling</a></li>
<li class="chapter" data-level="8.8" data-path="dna-and-the-structure-of-eukaryotic-chromosomes.html"><a href="dna-and-the-structure-of-eukaryotic-chromosomes.html#base-modifications-and-dna-packaging"><i class="fa fa-check"></i><b>8.8</b> Base modifications and DNA packaging</a></li>
<li class="chapter" data-level="8.9" data-path="dna-and-the-structure-of-eukaryotic-chromosomes.html"><a href="dna-and-the-structure-of-eukaryotic-chromosomes.html#damage"><i class="fa fa-check"></i><b>8.9</b> Damage</a></li>
<li class="chapter" data-level="8.10" data-path="dna-and-the-structure-of-eukaryotic-chromosomes.html"><a href="dna-and-the-structure-of-eukaryotic-chromosomes.html#biological-functions"><i class="fa fa-check"></i><b>8.10</b> Biological functions</a></li>
<li class="chapter" data-level="8.11" data-path="dna-and-the-structure-of-eukaryotic-chromosomes.html"><a href="dna-and-the-structure-of-eukaryotic-chromosomes.html#genes-and-genomes"><i class="fa fa-check"></i><b>8.11</b> Genes and genomes</a></li>
<li class="chapter" data-level="8.12" data-path="dna-and-the-structure-of-eukaryotic-chromosomes.html"><a href="dna-and-the-structure-of-eukaryotic-chromosomes.html#transcription-and-translation"><i class="fa fa-check"></i><b>8.12</b> Transcription and translation</a></li>
<li class="chapter" data-level="8.13" data-path="dna-and-the-structure-of-eukaryotic-chromosomes.html"><a href="dna-and-the-structure-of-eukaryotic-chromosomes.html#replication"><i class="fa fa-check"></i><b>8.13</b> Replication</a></li>
<li class="chapter" data-level="8.14" data-path="dna-and-the-structure-of-eukaryotic-chromosomes.html"><a href="dna-and-the-structure-of-eukaryotic-chromosomes.html#interactions-with-proteins"><i class="fa fa-check"></i><b>8.14</b> Interactions with proteins</a></li>
<li class="chapter" data-level="8.15" data-path="dna-and-the-structure-of-eukaryotic-chromosomes.html"><a href="dna-and-the-structure-of-eukaryotic-chromosomes.html#dna-binding-proteins"><i class="fa fa-check"></i><b>8.15</b> DNA-binding proteins</a></li>
<li class="chapter" data-level="8.16" data-path="dna-and-the-structure-of-eukaryotic-chromosomes.html"><a href="dna-and-the-structure-of-eukaryotic-chromosomes.html#dna-modifying-enzymes"><i class="fa fa-check"></i><b>8.16</b> DNA-modifying enzymes</a><ul>
<li class="chapter" data-level="8.16.1" data-path="dna-and-the-structure-of-eukaryotic-chromosomes.html"><a href="dna-and-the-structure-of-eukaryotic-chromosomes.html#nucleases-and-ligases"><i class="fa fa-check"></i><b>8.16.1</b> Nucleases and ligases</a></li>
</ul></li>
<li class="chapter" data-level="8.17" data-path="dna-and-the-structure-of-eukaryotic-chromosomes.html"><a href="dna-and-the-structure-of-eukaryotic-chromosomes.html#topoisomerases-and-helicases"><i class="fa fa-check"></i><b>8.17</b> Topoisomerases and helicases</a></li>
<li class="chapter" data-level="8.18" data-path="dna-and-the-structure-of-eukaryotic-chromosomes.html"><a href="dna-and-the-structure-of-eukaryotic-chromosomes.html#polymerases"><i class="fa fa-check"></i><b>8.18</b> Polymerases</a></li>
<li class="chapter" data-level="8.19" data-path="dna-and-the-structure-of-eukaryotic-chromosomes.html"><a href="dna-and-the-structure-of-eukaryotic-chromosomes.html#genetic-recombination"><i class="fa fa-check"></i><b>8.19</b> Genetic recombination</a></li>
<li class="chapter" data-level="8.20" data-path="dna-and-the-structure-of-eukaryotic-chromosomes.html"><a href="dna-and-the-structure-of-eukaryotic-chromosomes.html#evolution"><i class="fa fa-check"></i><b>8.20</b> Evolution</a></li>
<li class="chapter" data-level="8.21" data-path="dna-and-the-structure-of-eukaryotic-chromosomes.html"><a href="dna-and-the-structure-of-eukaryotic-chromosomes.html#genetic-engineering"><i class="fa fa-check"></i><b>8.21</b> Genetic engineering</a></li>
<li class="chapter" data-level="8.22" data-path="dna-and-the-structure-of-eukaryotic-chromosomes.html"><a href="dna-and-the-structure-of-eukaryotic-chromosomes.html#dna-profiling"><i class="fa fa-check"></i><b>8.22</b> DNA profiling</a></li>
<li class="chapter" data-level="8.23" data-path="dna-and-the-structure-of-eukaryotic-chromosomes.html"><a href="dna-and-the-structure-of-eukaryotic-chromosomes.html#eukaryotic-chromosome-structure"><i class="fa fa-check"></i><b>8.23</b> Eukaryotic chromosome structure</a></li>
<li class="chapter" data-level="8.24" data-path="dna-and-the-structure-of-eukaryotic-chromosomes.html"><a href="dna-and-the-structure-of-eukaryotic-chromosomes.html#history"><i class="fa fa-check"></i><b>8.24</b> History</a></li>
<li class="chapter" data-level="8.25" data-path="dna-and-the-structure-of-eukaryotic-chromosomes.html"><a href="dna-and-the-structure-of-eukaryotic-chromosomes.html#structure"><i class="fa fa-check"></i><b>8.25</b> Structure</a></li>
<li class="chapter" data-level="8.26" data-path="dna-and-the-structure-of-eukaryotic-chromosomes.html"><a href="dna-and-the-structure-of-eukaryotic-chromosomes.html#nucleosome"><i class="fa fa-check"></i><b>8.26</b> Nucleosome</a></li>
<li class="chapter" data-level="8.27" data-path="dna-and-the-structure-of-eukaryotic-chromosomes.html"><a href="dna-and-the-structure-of-eukaryotic-chromosomes.html#structure-1"><i class="fa fa-check"></i><b>8.27</b> Structure</a><ul>
<li class="chapter" data-level="8.27.1" data-path="dna-and-the-structure-of-eukaryotic-chromosomes.html"><a href="dna-and-the-structure-of-eukaryotic-chromosomes.html#structure-of-the-core-particle"><i class="fa fa-check"></i><b>8.27.1</b> Structure of the core particle</a></li>
<li class="chapter" data-level="8.27.2" data-path="dna-and-the-structure-of-eukaryotic-chromosomes.html"><a href="dna-and-the-structure-of-eukaryotic-chromosomes.html#protein-interactions-within-the-nucleosome"><i class="fa fa-check"></i><b>8.27.2</b> Protein interactions within the nucleosome</a></li>
</ul></li>
<li class="chapter" data-level="8.28" data-path="dna-and-the-structure-of-eukaryotic-chromosomes.html"><a href="dna-and-the-structure-of-eukaryotic-chromosomes.html#histone---dna-interactions"><i class="fa fa-check"></i><b>8.28</b> Histone - DNA interactions</a></li>
<li class="chapter" data-level="8.29" data-path="dna-and-the-structure-of-eukaryotic-chromosomes.html"><a href="dna-and-the-structure-of-eukaryotic-chromosomes.html#histone-tail-domains"><i class="fa fa-check"></i><b>8.29</b> Histone tail domains</a></li>
<li class="chapter" data-level="8.30" data-path="dna-and-the-structure-of-eukaryotic-chromosomes.html"><a href="dna-and-the-structure-of-eukaryotic-chromosomes.html#higher-order-structure"><i class="fa fa-check"></i><b>8.30</b> Higher order structure</a></li>
<li class="chapter" data-level="8.31" data-path="dna-and-the-structure-of-eukaryotic-chromosomes.html"><a href="dna-and-the-structure-of-eukaryotic-chromosomes.html#dynamics"><i class="fa fa-check"></i><b>8.31</b> Dynamics</a></li>
<li class="chapter" data-level="8.32" data-path="dna-and-the-structure-of-eukaryotic-chromosomes.html"><a href="dna-and-the-structure-of-eukaryotic-chromosomes.html#nucleosome-sliding"><i class="fa fa-check"></i><b>8.32</b> Nucleosome sliding</a></li>
<li class="chapter" data-level="8.33" data-path="dna-and-the-structure-of-eukaryotic-chromosomes.html"><a href="dna-and-the-structure-of-eukaryotic-chromosomes.html#dna-site-exposure"><i class="fa fa-check"></i><b>8.33</b> DNA site exposure</a></li>
<li class="chapter" data-level="8.34" data-path="dna-and-the-structure-of-eukaryotic-chromosomes.html"><a href="dna-and-the-structure-of-eukaryotic-chromosomes.html#nucleosome-free-region"><i class="fa fa-check"></i><b>8.34</b> Nucleosome free region</a></li>
<li class="chapter" data-level="8.35" data-path="dna-and-the-structure-of-eukaryotic-chromosomes.html"><a href="dna-and-the-structure-of-eukaryotic-chromosomes.html#modulating-nucleosome-structure"><i class="fa fa-check"></i><b>8.35</b> Modulating nucleosome structure</a></li>
<li class="chapter" data-level="8.36" data-path="dna-and-the-structure-of-eukaryotic-chromosomes.html"><a href="dna-and-the-structure-of-eukaryotic-chromosomes.html#histone-post-translational-modifications"><i class="fa fa-check"></i><b>8.36</b> Histone post-translational modifications</a></li>
<li class="chapter" data-level="8.37" data-path="dna-and-the-structure-of-eukaryotic-chromosomes.html"><a href="dna-and-the-structure-of-eukaryotic-chromosomes.html#histone-variants"><i class="fa fa-check"></i><b>8.37</b> Histone variants</a></li>
<li class="chapter" data-level="8.38" data-path="dna-and-the-structure-of-eukaryotic-chromosomes.html"><a href="dna-and-the-structure-of-eukaryotic-chromosomes.html#atp-dependent-nucleosome-remodeling"><i class="fa fa-check"></i><b>8.38</b> ATP-dependent nucleosome remodeling</a></li>
<li class="chapter" data-level="8.39" data-path="dna-and-the-structure-of-eukaryotic-chromosomes.html"><a href="dna-and-the-structure-of-eukaryotic-chromosomes.html#dynamic-nucleosome-remodelling-across-the-yeast-genome"><i class="fa fa-check"></i><b>8.39</b> Dynamic nucleosome remodelling across the Yeast genome</a></li>
<li class="chapter" data-level="8.40" data-path="dna-and-the-structure-of-eukaryotic-chromosomes.html"><a href="dna-and-the-structure-of-eukaryotic-chromosomes.html#nucleosome-assembly-in-vitro"><i class="fa fa-check"></i><b>8.40</b> Nucleosome assembly in vitro</a></li>
<li class="chapter" data-level="8.41" data-path="dna-and-the-structure-of-eukaryotic-chromosomes.html"><a href="dna-and-the-structure-of-eukaryotic-chromosomes.html#nucleosome-assembly-in-vivo"><i class="fa fa-check"></i><b>8.41</b> Nucleosome assembly in vivo</a></li>
<li class="chapter" data-level="8.42" data-path="dna-and-the-structure-of-eukaryotic-chromosomes.html"><a href="dna-and-the-structure-of-eukaryotic-chromosomes.html#histones"><i class="fa fa-check"></i><b>8.42</b> Histones</a><ul>
<li class="chapter" data-level="8.42.1" data-path="dna-and-the-structure-of-eukaryotic-chromosomes.html"><a href="dna-and-the-structure-of-eukaryotic-chromosomes.html#structure-2"><i class="fa fa-check"></i><b>8.42.1</b> Structure</a></li>
</ul></li>
<li class="chapter" data-level="8.43" data-path="dna-and-the-structure-of-eukaryotic-chromosomes.html"><a href="dna-and-the-structure-of-eukaryotic-chromosomes.html#functions-of-histone-modifications"><i class="fa fa-check"></i><b>8.43</b> Functions of histone modifications</a></li>
<li class="chapter" data-level="8.44" data-path="dna-and-the-structure-of-eukaryotic-chromosomes.html"><a href="dna-and-the-structure-of-eukaryotic-chromosomes.html#chemistry-of-histone-modifications"><i class="fa fa-check"></i><b>8.44</b> Chemistry of histone modifications</a></li>
<li class="chapter" data-level="8.45" data-path="dna-and-the-structure-of-eukaryotic-chromosomes.html"><a href="dna-and-the-structure-of-eukaryotic-chromosomes.html#functions-in-transcription"><i class="fa fa-check"></i><b>8.45</b> Functions in transcription</a></li>
</ul></li>
<li class="chapter" data-level="9" data-path="dna-replication.html"><a href="dna-replication.html"><i class="fa fa-check"></i><b>9</b> DNA replication</a><ul>
<li class="chapter" data-level="9.1" data-path="dna-replication.html"><a href="dna-replication.html#eukaryotic-dna-replication"><i class="fa fa-check"></i><b>9.1</b> Eukaryotic DNA replication</a></li>
</ul></li>
<li class="chapter" data-level="10" data-path="transcription.html"><a href="transcription.html"><i class="fa fa-check"></i><b>10</b> Transcription</a><ul>
<li class="chapter" data-level="10.1" data-path="transcription.html"><a href="transcription.html#reverse-transcription"><i class="fa fa-check"></i><b>10.1</b> Reverse transcription</a></li>
</ul></li>
<li class="chapter" data-level="11" data-path="translation.html"><a href="translation.html"><i class="fa fa-check"></i><b>11</b> Translation</a><ul>
<li class="chapter" data-level="11.1" data-path="translation.html"><a href="translation.html#genetic-code-1"><i class="fa fa-check"></i><b>11.1</b> Genetic code</a></li>
</ul></li>
<li class="chapter" data-level="12" data-path="mutation-dna-repair-and-recombination.html"><a href="mutation-dna-repair-and-recombination.html"><i class="fa fa-check"></i><b>12</b> Mutation, DNA Repair And Recombination</a></li>
<li class="chapter" data-level="13" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html"><i class="fa fa-check"></i><b>13</b> Techniques of Molecular Genetics</a><ul>
<li class="chapter" data-level="13.1" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#polymerase-chain-reaction"><i class="fa fa-check"></i><b>13.1</b> Polymerase chain reaction</a></li>
<li class="chapter" data-level="13.2" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#dna-sequencing"><i class="fa fa-check"></i><b>13.2</b> DNA Sequencing</a></li>
<li class="chapter" data-level="13.3" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#rna-seq"><i class="fa fa-check"></i><b>13.3</b> RNA-Seq</a></li>
<li class="chapter" data-level="13.4" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#dna-microarray"><i class="fa fa-check"></i><b>13.4</b> DNA microarray</a></li>
<li class="chapter" data-level="13.5" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#recombinant-dna"><i class="fa fa-check"></i><b>13.5</b> Recombinant DNA</a></li>
<li class="chapter" data-level="13.6" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#molecular-cloning"><i class="fa fa-check"></i><b>13.6</b> Molecular cloning</a></li>
<li class="chapter" data-level="13.7" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#mutagenesis"><i class="fa fa-check"></i><b>13.7</b> Mutagenesis</a></li>
<li class="chapter" data-level="13.8" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#gene-knockout"><i class="fa fa-check"></i><b>13.8</b> Gene knockout</a></li>
<li class="chapter" data-level="13.9" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#rna-interference"><i class="fa fa-check"></i><b>13.9</b> RNA interference</a></li>
<li class="chapter" data-level="13.10" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#cre-lox-recombination"><i class="fa fa-check"></i><b>13.10</b> Cre-Lox recombination</a></li>
<li class="chapter" data-level="13.11" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#crispr-gene-editing"><i class="fa fa-check"></i><b>13.11</b> CRISPR gene editing</a></li>
</ul></li>
<li class="chapter" data-level="14" data-path="genomics-1.html"><a href="genomics-1.html"><i class="fa fa-check"></i><b>14</b> Genomics</a><ul>
<li class="chapter" data-level="14.1" data-path="genomics-1.html"><a href="genomics-1.html#genome-wide-association-study"><i class="fa fa-check"></i><b>14.1</b> Genome-wide association study</a></li>
</ul></li>
<li class="chapter" data-level="15" data-path="applications-of-molecular-genetics.html"><a href="applications-of-molecular-genetics.html"><i class="fa fa-check"></i><b>15</b> Applications of Molecular Genetics</a><ul>
<li class="chapter" data-level="15.1" data-path="applications-of-molecular-genetics.html"><a href="applications-of-molecular-genetics.html#genetic-testing"><i class="fa fa-check"></i><b>15.1</b> Genetic testing</a></li>
<li class="chapter" data-level="15.2" data-path="applications-of-molecular-genetics.html"><a href="applications-of-molecular-genetics.html#genetically-modified-organisms"><i class="fa fa-check"></i><b>15.2</b> Genetically modified organisms</a></li>
<li class="chapter" data-level="15.3" data-path="applications-of-molecular-genetics.html"><a href="applications-of-molecular-genetics.html#gene-therapy"><i class="fa fa-check"></i><b>15.3</b> Gene therapy</a></li>
</ul></li>
<li class="chapter" data-level="16" data-path="regulation-of-gene-expression.html"><a href="regulation-of-gene-expression.html"><i class="fa fa-check"></i><b>16</b> Regulation of Gene Expression</a><ul>
<li class="chapter" data-level="16.1" data-path="regulation-of-gene-expression.html"><a href="regulation-of-gene-expression.html#regulated-stages-of-gene-expression"><i class="fa fa-check"></i><b>16.1</b> Regulated stages of gene expression</a><ul>
<li class="chapter" data-level="16.1.1" data-path="regulation-of-gene-expression.html"><a href="regulation-of-gene-expression.html#structural"><i class="fa fa-check"></i><b>16.1.1</b> Structural</a></li>
<li class="chapter" data-level="16.1.2" data-path="regulation-of-gene-expression.html"><a href="regulation-of-gene-expression.html#chemical"><i class="fa fa-check"></i><b>16.1.2</b> Chemical</a></li>
</ul></li>
<li class="chapter" data-level="16.2" data-path="regulation-of-gene-expression.html"><a href="regulation-of-gene-expression.html#regulation-of-transcription"><i class="fa fa-check"></i><b>16.2</b> Regulation of transcription</a></li>
<li class="chapter" data-level="16.3" data-path="regulation-of-gene-expression.html"><a href="regulation-of-gene-expression.html#post-transcriptional-regulation"><i class="fa fa-check"></i><b>16.3</b> Post-transcriptional regulation</a></li>
<li class="chapter" data-level="16.4" data-path="regulation-of-gene-expression.html"><a href="regulation-of-gene-expression.html#three-prime-untranslated-regions-and-micrornas"><i class="fa fa-check"></i><b>16.4</b> Three prime untranslated regions and microRNAs</a></li>
<li class="chapter" data-level="16.5" data-path="regulation-of-gene-expression.html"><a href="regulation-of-gene-expression.html#regulation-of-translation"><i class="fa fa-check"></i><b>16.5</b> Regulation of translation</a><ul>
<li class="chapter" data-level="16.5.1" data-path="regulation-of-gene-expression.html"><a href="regulation-of-gene-expression.html#examples-of-gene-regulation"><i class="fa fa-check"></i><b>16.5.1</b> Examples of gene regulation</a></li>
<li class="chapter" data-level="16.5.2" data-path="regulation-of-gene-expression.html"><a href="regulation-of-gene-expression.html#circuitry"><i class="fa fa-check"></i><b>16.5.2</b> Circuitry</a></li>
</ul></li>
<li class="chapter" data-level="16.6" data-path="regulation-of-gene-expression.html"><a href="regulation-of-gene-expression.html#theoretical-circuits"><i class="fa fa-check"></i><b>16.6</b> Theoretical circuits</a></li>
</ul></li>
<li class="chapter" data-level="17" data-path="inheritance-of-complex-traits.html"><a href="inheritance-of-complex-traits.html"><i class="fa fa-check"></i><b>17</b> Inheritance of Complex Traits</a></li>
<li class="chapter" data-level="18" data-path="genetic-control-of-development.html"><a href="genetic-control-of-development.html"><i class="fa fa-check"></i><b>18</b> Genetic Control of Development</a><ul>
<li class="chapter" data-level="18.1" data-path="genetic-control-of-development.html"><a href="genetic-control-of-development.html#the-control-of-body-structure"><i class="fa fa-check"></i><b>18.1</b> The control of body structure</a><ul>
<li class="chapter" data-level="18.1.1" data-path="genetic-control-of-development.html"><a href="genetic-control-of-development.html#deep-homology"><i class="fa fa-check"></i><b>18.1.1</b> Deep homology</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="19" data-path="evolutionary-genetics.html"><a href="evolutionary-genetics.html"><i class="fa fa-check"></i><b>19</b> Evolutionary Genetics</a><ul>
<li class="chapter" data-level="19.1" data-path="evolutionary-genetics.html"><a href="evolutionary-genetics.html#the-modern-synthesis-of-the-early-20th-century"><i class="fa fa-check"></i><b>19.1</b> The modern synthesis of the early 20th century</a></li>
<li class="chapter" data-level="19.2" data-path="evolutionary-genetics.html"><a href="evolutionary-genetics.html#four-processes"><i class="fa fa-check"></i><b>19.2</b> Four processes</a><ul>
<li class="chapter" data-level="19.2.1" data-path="evolutionary-genetics.html"><a href="evolutionary-genetics.html#selection"><i class="fa fa-check"></i><b>19.2.1</b> Selection</a></li>
<li class="chapter" data-level="19.2.2" data-path="evolutionary-genetics.html"><a href="evolutionary-genetics.html#dominance"><i class="fa fa-check"></i><b>19.2.2</b> Dominance</a></li>
<li class="chapter" data-level="19.2.3" data-path="evolutionary-genetics.html"><a href="evolutionary-genetics.html#epistasis"><i class="fa fa-check"></i><b>19.2.3</b> Epistasis</a></li>
<li class="chapter" data-level="19.2.4" data-path="evolutionary-genetics.html"><a href="evolutionary-genetics.html#mutation"><i class="fa fa-check"></i><b>19.2.4</b> Mutation</a></li>
</ul></li>
<li class="chapter" data-level="19.3" data-path="evolutionary-genetics.html"><a href="evolutionary-genetics.html#human-genetic-variation"><i class="fa fa-check"></i><b>19.3</b> Human genetic variation</a></li>
</ul></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Textbook For SCI200 Genetics at Roxbury Community College</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="dna-and-the-structure-of-eukaryotic-chromosomes" class="section level1">
<h1><span class="header-section-number">8</span> DNA And The Structure of Eukaryotic Chromosomes</h1>
<div id="dna" class="section level2">
<h2><span class="header-section-number">8.1</span> DNA</h2>
<p><a href="https://en.wikipedia.org/wiki/DNA">Deoxyribonucleic acid (DNA)</a> is a molecule composed of two chains that coil around each other to form a double helix carrying genetic instructions for the development, functioning, growth and reproduction of all known organisms and many viruses. DNA and ribonucleic acid (RNA) are nucleic acids; alongside proteins, lipids and complex carbohydrates (polysaccharides), nucleic acids are one of the four major types of macromolecules that are essential for all known forms of life.</p>
<p>DNA was first isolated by the Swiss physician Friedrich Miescher who, in 1869, discovered a microscopic substance in the pus of discarded surgical bandages. As it resided in the nuclei of cells, he called it “nuclein”. In 1878, Albrecht Kossel isolated the non-protein component of “nuclein”, nucleic acid, and later isolated its five primary nucleobases.</p>
<p>In 1909, Phoebus Levene identified the base, sugar, and phosphate nucleotide unit of the RNA (then named “yeast nucleic acid”). In 1929, Levene identified deoxyribose sugar in “thymus nucleic acid” (DNA). Levene suggested that DNA consisted of a string of four nucleotide units linked together through the phosphate groups (“tetranucleotide hypothesis”). Levene thought the chain was short and the bases repeated in a fixed order. In 1927, Nikolai Koltsov proposed that inherited traits would be inherited via a “giant hereditary molecule” made up of “two mirror strands that would replicate in a semi-conservative fashion using each strand as a template”. In 1928, Frederick Griffith in his experiment discovered that traits of the “smooth” form of Pneumococcus could be transferred to the “rough” form of the same bacteria by mixing killed “smooth” bacteria with the live “rough” form. This system provided the first clear suggestion that DNA carries genetic information.</p>
<p>In 1933, while studying virgin sea urchin eggs, Jean Brachet suggested that DNA is found in the cell nucleus and that RNA is present exclusively in the cytoplasm. At the time, “yeast nucleic acid” (RNA) was thought to occur only in plants, while “thymus nucleic acid” (DNA) only in animals. The latter was thought to be a tetramer, with the function of buffering cellular pH.</p>
<p>In 1937, William Astbury produced the first X-ray diffraction patterns that showed that DNA had a regular structure.</p>
<p>In 1943, Oswald Avery, along with co-workers Colin MacLeod and Maclyn McCarty, identified DNA as the transforming principle, supporting Griffith’s suggestion (Avery–MacLeod–McCarty experiment). DNA’s role in heredity was confirmed in 1952 when Alfred Hershey and Martha Chase in the Hershey–Chase experiment showed that DNA is the genetic material of the enterobacteria phage T2.</p>
<p>Late in 1951, Francis Crick started working with James Watson at the Cavendish Laboratory within the University of Cambridge. In February 1953, Linus Pauling and Robert Corey proposed a model for nucleic acids containing three intertwined chains, with the phosphates near the axis, and the bases on the outside. In May 1952, Raymond Gosling a graduate student working under the supervision of Rosalind Franklin took an X-ray diffraction image, labeled as “Photo 51”, at high hydration levels of DNA. This photo was given to Watson and Crick by Maurice Wilkins and was critical to their obtaining the correct structure of DNA. Franklin told Crick and Watson tat the backbones had to be on the outside. Before then, Linus Pauling, and Watson and Crick, had erroneous models with the chains inside and the bases pointing outwards. Her identification of the space group for DNA crystals revealed to Crick that the two DNA strands were antiparallel.</p>
<p>In February 1953, Watson and Crick completed their model, which is now accepted as the first correct model of the double-helix of DNA. On 28 February 1953 Crick interrupted patrons’ lunchtime at The Eagle pub in Cambridge to announce that he and Watson had “discovered the secret of life”.</p>
<p>In the 25 April 1953 issue of the journal Nature, were published a series of five articles giving the Watson and Crick double-helix structure DNA, and evidence supporting it. The structure was reported in a letter titled “MOLECULAR STRUCTURE OF NUCLEIC ACIDS A Structure for Deoxyribose Nucleic Acid”, in which they said, “It has not escaped our notice that the specific pairing we have postulated immediately suggests a possible copying mechanism for the genetic material.” Followed by a letter from Franklin and Gosling, which was the first publication of their own X-ray diffraction data, and of their original analysis method. Then followed a letter by Wilkins, and two of his colleagues, which contained an analysis of in vivo B-DNA X-ray patterns, and supported the presence in vivo of the Watson and Crick structure.</p>
<p>In 1962, after Franklin’s death, Watson, Crick, and Wilkins jointly received the Nobel Prize in Physiology or Medicine. Nobel Prizes are awarded only to living recipients. A debate continues about who should receive credit for the discovery.</p>
<p>In an influential presentation in 1957, Crick laid out the central dogma of molecular biology, which foretold the relationship between DNA, RNA, and proteins, and articulated the “adaptor hypothesis”. Final confirmation of the replication mechanism that was implied by the double-helical structure followed in 1958 through the Meselson–Stahl experiment. Further work by Crick and co-workers showed that the genetic code was based on non-overlapping triplets of bases, called codons, allowing Har Gobind Khorana, Robert W. Holley, and Marshall Warren Nirenberg to decipher the genetic code. These findings represent the birth of molecular biology.h</p>
<p>The two DNA strands are also known as polynucleotides as they are composed of simpler monomeric units called nucleotides.2] Each nucleotide is composed of one of four nitrogen-containing nucleobases (cytosine [C], guanine [G], adenine [A] or thymine [T]), a sugar called deoxyribose, and a phosphate group. The nucleotides are joined to one another in a chain by covalent bonds between the sugar of one nucleotide and the phosphate of the next, resulting in an alternating sugar-phosphate backbone. The nitrogenous bases of the two separate polynucleotide strands are bound together, according to base pairing rules (A with T and C with G), with hydrogen bonds to make double-stranded DNA. The complementary nitrogenous bases are divided into two groups, pyrimidines and purines. In DNA, the pyrimidines are thymine and cytosine; the purines are adenine and guanine.</p>
<p>Both strands of double-stranded DNA store the same biological information. This information is replicated as and when the two strands separate. A large part of DNA (more than 98% for humans) is non-coding, meaning that these sections do not serve as patterns for protein sequences. The two strands of DNA run in opposite directions to each other and are thus antiparallel. Attached to each sugar is one of four types of nucleobases (informally, bases). It is the sequence of these four nucleobases along the backbone that encodes genetic information. RNA strands are created using DNA strands as a template in a process called transcription, where DNA bases are exchanged for their corresponding bases except in the case of thymine (T), which RNA substitutes for uracil (U). Under the genetic code, these RNA strands specify the sequence of amino acids within proteins in a process called translation.</p>
<p>Within eukaryotic cells, DNA is organized into long structures called chromosomes. Before typical cell division, these chromosomes are duplicated in the process of DNA replication, providing a complete set of chromosomes for each daughter cell. Eukaryotic organisms (animals, plants, fungi and protists) store most of their DNA inside the cell nucleus as nuclear DNA, and some in the mitochondria as mitochondrial DNA or in chloroplasts as chloroplast DNA. In contrast, prokaryotes (bacteria and archaea) store their DNA only in the cytoplasm, in circular chromosomes. Within eukaryotic chromosomes, chromatin proteins, such as histones, compact and organize DNA. These compacting structures guide the interactions between DNA and other proteins, helping control which parts of the DNA are transcribed.</p>
<p>DNA was first isolated by Friedrich Miescher in 1869. Its molecular structure was first identified by Francis Crick and James Watson at the Cavendish Laboratory within the University of Cambridge in 1953, whose model-building efforts were guided by X-ray diffraction data acquired by Raymond Gosling, who was a post-graduate student of Rosalind Franklin. DNA is used by researchers as a molecular tool to explore physical laws and theories, such as the ergodic theorem and the theory of elasticity. The unique material properties of DNA have made it an attractive molecule for material scientists and engineers interested in micro- and nano-fabrication. Among notable advances in this field are DNA origami and DNA-based hybrid materials.</p>
</div>
<div id="properties" class="section level2">
<h2><span class="header-section-number">8.2</span> Properties</h2>
<p>DNA is a long polymer made from repeating units called nucleotides, each of which is usually symbolized by a single letter: either A, T, C, or G. The structure of DNA is dynamic along its length, being capable of coiling into tight loops and other shapes. In all species it is composed of two helical chains, bound to each other by hydrogen bonds. Both chains are coiled around the same axis, and have the same pitch of 34 angstroms (Å) (3.4 nanometres). The pair of chains has a radius of 10 angstroms (1.0 nanometre). According to another study, when measured in a different solution, the DNA chain measured 22 to 26 angstroms wide (2.2 to 2.6 nanometres), and one nucleotide unit measured 3.3 Å (0.33 nm) long. Although each individual nucleotide is very small, a DNA polymer can be very large and contain hundreds of millions, such as in chromosome 1. Chromosome 1 is the largest human chromosome with approximately 220 million base pairs, and would be 85 mm long if straightened.</p>
<p>DNA does not usually exist as a single strand, but instead as a pair of strands that are held tightly together. These two long strands coil around each other, in the shape of a double helix. The nucleotide contains both a segment of the backbone of the molecule (which holds the chain together) and a nucleobase (which interacts with the other DNA strand in the helix). A nucleobase linked to a sugar is called a nucleoside, and a base linked to a sugar and to one or more phosphate groups is called a nucleotide. A biopolymer comprising multiple linked nucleotides (as in DNA) is called a polynucleotide.</p>
<p>The backbone of the DNA strand is made from alternating phosphate and sugar residues. The sugar in DNA is 2-deoxyribose, which is a pentose (five-carbon) sugar. The sugars are joined together by phosphate groups that form phosphodiester bonds between the third and fifth carbon atoms of adjacent sugar rings. These are known as the 3′-end (three prime end), and 5′-end (five prime end) carbons, the prime symbol being used to distinguish these carbon atoms from those of the base to which the deoxyribose forms a glycosidic bond. When imagining DNA, each phosphoryl is normally considered to “belong” to the nucleotide whose 5′ carbon forms a bond therewith. Any DNA strand therefore normally has one end at which there is a phosphoryl attached to the 5′ carbon of a ribose (the 5′ phosphoryl) and another end at which there is a free hydroxyl attached to the 3′ carbon of a ribose (the 3′ hydroxyl). The orientation of the 3′ and 5′ carbons along the sugar-phosphate backbone confers directionality (sometimes called polarity) to each DNA strand. In a nucleic acid double helix, the direction of the nucleotides in one strand is opposite to their direction in the other strand: the strands are antiparallel. The asymmetric ends of DNA strands are said to have a directionality of five prime end (5′ ), and three prime end (3′), with the 5′ end having a terminal phosphate group and the 3′ end a terminal hydroxyl group. One major difference between DNA and RNA is the sugar, with the 2-deoxyribose in DNA being replaced by the alternative pentose sugar ribose in RNA.</p>
<p>The DNA double helix is stabilized primarily by two forces: hydrogen bonds between nucleotides and base-stacking interactions among aromatic nucleobases. In the cytosol of the cell, the conjugated pi bonds of nucleotide bases align perpendicular to the axis of the DNA molecule, minimizing their interaction with the solvation shell. The four bases found in DNA are adenine (A), cytosine (C), guanine (G) and thymine (T). These four bases are attached to the sugar-phosphate to form the complete nucleotide, as shown for adenosine monophosphate. Adenine pairs with thymine and guanine pairs with cytosine, forming A-T and G-C base pairs.</p>
</div>
<div id="nucleobase-classification" class="section level2">
<h2><span class="header-section-number">8.3</span> Nucleobase classification</h2>
<p>The nucleobases are classified into two types: the purines, A and G, which are fused five- and six-membered heterocyclic compounds, and the pyrimidines, the six-membered rings C and T. A fifth pyrimidine nucleobase, uracil (U), usually takes the place of thymine in RNA and differs from thymine by lacking a methyl group on its ring. In addition to RNA and DNA, many artificial nucleic acid analogues have been created to study the properties of nucleic acids, or for use in biotechnology.</p>
</div>
<div id="grooves" class="section level2">
<h2><span class="header-section-number">8.4</span> Grooves</h2>
<p>Twin helical strands form the DNA backbone. Another double helix may be found tracing the spaces, or grooves, between the strands. These voids are adjacent to the base pairs and may provide a binding site. As the strands are not symmetrically located with respect to each other, the grooves are unequally sized. One groove, the major groove, is 22 angstroms (Å) wide and the other, the minor groove, is 12 Å wide. The width of the major groove means that the edges of the bases are more accessible in the major groove than in the minor groove. As a result, proteins such as transcription factors that can bind to specific sequences in double-stranded DNA usually make contact with the sides of the bases exposed in the major groove. This situation varies in unusual conformations of DNA within the cell (see below), but the major and minor grooves are always named to reflect the differences in size that would be seen if the DNA is twisted back into the ordinary B form.</p>
</div>
<div id="base-pairing" class="section level2">
<h2><span class="header-section-number">8.5</span> Base pairing</h2>
<p>In a DNA double helix, each type of nucleobase on one strand bonds with just one type of nucleobase on the other strand. This is called complementary base pairing. Here, purines form hydrogen bonds to pyrimidines, with adenine bonding only to thymine in two hydrogen bonds, and cytosine bonding only to guanine in three hydrogen bonds. This arrangement of two nucleotides binding together across the double helix is called a Watson-Crick base pair. Another type of base pairing is Hoogsteen base pairing where two hydrogen bonds form between guanine and cytosine. As hydrogen bonds are not covalent, they can be broken and rejoined relatively easily. The two strands of DNA in a double helix can thus be pulled apart like a zipper, either by a mechanical force or high temperature. As a result of this base pair complementarity, all the information in the double-stranded sequence of a DNA helix is duplicated on each strand, which is vital in DNA replication. This reversible and specific interaction between complementary base pairs is critical for all the functions of DNA in organisms.</p>
<p>The two types of base pairs form different numbers of hydrogen bonds, AT forming two hydrogen bonds, and GC forming three hydrogen bonds (see figures, right). DNA with high GC-content is more stable than DNA with low GC-content.</p>
<p>As noted above, most DNA molecules are actually two polymer strands, bound together in a helical fashion by noncovalent bonds; this double-stranded (dsDNA) structure is maintained largely by the intrastrand base stacking interactions, which are strongest for G,C stacks. The two strands can come apart—a process known as melting—to form two single-stranded DNA (ssDNA) molecules. Melting occurs at high temperature, low salt and high pH (ow pH also melts DNA, but since DNA is unstable due to acid depurination, low pH is rarely used).</p>
<p>The stability of the dsDNA form depends not only on the GC-content (% G,C basepairs) but also on sequence (since stacking is sequence specific) and also length (longer molecules are more stable). The stability can be measured in various ways; a common way is the “melting temperature”, which is the temperature at which 50% of the ds molecules are converted to ss molecules; melting temperature is dependent on ionic strength and the concentration of DNA. As a result, it is both the percentage of GC base pairs and the overall length of a DNA double helix that determines the strength of the association between the two strands of DNA. Long DNA helices with a high GC-content have stronger-interacting strands, while short helices with high AT content have weaker-interacting strands. In biology, parts of the DNA double helix that need to separate easily, such as the TATAAT Pribnow box in some promoters, tend to have a high AT content, making the strands easier to pull apart.</p>
<p>In the laboratory, the strength of this interaction can be measured by finding the temperature necessary to break the hydrogen bonds, their melting temperature (also called Tm value). When all the base pairs in a DNA double helix melt, the strands separate and exist in solution as two entirely independent molecules. These single-stranded DNA molecules have no single common shape, but some conformations are more stable than others.</p>
</div>
<div id="sense-and-antisense" class="section level2">
<h2><span class="header-section-number">8.6</span> Sense and antisense</h2>
<p>A DNA sequence is called a “sense” sequence if it is the same as that of a messenger RNA copy that is translated into protein. The sequence on the opposite strand is called the “antisense” sequence. Both sense and antisense sequences can exist on different parts of the same strand of DNA (i.e. both strands can contain both sense and antisense sequences). In both prokaryotes and eukaryotes, antisense RNA sequences are produced, but the functions of these RNAs are not entirely clear. One proposal is that antisense RNAs are involved in regulating gene expression through RNA-RNA base pairing.</p>
<p>A few DNA sequences in prokaryotes and eukaryotes, and more in plasmids and viruses, blur the distinction between sense and antisense strands by having overlapping genes. In these cases, some DNA sequences do double duty, encoding one protein when read along one strand, and a second protein when read in the opposite direction along the other strand. In bacteria, this overlap may be involved in the regulation of gene transcription, while in viruses, overlapping genes increase the amount of information that can be encoded within the small viral genome.</p>
</div>
<div id="supercoiling" class="section level2">
<h2><span class="header-section-number">8.7</span> Supercoiling</h2>
<p>Further information: DNA supercoil
DNA can be twisted like a rope in a process called DNA supercoiling. With DNA in its “relaxed” state, a strand usually circles the axis of the double helix once every 10.4 base pairs, but if the DNA is twisted the strands become more tightly or more loosely wound. If the DNA is twisted in the direction of the helix, this is positive supercoiling, and the bases are held more tightly together. If they are twisted in the opposite direction, this is negative supercoiling, and the bases come apart more easily. In nature, most DNA has slight negative supercoiling that is introduced by enzymes called topoisomerases. These enzymes are also needed to relieve the twisting stresses introduced into DNA strands during processes such as transcription and DNA replication.</p>
</div>
<div id="base-modifications-and-dna-packaging" class="section level2">
<h2><span class="header-section-number">8.8</span> Base modifications and DNA packaging</h2>
<p>The expression of genes is influenced by how the DNA is packaged in chromosomes, in a structure called chromatin. Base modifications can be involved in packaging, with regions that have low or no gene expression usually containing high levels of methylation of cytosine bases. DNA packaging and its influence on gene expression can also occur by covalent modifications of the histone protein core around which DNA is wrapped in the chromatin structure or else by remodeling carried out by chromatin remodeling complexes (see Chromatin remodeling). There is, further, crosstalk between DNA methylation and histone modification, so they can coordinately affect chromatin and gene expression.</p>
<p>For one example, cytosine methylation produces 5-methylcytosine, which is important for X-inactivation of chromosomes. The average level of methylation varies between organisms—the worm Caenorhabditis elegans lacks cytosine methylation, while vertebrates have higher levels, with up to 1% of their DNA containing 5-methylcytosine. Despite the importance of 5-methylcytosine, it can deaminate to leave a thymine base, so methylated cytosines are particularly prone to mutations. Other base modifications include adenine methylation in bacteria, the presence of 5-hydroxymethylcytosine in the brain, and the glycosylation of uracil to produce the “J-base” in kinetoplastids.</p>
</div>
<div id="damage" class="section level2">
<h2><span class="header-section-number">8.9</span> Damage</h2>
<p>DNA can be damaged by many sorts of mutagens, which change the DNA sequence. Mutagens include oxidizing agents, alkylating agents and also high-energy electromagnetic radiation such as ultraviolet light and X-rays. The type of DNA damage produced depends on the type of mutagen. For example, UV light can damage DNA by producing thymine dimers, which are cross-links between pyrimidine bases. On the other hand, oxidants such as free radicals or hydrogen peroxide produce multiple forms of damage, including base modifications, particularly of guanosine, and double-strand breaks. A typical human cell contains about 150,000 bases that have suffered oxidative damage. Of these oxidative lesions, the most dangerous are double-strand breaks, as these are difficult to repair and can produce point mutations, insertions, deletions from the DNA sequence, and chromosomal translocations. These mutations can cause cancer. Because of inherent limits in the DNA repair mechanisms, if humans lived long enough, they would all eventually develop cancer. DNA damages that are naturally occurring, due to normal cellular processes that produce reactive oxygen species, the hydrolytic activities of cellular water, etc., also occur frequently. Although most of these damages are repaired, in any cell some DNA damage may remain despite the action of repair processes. These remaining DNA damages accumulate with age in mammalian postmitotic tissues. This accumulation appears to be an important underlying cause of aging.</p>
<p>Many mutagens fit into the space between two adjacent base pairs, this is called intercalation. Most intercalators are aromatic and planar molecules; examples include ethidium bromide, acridines, daunomycin, and doxorubicin. For an intercalator to fit between base pairs, the bases must separate, distorting the DNA strands by unwinding of the double helix. This inhibits both transcription and DNA replication, causing toxicity and mutations. As a result, DNA intercalators may be carcinogens, and in the case of thalidomide, a teratogen. Others such as benzo[a]pyrene diol epoxide and aflatoxin form DNA adducts that induce errors in replication. Nevertheless, due to their ability to inhibit DNA transcription and replication, other similar toxins are also used in chemotherapy to inhibit rapidly growing cancer cells.</p>
</div>
<div id="biological-functions" class="section level2">
<h2><span class="header-section-number">8.10</span> Biological functions</h2>
<p>DNA usually occurs as linear chromosomes in eukaryotes, and circular chromosomes in prokaryotes. The set of chromosomes in a cell makes up its genome; the human genome has approximately 3 billion base pairs of DNA arranged into 46 chromosomes. The information carried by DNA is held in the sequence of pieces of DNA called genes. Transmission of genetic information in genes is achieved via complementary base pairing. For example, in transcription, when a cell uses the information in a gene, the DNA sequence is copied into a complementary RNA sequence through the attraction between the DNA and the correct RNA nucleotides. Usually, this RNA copy is then used to make a matching protein sequence in a process called translation, which depends on the same interaction between RNA nucleotides. In alternative fashion, a cell may simply copy its genetic information in a process called DNA replication. The details of these functions are covered in other articles; here the focus is on the interactions between DNA and other molecules that mediate the function of the genome.</p>
</div>
<div id="genes-and-genomes" class="section level2">
<h2><span class="header-section-number">8.11</span> Genes and genomes</h2>
<p>Genomic DNA is tightly and orderly packed in the process called DNA condensation, to fit the small available volumes of the cell. In eukaryotes, DNA is located in the cell nucleus, with small amounts in mitochondria and chloroplasts. In prokaryotes, the DNA is held within an irregularly shaped body in the cytoplasm called the nucleoid. The genetic information in a genome is held within genes, and the complete set of this information in an organism is called its genotype. A gene is a unit of heredity and is a region of DNA that influences a particular characteristic in an organism. Genes contain an open reading frame that can be transcribed, and regulatorysequences such as promoters and enhancers, which control transcription of the open reading frame.</p>
<p>In many species, only a small fraction of the total sequence of the genome encodes protein. For example, only about 1.5% of the human genome consists of protein-coding exons, with over 50% of human DNA consisting of non-coding repetitive sequences. The reasons for the presence of so much noncoding DNA in eukaryotic genomes and the extraordinary differences in genome size, or C-value, among species, represent a long-standing puzzle known as the “C-value enigma”. However, some DNA sequences that do not code protein may still encode functional non-coding RNA molecules, which are involved in the regulation of gene expression.</p>
<p>Some noncoding DNA sequences play structural roles in chromosomes. Telomeres and centromeres typically contain few genes but are important for the function and stability of chromosomes. An abundant form of noncoding DNA in humans are pseudogenes, which are copies of genes that have been disabled by mutation. These sequences are usually just molecular fossils, although they can occasionally serve as raw genetic material for the creation of new genes through the process of gene duplication and divergence.</p>
</div>
<div id="transcription-and-translation" class="section level2">
<h2><span class="header-section-number">8.12</span> Transcription and translation</h2>
<p>Further information: Genetic code, Transcription (genetics), and Protein biosynthesis
A gene is a sequence of DNA that contains genetic information and can influence the phenotype of an organism. Within a gene, the sequence of bases along a DNA strand defines a messenger RNA sequence, which then defines one or more protein sequences. The relationship between the nucleotide sequences of genes and the amino-acid sequences of proteins is determined by the rules of translation, known collectively as the genetic code. The genetic code consists of three-letter ‘words’ called codons formed from a sequence of three nucleotides (e.g. ACT, CAG, TTT).</p>
<p>In transcription, the codons of a gene are copied into messenger RNA by RNA polymerase. This RNA copy is then decoded by a ribosome that reads the RNA sequence by base-pairing the messenger RNA to transfer RNA, which carries amino acids. Since there are 4 bases in 3-letter combinations, there are 64 possible codons (43 combinations). These encode the twenty standard amino acids, giving most amino acids more than one possible codon. There are also three ‘stop’ or ‘nonsense’ codons signifying the end of the coding region; these are the TAA, TGA, and TAG codons.</p>
</div>
<div id="replication" class="section level2">
<h2><span class="header-section-number">8.13</span> Replication</h2>
<p>Cell division is essential for an organism to grow, but, when a cell divides, it must replicate the DNA in its genome so that the two daughter cells have the same genetic information as their parent. The double-stranded structure of DNA provides a simple mechanism for DNA replication. Here, the two strands are separated and then each strand’s complementary DNA sequence is recreated by an enzyme called DNA polymerase. This enzyme makes the complementary strand by finding the correct base through complementary base pairing and bonding it onto the original strand. As DNA polymerases can only extend a DNA strand in a 5′ to 3′ direction, different mechanisms are used to copy the antiparallel strands of the double helix. In this way, the base on the old strand dictates which base appears on the new strand, and the cell ends up with a perfect copy of its DNA.</p>
</div>
<div id="interactions-with-proteins" class="section level2">
<h2><span class="header-section-number">8.14</span> Interactions with proteins</h2>
<p>All the functions of DNA depend on interactions with proteins. These protein interactions can be non-specific, or the protein can bind specifically to a single DNA sequence. Enzymes can also bind to DNA and of these, the polymerases that copy the DNA base sequence in transcription and DNA replication are particularly important.</p>
</div>
<div id="dna-binding-proteins" class="section level2">
<h2><span class="header-section-number">8.15</span> DNA-binding proteins</h2>
<p>Structural proteins that bind DNA are well-understood examples of non-specific DNA-protein interactions. Within chromosomes, DNA is held in complexes with structural proteins. These proteins organize the DNA into a compact structure called chromatin. In eukaryotes, this structure involves DNA binding to a complex of small basic proteins called histones, while in prokaryotes multiple types of proteins are involved. The histones form a disk-shaped complex called a nucleosome, which contains two complete turns of double-stranded DNA wrapped around its surface. These non-specific interactions are formed through basic residues in the histones, making ionic bonds to the acidic sugar-phosphate backbone of the DNA, and are thus largely independent of the base sequence. Chemical modifications of these basic amino acid residues include methylation, phosphorylation, and acetylation. These chemical changes alter the strength of the interaction between the DNA and the histones, making the DNA more or less accessible to transcription factors and changing the rate of transcription. Other non-specific DNA-binding proteins in chromatin include the high-mobility group proteins, which bind to bent or distorted DNA. These proteins are important in bending arrays of nucleosomes and arranging them into the larger structures that make up chromosomes.</p>
<p>A distinct group of DNA-binding proteins is the DNA-binding proteins that specifically bind single-stranded DNA. In humans, replication protein A is the best-understood member of this family and is used in processes where the double helix is separated, including DNA replication, recombination, and DNA repair. These binding proteins seem to stabilize single-stranded DNA and protect it from forming stem-loops or being degraded by nucleases.</p>
<p>In contrast, other proteins have evolved to bind to particular DNA sequences. The most intensively studied of these are the various transcription factors, which are proteins that regulate transcription. Each transcription factor binds to one particular set of DNA sequences and activates or inhibits the transcription of genes that have these sequences close to their promoters. The transcription factors do this in two ways. Firstly, they can bind the RNA polymerase responsible for transcription, either directly or through other mediator proteins; this locates the polymerase at the promoter and allows it to begin transcription. Alternatively, transcription fctors can bind enzymes that modify the histones at the promoter. This changes the accessibility of the DNA template to the polymerase.</p>
<p>As these DNA targets can occur throughout an organism’s genome, changes in the activity of one type of transcription factor can affect thousands of genes. Consequently, these proteins are often the targets of the signal transduction processes that control responses to environmental changes or cellular differentiation and development. The specificity of these transcription factors’ interactions with DNA come from the proteins making multiple contacts to the edges of the DNA bases, allowing them to “read” the DNA sequence. Most of these base-interactions are made in the major groove, where the bases are most accessible.</p>
</div>
<div id="dna-modifying-enzymes" class="section level2">
<h2><span class="header-section-number">8.16</span> DNA-modifying enzymes</h2>
<div id="nucleases-and-ligases" class="section level3">
<h3><span class="header-section-number">8.16.1</span> Nucleases and ligases</h3>
<p>Nucleases are enzymes that cut DNA strands by catalyzing the hydrolysis of the phosphodiester bonds. Nucleases that hydrolyse nucleotides from the ends of DNA strands are called exonucleases, while endonucleases cut within strands. The most frequently used nucleases in molecular biology are the restriction endonucleases, which cut DNA at specific sequences. For instance, the EcoRV enzyme shown to the left recognizes the 6-base sequence 5′-GATATC-3′ and makes a cut at the horizontal line. In nature, these enzymes protect bacteria against phage infection by digesting the phage DNA when it enters the bacterial cell, actng as part of the restriction modification system. In technology, these sequence-specific nucleases are used in molecular cloning and DNA fingerprinting.</p>
<p>Enzymes called DNA ligases can rejoin cut or broken DNA strands. Ligases are particularly important in lagging strand DNA replication, as they join together the short segments of DNA produced at the replication fork into a complete copy of the DNA template. They are also used in DNA repair and genetic recombination.</p>
</div>
</div>
<div id="topoisomerases-and-helicases" class="section level2">
<h2><span class="header-section-number">8.17</span> Topoisomerases and helicases</h2>
<p>Topoisomerases are enzymes with both nuclease and ligase activity. These proteins change the amount of supercoiling in DNA. Some of these enzymes work by cutting the DNA helix and allowing one section to rotate, thereby reducing its level of supercoiling; the enzyme then seals the DNA break. Other types of these enzymes are capable of cutting one DNA helix and then passing a second strand of DNA through this break, before rejoining the helix. Topoisomerases are required for many processes involving DNA, such as DNA replication and transcription.</p>
<p>Helicases are proteins that are a type of molecular motor. They use the chemical energy in nucleoside triphosphates, predominantly adenosine triphosphate (ATP), to break hydrogen bonds between bases and unwind the DNA double helix into single strands. These enzymes are essential for most processes where enzymes need to access the DNA bases.</p>
</div>
<div id="polymerases" class="section level2">
<h2><span class="header-section-number">8.18</span> Polymerases</h2>
<p>Polymerases are enzymes that synthesize polynucleotide chains from nucleoside triphosphates. The sequence of their products is created based on existing polynucleotide chains—which are called templates. These enzymes function by repeatedly adding a nucleotide to the 3′ hydroxyl group at the end of the growing polynucleotide chain. As a consequence, all polymerases work in a 5′ to 3′ direction. In the active site of these enzymes, the incoming nucleoside triphosphate base-pairs to the template: this allows polymerases to accurately synthesize the complementary strand of their template. Polymerases are classified according to the type of template that they use.</p>
<p>In DNA replication, DNA-dependent DNA polymerases make copies of DNA polynucleotide chains. To preserve biological information, it is essential that the sequence of bases in each copy are precisely complementary to the sequence of bases in the template strand. Many DNA polymerases have a proofreading activity. Here, the polymerase recognizes the occasional mistakes in the synthesis reaction by the lack of base pairing between the mismatched nucleotides. If a mismatch is detected, a 3′ to 5′ exonuclease activity is activated and the incorrect base removed. In most organisms, DNA polymerases function in a large complex called the replisome that contains multiple accessory subunits, such as the DNA clamp or helicases.</p>
<p>RNA-dependent DNA polymerases are a specialized class of polymerases that copy the sequence of an RNA strand into DNA. They include reverse transcriptase, which is a viral enzyme involved in the infection of cells by retroviruses, and telomerase, which is required for the replication of telomeres. For example, HIV reverse transcriptase is an enzyme for AIDS virus replication. Telomerase is an unusual polymerase because it contains its own RNA template as part of its structure. It synthesizes telomeres at the ends of chromosomes. Telomeres prevent fusion of the ends of neighboring chromosomes and protect chromosome ends from damage.</p>
<p>Transcription is carried out by a DNA-dependent RNA polymerase that copies the sequence of a DNA strand into RNA. To begin transcribing a gene, the RNA polymerase binds to a sequence of DNA called a promoter and separates the DNA strands. It then copies the gene sequence into a messenger RNA transcript until it reaches a region of DNA called the terminator, where it halts and detaches from the DNA. As with human DNA-dependent DNA polymerases, RNA polymerase II, the enzyme that transcribes most of the genes in the human genome, operates as part of a large protein complex with multiple regulatory and accessory subunits.</p>
</div>
<div id="genetic-recombination" class="section level2">
<h2><span class="header-section-number">8.19</span> Genetic recombination</h2>
<p>A DNA helix usually does not interact with other segments of DNA, and in human cells, the different chromosomes even occupy separate areas in the nucleus called “chromosome territories”. This physical separation of different chromosomes is important for the ability of DNA to function as a stable repository for information, as one of the few times chromosomes interact is in chromosomal crossover which occurs during sexual reproduction, when genetic recombination occurs. Chromosomal crossover is when two DNA helices break, swap a section and then rejoin.</p>
<p>Recombination allows chromosomes to exchange genetic information and produces new combinations of genes, which increases the efficiency of natural selection and can be important in the rapid evolution of new proteins. Genetic recombination can also be involved in DNA repair, particularly in the cell’s response to double-strand breaks.</p>
<p>The most common form of chromosomal crossover is homologous recombination, where the two chromosomes involved share very similar sequences. Non-homologous recombination can be damaging to cells, as it can produce chromosomal translocations and genetic abnormalities. The recombination reaction is catalyzed by enzymes known as recombinases, such as RAD51. The first step in recombination is a double-stranded break caused by either an endonuclease or damage to the DNA. A series of steps catalyzed in part by the recombinase then leads to joining of the two helices by at least one Holliday junction, in which a segment of a single strand in each helix is annealed to the complementary strand in the other helix. The Holliday junction is a tetrahedral junction structure that can be moved along the pair of chromosomes, swapping one strand for another. The recombination reaction is then halted by cleavage of the junction and re-ligation of the released DNA. Only strands of like polarity exchange DNA during recombination. There are two types of cleavage: east-west cleavage and north-south cleavage. The north-south cleavage nicks both strands of DNA, while the east-west cleavage has one strand of DNA intact. The formation of a Holliday junction during recombination makes it possible for genetic diversity, genes to exchange on chromosomes, and expression of wild-type viral genomes.</p>
</div>
<div id="evolution" class="section level2">
<h2><span class="header-section-number">8.20</span> Evolution</h2>
<p>Further information: RNA world hypothesis
DNA contains the genetic information that allows all forms of life to function, grow and reproduce. However, it is unclear how long in the 4-billion-year history of life DNA has performed this function, as it has been proposed that the earliest forms of life may have used RNA as their genetic material. RNA may have acted as the central part of early cell metabolism as it can both transmit genetic information and carry out catalysis as part of ribozymes. This ancient RNA world where nucleic acid would have been used for both catalysis and genetics may have influenced the evolution of the current genetic code based on four nucleotide bases. This would occur, since the number of different bases in such an organism is a trade-off between a small number of bases increasing replication accuracy and a large number of bases increasing the catalytic efficiency of ribozymes. However, there is no direct evidence of ancient genetic systems, as recovery of DNA from most fossils is impossible because DNA survives in the environment for less than one million years, and slowly degrades into short fragments in solution. Claims for older DNA have been made, most notably a report of the isolation of a viable bacterium from a salt crystal 250 million years old, but these claims are controversial.</p>
<p>Building blocks of DNA (adenine, guanine, and related organic molecules) may have been formed extraterrestrially in outer space. Complex DNA and RNA organic compounds of life, including uracil, cytosine, and thymine, have also been formed in the laboratory under conditions mimicking those found in outer space, using starting chemicals, such as pyrimidine, found in meteorites. Pyrimidine, like polycyclic aromatic hydrocarbons (PAHs), the most carbon-rich chemical found in the universe, may have been formed in red giants or in interstellar cosmic dust and gas clouds.</p>
</div>
<div id="genetic-engineering" class="section level2">
<h2><span class="header-section-number">8.21</span> Genetic engineering</h2>
<p>Further information: Molecular biology, Nucleic acid methods, and Genetic engineering
Methods have been developed to purify DNA from organisms, such as phenol-chloroform extraction, and to manipulate it in the laboratory, such as restriction digests and the polymerase chain reaction. Modern biology and biochemistry make intensive use of these techniques in recombinant DNA technology. Recombinant DNA is a man-made DNA sequence that has been assembled from other DNA sequences. They can be transformed into organisms in the form of plasmids or in the appropriate format, by using a viral vector. The genetically modified organisms produced can be used to produce products such as recombinant proteins, used in medical research, or be grown in agriculture.</p>
</div>
<div id="dna-profiling" class="section level2">
<h2><span class="header-section-number">8.22</span> DNA profiling</h2>
<p>Further information: DNA profiling
Forensic scientists can use DNA in blood, semen, skin, saliva or hair found at a crime scene to identify a matching DNA of an individual, such as a perpetrator. This process is formally termed DNA profiling, also called DNA fingerprinting. In DNA profiling, the lengths of variable sections of repetitive DNA, such as short tandem repeats and minisatellites, are compared between people. This method is usually an extremely reliable technique for identifying a matching DNA. However, identification can be complicated if the scene is contaminated with DNA from several people. DNA profiling was developed in 1984 by British geneticist Sir Alec Jeffreys, and first used in forensic science to convict Colin Pitchfork in the 1988 Enderby murders case.</p>
<p>The development of forensic science and the ability to now obtain genetic matching on minute samples of blood, skin, saliva, or hair has led to re-examining many cases. Evidence can now be uncovered that was scientifically impossible at the time of the original examination. Combined with the removal of the double jeopardy law in some places, this can allow cases to be reopened where prior trials have failed to produce sufficient evidence to convince a jury. People charged with serious crimes may be required to provide a sample of DNA for matching purposes. The most obvious defense to DNA matches obtained forensically is to claim that cross-contamination of evidence has occurred. This has resulted in meticulous strict handling procedures with new cases of serious crime.</p>
<p>DNA profiling is also used successfully to positively identify victims of mass casualty incidents, bodies or body parts in serious accidents, and individual victims in mass war graves, via matching to family members.</p>
<p>DNA profiling is also used in DNA paternity testing to determine if someone is the biological parent or grandparent of a child with the probability of parentage is typically 99.99% when the alleged parent is biologically related to the child. Normal DNA sequencing methods happen after birth, but there are new methods to test paternity while a mother is still pregnant.</p>
</div>
<div id="eukaryotic-chromosome-structure" class="section level2">
<h2><span class="header-section-number">8.23</span> Eukaryotic chromosome structure</h2>
<p>Eukaryotic chromosome structure refers to the levels of packaging from the raw DNA molecules to the chromosomal structures seen during metaphase in mitosis or meiosis. Chromosomes contain long strands of DNA containing genetic information. Compared to prokaryotic chromosomes, eukaryotic chromosomes are much larger in size and are linear chromosomes. Eukaryotic chromosomes are also stored in the nucleus of the cell, while chromosomes of prokaryotic cells are not stored in a nucleus.</p>
</div>
<div id="history" class="section level2">
<h2><span class="header-section-number">8.24</span> History</h2>
<p>Some of the first sientists to recognize the structures now known as chromosomes were Schleiden, Virchow, and Bütschli, O. The term was coined by Heinrich Wilhelm Gottfried von Waldeyer-Hartz, referring to the term chromatin, was introduced by Walther Flemming.</p>
<p>In the 1900s, many scientists concluded the same ideas of heredity that Gregor Mendel had previously. Scientists also discovered plant and animal cells have a central compartment called the nucleus. They soon realized chromosomes were found inside the nucleus and contained different information for many different traits.</p>
</div>
<div id="structure" class="section level2">
<h2><span class="header-section-number">8.25</span> Structure</h2>
<p>In eukaryotes, such as humans, roughly 3.2 billion nucleotides are spread out over 23 different chromosomes (males have a Y chromosome instead of X compared to females). Each chromosome consists enormously long linear DNA molecule associated with proteins that fold and pack the fine thread of DNA into a more compact structure.</p>
<p>The nucleosome consists of a DNA double helix bound to an octamer of core histones (2 dimers of H2A and H2B, and an H3/H4 tetramer). About 147 base pairs of DNA coil around 1 octamer, and ~20 base pairs are sequestered by the addition of the linker histone (H1), and various length of “linker” DNA (~0-100 bp) separate the nucleosomes. The double helix was discovered in 1953 by James Watson and Francis Crick. Other researchers made very important, but unconnected findings about the composition of DNA. But it was up to Watson and Crick to put all of those findings together to come up with a model for DNA. Later, chemist Alexander Todd determined that the backbone of a DNA molecule contained repeating phosphate and deoxyribose sugar groups. The biochemist Erwin Chargaff found that adenine and thymine always paired while cytosine and guanine always paired. High resolution X-ray images of DNA that were obtained by Maurice Wilkins and Rosalind Franklin suggested a helical, or corkscrew like shape.</p>
<p>Packaging of DNA is facilitated by the electrostatic charge distribution: phosphate groups cause DNA to have a negative charge, whilst the histones are positively charged. Most eukaryotic cells contain histones (with a few exceptions) as well as the kingdom Archaea. Histones are positively charged molecules as they contain lysine and arginine in larger quantities and DNA is negatively charged. So they make a strong ionic bond in between them to form nucleosome. Packaging is also accomplished by specialized proteins that bind and fold the DNA. This generates a series of loops and coils that provide increasingly higher levels of organization and prevent the DNA from becoming tangled and unmanageable. This complex of DNA and proteins are called chromatin. But in addition to proteins involved with packaging, chromosomes are associated with proteins involved with DNA replication, DNA repair, and gene expression.</p>
<p>Commonly, many people think the structure of a chromosome is in an “X” shape. But this is only present when the cell divides. Researchers have now been able to model the structure of chromosomes when they are active. This is extremely important because the way that DNA folds up in chromosome structures is linked to the way DNA is used. Scientists have been able to develop the 3D structures of chromosomes in a single cell. The scientists used hundreds of measurements of where different parts of the DNA get close to one another to help create this model. This research was done by scientists at the Department of Biochemistry at Cambridge, working with others from the Babraham Institute and the Weizmann Institute.</p>
</div>
<div id="nucleosome" class="section level2">
<h2><span class="header-section-number">8.26</span> Nucleosome</h2>
<p>A nucleosome is the basic structural unit of DNA packaging in eukaryotes. The structure of a nucleosome consists of a segment of DNA wound around eight histone proteins and resembles thread wrapped around a spool.</p>
<p>DNA must be compacted into nucleosomes to fit within the cell nucleus. In addition to nucleosome wrapping, eukaryotic chromatin is further compacted by being folded into a series of more complex structures, eventually forming a chromosome.</p>
<p>Nucleosomes are thought to carry epigenetically inherited information in the form of covalent modifications of their core histones. Nucleosome positions in the genome are not random, and it is important to know where each nucleosome is located because this determines the accessibility of the DNA to regulatory proteins.</p>
<p>Nucleosomes were first observed as particles in the electron microscope by Don and Ada Olins in 1974, and their existence and structure (as histone octamers surrounded by approximately 200 base pairs of DNA) were proposed by Roger Kornberg. The role of the nucleosome as a general gene repressor was demonstrated by Lorch et al. in vitro, and by Han and Grunstein in vivo in 1987 and 1988, respectively.</p>
<p>The nucleosome core particle consists of approximately 146 base pairs (bp) of DNA wrapped in 1.67 left-handed superhelical turns around a histone octamer, consisting of 2 copies each of the core histones H2A, H2B, H3, and H4. Core particles are connected by stretches of linker DNA, which can be up to about 80 bp long. Technically, a nucleosome is defined as the core particle plus one of these linker regions; however the word is often synonymous with the core particle. Genome-wide nucleosome positioning maps are now available for many model organisms including mouse liver and brain.</p>
<p>Linker histones such as H1 and its isoforms are involved in chromatin compaction and sit at the base of the nucleosome near the DNA entry and exit binding to the linker region of the DNA. Non-condensed nucleosomes without the linker histone resemble “beads on a string of DNA” under an electron microscope.</p>
<p>In contrast to most eukaryotic cells, mature sperm cells largely use protamines to package their genomic DNA, most likely to achieve an even higher packaging ratio. Histone equivalents and a simplified chromatin structure have also been found in Archaea, suggesting that eukaryotes are not the only organisms that use nucleosomes.</p>
</div>
<div id="structure-1" class="section level2">
<h2><span class="header-section-number">8.27</span> Structure</h2>
<div id="structure-of-the-core-particle" class="section level3">
<h3><span class="header-section-number">8.27.1</span> Structure of the core particle</h3>
<p>Pioneering structural studies in the 1980s by Aaron Klug’s group provided the first evidence that an octamer of histone proteins wraps DNA around itself in about 1.7 turns of a left-handed superhelix. In 1997 the first near atomic resolution crystal structure of the nucleosome was solved by the Richmond group, showing the most important details of the particle. The human alpha-satellite palindromic DNA critical to achieving the 1997 nucleosome crystal structure was developed by the Bunick group at Oak Ridge National Laboratory in Tennessee. The structures of over 20 different nucleosome core particles have been solved to date, including those containing histone variants and histones from different species. The structure of the nucleosome core particle is remarkably conserved, and even a change of over 100 residues between frog and yeast histones results in electron density maps with an overall root mean square deviation of only 1.6Å.</p>
<p>The nucleosome core particle (shown in the figure) consists of about 146 base pair of DNA wrapped in 1.67 left-handed superhelical turns around the histone octamer, consisting of 2 copies each of the core histones H2A, H2B, H3, and H4. Adjacent nucleosomes are joined by a stretch of free DNA termed linker DNA (which varies from 10 - 80 bp in length depending on species and tissue type).</p>
<p>Nucleosome core particles are observed when chromatin in interphase is treated to cause the chromatin to unfold partially. The resulting image, via an electron microscope, is “beads on a string”. The string is the DNA, while each bead in the nucleosome is a core particle. The nucleosome core particle is composed of DNA and histone proteins.</p>
<p>Partial DNAse digestion of chromatin reveals its nucleosome structure. Because DNA portions of nucleosome core particles are less accessible for DNAse than linking sections, DNA gets digested into fragments of lengths equal to multiplicity of distance between nucleosomes (180, 360, 540 base pairs etc.). Hence a very characteristic pattern similar to a ladder is visible during gel electrophoresis of that DNA. Such digestion can occur also under natural conditions during apoptosis (“cell suicide” or programmed cell death), because autodestruction of DNA typically is its role.</p>
</div>
<div id="protein-interactions-within-the-nucleosome" class="section level3">
<h3><span class="header-section-number">8.27.2</span> Protein interactions within the nucleosome</h3>
<p>The core histone proteins contains a characteristic structural motif termed the “histone fold”, which consists of three alpha-helices (α1-3) separated by two loops (L1-2). In solution, the histones form H2A-H2B heterodimers and H3-H4 heterotetramers. Histones dimerise about their long α2 helices in an anti-parallel orientation, and, in the case of H3 and H4, two such dimers form a 4-helix bundle stabilised by extensive H3-H3’ interaction. The H2A/H2B dimer binds onto the H3/H4 tetramer due to interactions between H4 and H2B, which include the formation of a hydrophobic cluster. The histone octamer is formed by a central H3/H4 tetramer sandwiched between two H2A/H2B dimers. Due to the highly basic charge of all four core histones, the histone octamer is stable only in the presence of DNA or very high salt concentrations.</p>
</div>
</div>
<div id="histone---dna-interactions" class="section level2">
<h2><span class="header-section-number">8.28</span> Histone - DNA interactions</h2>
<p>The nucleosome contains over 120 direct protein-DNA interactions and several hundred water-mediated ones. Direct protein - DNA interactions are not spread evenly about the octamer surface but rather located at discrete sites. These are due to the formation of two types of DNA binding sites within the octamer; the α1α1 site, which uses the α1 helix from two adjacent histones, and the L1L2 site formed by the L1 and L2 loops. Salt links and hydrogen bonding between both side-chain basic and hydroxyl groups and main-chain amides with the DNA backbone phosphates form the bulk of interactions with the DNA. This is important, given that the ubiquitous distribution of nucleosomes along genomes requires it to be a non-sequence-specific DNA-binding factor. Although nucleosomes tend to prefer some DNA sequences over others, they are capable of binding practically to any sequence, which is thought to be due to the flexibility in the formation of these water-mediated interactions. In addition, non-polar interactions are made between protein side-chains and the deoxyribose groups, and an arginine side-chain intercalates into the DNA minor groove at all 14 sites where it faces the octamer surface. The distribution and strength of DNA-binding sites about the octamer surface distorts the DNA within the nucleosome core. The DNA is non-uniformly bent and also contains twist defects. The twist of free B-form DNA in solution is 10.5 bp per turn. However, the overall twist of nucleosomal DNA is only 10.2 bp per turn, varying from a value of 9.4 to 10.9 bp per turn.</p>
</div>
<div id="histone-tail-domains" class="section level2">
<h2><span class="header-section-number">8.29</span> Histone tail domains</h2>
<p>The histone tail extensions constitute up to 30% by mass of histones, but are not visible in the crystal structures of nucleosomes due to their high intrinsic flexibility, and have been thought to be largely unstructured. The N-terminal tails of histones H3 and H2B pass through a channel formed by the minor grooves of the two DNA strands, protruding from the DNA every 20 bp. The N-terminal tail of histone H4, on the other hand, has a region of highly basic amino acids (16-25), which, in the crystal structure, forms an interaction with the highly acidic surface region of a H2A-H2B dimer of another nucleosome, being potentially relevant for the higher-order structure of nucleosomes. This interaction is thought to occur under physiological conditions also, and suggests that acetylation of the H4 tail distorts the higher-order structure of chromatin.</p>
</div>
<div id="higher-order-structure" class="section level2">
<h2><span class="header-section-number">8.30</span> Higher order structure</h2>
<p>The organization of the DNA that is achieved by the nucleosome cannot fully explain the packaging of DNA observed in the cell nucleus. Further compaction of chromatin into the cell nucleus is necessary, but is not yet well understood. The current understanding is that repeating nucleosomes with intervening “linker” DNA form a 10-nm-fiber, described as “beads on a string”, and have a packing ratio of about five to ten. A chain of nucleosomes can be arranged in a 30 nm fiber, a compacted structure with a packing ratio of ~50 and whose formation is dependent on the presence of the H1 histone.</p>
<p>A crystal structure of a tetranucleosome has been presented and used to build up a proposed structure of the 30 nm fiber as a two-start helix. There is still a certain amount of contention regarding this model, as it is incompatible with recent electron microscopy data. Beyond this, the structure of chromatin is poorly understood, but it is classically suggested that the 30 nm fiber is arranged into loops along a central protein scaffold to form transcriptionally active euchromatin. Further compaction leads to transcriptionally inactive heterochromatin.</p>
</div>
<div id="dynamics" class="section level2">
<h2><span class="header-section-number">8.31</span> Dynamics</h2>
<p>Although the nucleosome is a very stable protein-DNA complex, it is not static and has been shown to undergo a number of different structural re-arrangements including nucleosome sliding and DNA site exposure. Depending on the context, nucleosomes can inhibit or facilitate transcription factor binding. Nucleosome positions are controlled by three major contributions: First, the intrinsic binding affinity of the histone octamer depends on the DNA sequence. Second, the nucleosome can be displaced or recruited by the competitive or cooperative binding of other protein factors. Third, the nucleosome may be actively translocated by ATP-dependent remodeling complexes.</p>
</div>
<div id="nucleosome-sliding" class="section level2">
<h2><span class="header-section-number">8.32</span> Nucleosome sliding</h2>
<p>Work performed in the Bradbury laboratory showed that nucleosomes reconstituted onto the 5S DNA positioning sequence were able to reposition themselves translationally onto adjacent sequences when incubated thermally. Later work showed that this repositioning did not require disruption of the histone octamer but was consistent with nucleosomes being able to “slide” along the DNA in cis. In 2008, it was further revealed that CTCF binding sites act as nucleosome positioning anchors so that, when used to align various genomic signals, multiple flanking nucleosomes can be readily identified. Although nucleosomes are intrinsically mobile, eukaryotes have evolved a large family of ATP-dependent chromatin remodelling enzymes to alter chromatin structure, many of which do so via nucleosome sliding. In 2012, Beena Pillai’s laboratory has demonstrated that nucleosome sliding is one of the possible mechanism for large scale tissue specific expression of genes. The work shows that the transcription start site for genes expressed in a particular tissue, are nucleosome depleted while, the same set of genes in other tissue where they are not expressed, are nucleosome bound.</p>
</div>
<div id="dna-site-exposure" class="section level2">
<h2><span class="header-section-number">8.33</span> DNA site exposure</h2>
<p>Work from the Widom laboratory has shown that nucleosomal DNA is in equilibrium between a wrapped and unwrapped state. Measurements of these rates using time-resolved FRET revealed that DNA within the nucleosome remains fully wrapped for only 250 ms before it is unwrapped for 10-50 ms and then rapidly rewrapped. This implies that DNA does not need to be actively dissociated from the nucleosome but that there is a significant fraction of time during which it is fully accessible. Indeed, this can be extended to the observation that introducing a DNA-binding sequence within the nucleosome increases the accessibility of adjacent regions of DNA when bound. This propensity for DNA within the nucleosome to “breathe” has important functional consequences for all DNA-binding proteins that operate in a chromatin environment. In particular, the dynamic breathing of nucleosomes plays an important role in restricting the advancement of RNA polymerase II during transcription elongation.</p>
</div>
<div id="nucleosome-free-region" class="section level2">
<h2><span class="header-section-number">8.34</span> Nucleosome free region</h2>
<p>Promoters of active genes have nucleosome free regions (NFR). This allows for promoter DNA accessibility to various proteins, such as transcription factors. Nucleosome free region typically spans for 200 nucleotides in S. cerevisae Well-positioned nucleosomes form boundaries of NFR. These nucleosomes are called +1-nucleosome and −1-nucleosome and are located at canonical distances downstream and upstream, respectively, from transcription start site. +1-nucleosome and several downstream nucleosomes also tend to incorporate H2A.Z histone variant.</p>
</div>
<div id="modulating-nucleosome-structure" class="section level2">
<h2><span class="header-section-number">8.35</span> Modulating nucleosome structure</h2>
<p>Eukaryotic genomes are ubiquitously associated into chromatin; however, cells must spatially and temporally regulate specific loci independently of bulk chromatin. In order to achieve the high level of control required to co-ordinate nuclear processes such as DNA replication, repair, and transcription, cells have developed a variety of means to locally and specifically modulate chromatin structure and function. This can involve covalent modification of histones, the incorporation of histone variants, and non-covalent remodelling by ATP-dependent remodeling enzymes.</p>
</div>
<div id="histone-post-translational-modifications" class="section level2">
<h2><span class="header-section-number">8.36</span> Histone post-translational modifications</h2>
<p>Since they were discovered in the mid-1960s, histone modifications have been predicted to affect transcription. The fact that most of the early post-translational modifications found were concentrated within the tail extensions that protrude from the nucleosome core lead to two main theories regarding the mechanism of histone modification. The first of the theories suggested that they may affect electrostatic interactions between the histone tails and DNA to “loosen” chromatin structure. Later it was proposed that combinations of these modifications may create binding epitopes with which to recruit other proteins. Recently, given that more modifications have been found in the structured regions of histones, it has been put forward that these modifications may affect histone-DNA and histone-histone interactions within the nucleosome core. Modifications (such as acetylation or phosphorylation) that lower the charge of the globular histone core are predicted to “loosen” core-DNA association; the strength of the effect depends on location of the modification within the core. Some modifications have been shown to be correlated with gene silencing; others seem to be correlated with gene activation. Common modifications include acetylation, methylation, or ubiquitination of lysine; methylation of arginine; and phosphorylation of serine. The information stored in this way is considered epigenetic, since it is not encoded in the DNA but is still inherited to daughter cells. The maintenance of a repressed or activated status of a gene is often necessary for cellular differentiation.</p>
</div>
<div id="histone-variants" class="section level2">
<h2><span class="header-section-number">8.37</span> Histone variants</h2>
<p>Although histones are remarkably conserved throughout evolution, several variant forms have been identified. This diversification of histone function is restricted to H2A and H3, with H2B and H4 being mostly invariant. H2A can be replaced by H2AZ (which leads to reduced nucleosome stability) or H2AX (which is associated with DNA repair and T cell differentiation), whereas the inactive X chromosomes in mammals are enriched in macroH2A. H3 can be replaced by H3.3 (which correlates with activate genes and regulatory elements) and in centromeres H3 is replaced by CENPA.</p>
</div>
<div id="atp-dependent-nucleosome-remodeling" class="section level2">
<h2><span class="header-section-number">8.38</span> ATP-dependent nucleosome remodeling</h2>
<p>A number of distinct reactions are associated with the term ATP-dependent chromatin remodeling. Remodeling enzymes have been shown to slide nucleosomes along DNA, disrupt histone-DNA contacts to the extent of destabilizing the H2A/H2B dimer and to generate negative superhelical torsion in DNA and chromatin. Recently, the Swr1 remodeling enzyme has been shown to introduce the variant histone H2A.Z into nucleosomes. At present, it is not clear if all of these represent distinct reactions or merely alternative outcomes of a common mechanism. What is shared between all, and indeed the hallmark of ATP-dependent chromatin remodeling, is that they all result in altered DNA accessibility.</p>
<p>Studies looking at gene activation in vivo and, more astonishingly, remodeling in vitro have revealed that chromatin remodeling events and transcription-factor binding are cyclical and periodic in nature. While the consequences of this for the reaction mechanism of chromatin remodeling are not known, the dynamic nature of the system may allow it to respond faster to external stimuli. A recent study indicates that nucleosome positions change significantly during mouse embryonic stem cell development, and these changes are related to binding of developmental transcription factors.</p>
</div>
<div id="dynamic-nucleosome-remodelling-across-the-yeast-genome" class="section level2">
<h2><span class="header-section-number">8.39</span> Dynamic nucleosome remodelling across the Yeast genome</h2>
<p>Studies in 2007 have catalogued nucleosome positions in yeast and shown that nucleosomes are depleted in promoter regions and origins of replication. About 80% of the yeast genome appears to be covered by nucleosomes and the pattern of nucleosome positioning clearly relates to DNA regions that regulate transcription, regions that are transcribed and regions that initiate DNA replication. Most recently, a new study examined dynamic changes in nucleosome repositioning during a global transcriptional reprogramming event to elucidate the effects on nucleosome displacement during genome-wide transcriptional changes in yeast (Saccharomyces cerevisiae). The results suggested that nucleosomes that were localized to promoter regions are displaced in response to stress (like heat shock). In addition, the removal of nucleosomes usually corresponded to transcriptional activation and the replacement of nucleosomes usually corresponded to transcriptional repression, presumably because transcription factor binding sites became more or less accessible, respectively. In general, only one or two nucleosomes were repositioned at the promoter to effect these transcriptional changes. However, even in chromosomal regions that were not associated with transcriptional changes, nucleosome repositioning was observed, suggesting that the covering and uncovering of transcriptional DNA does not necessarily produce a transcriptional event. After transcription, the rDNA region has to protected from any damage, it suggested HMGB proteins play a major role in protecting the nucleosome free region.</p>
</div>
<div id="nucleosome-assembly-in-vitro" class="section level2">
<h2><span class="header-section-number">8.40</span> Nucleosome assembly in vitro</h2>
<p>Nucleosomes can be assembled in vitro by either using purified native or recombinant histones. One standard technique of loading the DNA around the histones involves the use of salt dialysis. A reaction consisting of the histone octamers and a naked DNA template can be incubated together at a salt concentration of 2 M. By steadily decreasing the salt concentration, the DNA will equilibrate to a position where it is wrapped around the histone octamers, forming nucleosomes. In appropriate conditions, this reconstitution process allows for the nucleosome positioning affinity of a given sequence to be mapped experimentally.</p>
<p>Disulfide crosslinked nucleosome core particles
A recent advance in the production of nucleosome core particles with enhanced stability involves site-specific disulfide crosslinks. Two different crosslinks can be introduced into the nucleosome core particle. A first one crosslinks the two copies of H2A via an introduced cysteine (N38C) resulting in histone octamer which is stable against H2A/H2B dimer loss during nucleosome reconstitution. A second crosslink can be introduced between the H3 N-terminal histone tail and the nucleosome DNA ends via an incorporated convertible nucleotide. The DNA-histone octamer crosslink stabilizes the nucleosome core particle against DNA dissociation at very low particle concentrations and at elevated salt concentrations.</p>
</div>
<div id="nucleosome-assembly-in-vivo" class="section level2">
<h2><span class="header-section-number">8.41</span> Nucleosome assembly in vivo</h2>
<p>Nucleosomes are the basic packing unit of DNA built from histone proteins around which DNA is coiled. They serve as a scaffold for formation of higher order chromatin structure as well as for a layer of regulatory control of gene expression. Nucleosomes are quickly assembled onto newly synthesized DNA behind the replication fork.</p>
<p>H3 and H4
Histones H3 and H4 from disassembled old nucleosomes are kept in the vicinity and randomly distributed on the newly synthesized DNA. They are assembled by the chromatin assembly factor-1 (CAF-1) complex, which consists of three subunits (p150, p60, and p48). Newly synthesized H3 and H4 are assembled by the replication coupling assembly factor (RCAF). RCAF contains the subunit Asf1, which binds to newly synthesized H3 and H4 proteins. The old H3 and H4 proteins retain their chemical modifications which contributes to the passing down of the epigenetic signature. The newly synthesized H3 and H4 proteins are gradually acetylated at different lysine residues as part of the chromatin maturation process. It is also thought that the old H3 and H4 proteins in the new nucleosomes recruit histone modifying enzymes that mark the new histones, contributing to epigenetic memory.</p>
<p>H2A and H2B
In contrast to old H3 and H4, the old H2A and H2B histone proteins are released and degraded; therefore, newly assembled H2A and H2B proteins are incorporated into new nucleosomes. H2A and H2B are assembled into dimers which are then loaded onto nucleosomes by the nucleosome assembly protein-1 (NAP-1) which also assists with nucleosome sliding. The nucleosomes are also spaced by ATP-dependent nucleosome-remodeling complexes containing enzymes such as Isw1 Ino80, and Chd1, and subsequently assembled into higher order structure.</p>
</div>
<div id="histones" class="section level2">
<h2><span class="header-section-number">8.42</span> Histones</h2>
<p>Classes and histone variants
Five major families of histones exist: H1/H5, H2A, H2B, H3, and H4. Histones H2A, H2B, H3 and H4 are known as the core histones, while histones H1/H5 are known as the linker histones.</p>
<p>The core histones all exist as dimers, which are similar in that they all possess the histone fold domain: three alpha helices linked by two loops. It is this helical structure that allows for interaction between distinct dimers, particularly in a head-tail fashion (also called the handshake motif). The resulting four distinct dimers then come together to form one octameric nucleosome core, approximately 63 Angstroms in diameter a solenoid (DNA)-like particle). Around 146 base pairs (bp) of DNA wrap around this core particle 1.65 times in a left-handed super-helical turn to give a particle of around 100 Angstroms across. The linker histone H1 binds the nucleosome at the entry and exit sites of the DNA, thus locking the DNA into place and allowing the formation of higher order structure. The most basic such formation is the 10 nm fiber or beads on a string conformation. This involves the wrapping of DNA around nucleosomes with approximately 50 base pairs of DNA separating each pair of nucleosomes (also referred to as linker DNA). Higher-order structures include the 30 nm fiber (forming an irregular zigzag) and 100 nm fiber, these being the structures found in normal cells. During mitosis and meiosis, the condensed chromosomes are assembled through interactions between nucleosomes and other regulatory proteins.</p>
<p>Histones are subdivided into canonical replication-dependent histones that are expressed during the S-phase of cell cycle and replication-independent histone variants, expressed during the whole cell cycle. In animals, genes encoding canonical histones are typically clustered along the chromosome, lack introns and use a stem loop structure at the 3’ end instead of a polyA tail. Genes encoding histone variants are usually not clustered, have introns and their mRNAs are regulated with polyA tails. Complex multicellular organisms typically have a higher number of histone variants providing a variety of different functions. Recent data are accumulating about the roles of diverse histone variants highlighting the functional links between variants and the delicate regulation of organism development. Histone variants from different organisms, their classification and variant specific features can be found in “HistoneDB 2.0 - Variants” database.</p>
<div id="structure-2" class="section level3">
<h3><span class="header-section-number">8.42.1</span> Structure</h3>
<p>The nucleosome core is formed of two H2A-H2B dimers and a H3-H4 tetramer, forming two nearly symmetrical halves by tertiary structure (C2 symmetry; one macromolecule is the mirror image of the other). The H2A-H2B dimers and H3-H4 tetramer also show pseudodyad symmetry. The 4 ‘core’ histones (H2A, H2B, H3 and H4) are relatively similar in structure and are highly conserved through evolution, all featuring a ‘helix turn helix turn helix’ motif (DNA-binding protein motif that recognize specific DNA sequence). They also share the feature of long ‘tails’ on one end of the amino acid structure - this being the location of post-translational modifiation (see below).</p>
<p>Archaeal histone only contains a H3-H4 like dimeric structure made out of the same protein. Such dimeric structures can stack into a tall superhelix (“supernucleosome”) onto which DNA coils in a manner similar to nucleosome spools. Only some archaeal histones have tails.</p>
<p>It has been proposed that histone proteins are evolutionarily related to the helical part of the extended AAA+ ATPase domain, the C-domain, and to the N-terminal substrate recognition domain of Clp/Hsp100 proteins. Despite the differences in their topology, these three folds share a homologous helix-strand-helix (HSH) motif.</p>
<p>Using an electron paramagnetic resonance spin-labeling technique, British researchers measured the distances between the spools around which eukaryotic cells wind their DNA. They determined the spacings range from 59 to 70 Å.</p>
<p>In all, histones make five types of interactions with DNA:</p>
<p>Helix-dipoles form alpha-helixes in H2B, H3, and H4 cause a net positive charge to accumulate at the point of interaction with negatively charged phosphate groups on DNA
Hydrogen bonds between the DNA backbone and the amide group on the main chain of histone proteins
Nonpolar interactions between the histone and deoxyribose sugars on DNA
Salt bridges and hydrogen bonds between side chains of basic amino acids (especially lysine and arginine) and phosphate oxygens on DNA
Non-specific minor groove insertions of the H3 and H2B N-terminal tails into two minor grooves each on the DNA molecule
The highly basic nature of histones, aside from facilitating DNA-histone interactions, contributes to their water solubility.</p>
<p>Histones are subject to post translational modification by enzymes primarily on their N-terminal tails, but also in their globular domains. Such modifications include methylation, citrullination, acetylation, phosphorylation, SUMOylation, ubiquitination, and ADP-ribosylation. This affects their function of gene regulation.</p>
<p>In general, genes that are active have less bound histone, while inactive genes are highly associated with histones during interphase. It also appears that the structure of histones has been evolutionarily conserved, as any deleterious mutations would be severely maladaptive. All histones have a highly positively charged N-terminus with many lysine and arginine residues.</p>
<p>Histones were discovered in 1884 by Albrecht Kossel. The word “histone” dates from the late 19th century and is derived from the German word “Histon”, a word itself of uncertain origin - perhaps from the Greek histanai or histos.</p>
<p>In the early 1960s, before the types of histones were known and before histones were known to be highly conserved across taxonomically diverse organisms, James F. Bonner and his collaborators began a study of these proteins that were known to be tightly associated with the DNA in the nucleus of higher organisms. Bonner and his postdoctoral fellow Ru Chih C. Huang showed that isolated chromatin would not support RNA transcription in the test tube, but if the histones were extracted from the chromatin, RNA could be transcribed from the remaining DNA. Their paper became a citation classic. Paul T’so and James Bonner had called together a World Congress on Histone Chemistry and Biology in 1964, in which it became clear that there was no consensus on the number of kinds of histone and that no one knew how they would compare when isolated from different organisms. Bonner and his collaborators then developed methods to separate each type of histone, purified individual histones, compared amino acid compositions in the same histone from different organisms, and compared amino acid sequences of the same histone from different organisms in collaboration with Emil Smith from UCLA. For example, they found Histone IV sequence to be highly conserved between peas and calf thymus. However, their work on the biochemical characteristics of individual histones did not reveal how the histones interacted with each other or with DNA to which they were tightly bound.</p>
<p>Also in the 1960s, Vincent Allfrey and Alfred Mirsky had suggested, based on their analyses of histones, that acetylation and methylation of histones could provide a transcriptional control mechanism, but did not have available the kind of detailed analysis that later investigators were able to conduct to show how such regulation could be gene-specific. Until the early 1990s, histones were dismissed by most as inert packing material for eukaryotic nuclear DNA, a view based in part on the models of Mark Ptashne and others, who believed that transcription was activated by protein-DNA and protein-protein interactions on largely naked DNA templates, as is the case in bacteria.</p>
<p>During the 1980s, Yahli Lorch and Roger Kornberg showed that a nucleosome on a core promoter prevents the initiation of transcription in vitro, and Michael Grunstein demonstrated that histones repress transcription in vivo, leading to the idea of the nucleosome as a general gene repressor. Relief from repression is believed to involve both histone modification and the action of chromatin-remodeling complexes. Vincent Allfrey and Alfred Mirsky had earlier proposed a role of histone modification in transcriptional activation, regarded as a molecular manifestation of epigenetics. Michael Grunstein and David Allis found support for this proposal, in the importance of histone acetylation for transcription in yeast and the activity of the transcriptional activator Gcn5 as a histone acetyltransferase.</p>
<p>The discovery of the H5 histone appears to date back to the 1970s, and it is now considered an isoform of Histone H1.</p>
<p>Conservation across species
Histones are found in the nuclei of eukaryotic cells, and in certain Archaea, namely Proteoarchaea and Euryarchaea, but not in bacteria. The unicellular algae known as dinoflagellates were previously thought to be the only eukaryotes that completely lack histones, however, later studies showed that their DNA still encodes histone genes. Unlike the core histones, lysine-rich linker histone (H1) proteins are found in bacteria, otherwise known as nucleoprotein HC1/HC2.</p>
<p>Archaeal histones may well resemble the evolutionary precursors to eukaryotic histones. Histone proteins are among the most highly conserved proteins in eukaryotes, emphasizing their important role in the biology of the nucleus.:939 In contrast mature sperm cells largely use protamines to package their genomic DNA, most likely because this allows them to achieve an even higher packaging ratio.</p>
<p>There are some variant forms in some of the major classes. They share amino acid sequence homology and core structural similarity to a specific class of major histones but also have their own feature that is distinct from the major histones. These minor histones usually carry out specific functions of the chromatin metabolism. For example, histone H3-like CENPA is associated with only the centromere region of the chromosome. Histone H2A variant H2A.Z is associated with the promoters of actively transcribed genes and also involved in the prevention of the spread of silent heterochromatin. Furthermore, H2A.Z has roles in chromatin for genome stability. Another H2A variant H2A.X is phosphorylated at S139 in regions around double-strand breaks and marks the region undergoing DNA repair. Histone H3.3 is associated with the body of actively transcribed genes.</p>
<p>Evolutionary origin
Histones are believed to have evolved from ribosomal proteins with which they share much in common, both being short and basic proteins.</p>
<p>Function
Compacting DNA strands
Histones act as spools around which DNA winds. This enables the compaction necessary to fit the large genomes of eukaryotes inside cell nuclei: the compacted molecule is 40,000 times shorter than an unpacked molecule.</p>
<p>Chromatin regulation
Histones undergo posttranslational modifications that alter their interaction with DNA and nuclear proteins. The H3 and H4 histones have long tails protruding from the nucleosome, which can be covalently modified at several places. Modifications of the tail include methylation, acetylation, phosphorylation, ubiquitination, SUMOylation, citrullination, and ADP-ribosylation. The core of the histones H2A and H2B can also be modified. Combinations of modifications are thought to constitute a code, the so-called “histone code”. Histone modifications act in diverse biological processes such as gene regulation, DNA repair, chromosome condensation (mitosis) and spermatogenesis (meiosis).</p>
<p>The common nomenclature of histone modifications is:</p>
<ul>
<li>The name of the histone (e.g., H3)</li>
<li>The single-letter amino acid abbreviation (e.g., K for Lysine) and the amino acid position in the protein</li>
<li>The type of modification (Me: methyl, P: phosphate, Ac: acetyl, Ub: ubiquitin)</li>
<li>The number of modifications (only Me is known to occur in more than one copy per residue. 1, 2 or 3 is mono-, di- or tri-methylation)</li>
</ul>
<p>So H3K4me1 denotes the monomethylation of the 4th residue (a lysine) from the start (i.e., the N-terminal) of the H3 protein.</p>
</div>
</div>
<div id="functions-of-histone-modifications" class="section level2">
<h2><span class="header-section-number">8.43</span> Functions of histone modifications</h2>
<p>understanding of most is still lacking. Collectively, it is thought that histone modifications may underlie a histone code, whereby combinations of histone modifications have specific meanings. However, most functional data concerns individual prominent histone modifications that are biochemically amenable to detailed study.</p>
</div>
<div id="chemistry-of-histone-modifications" class="section level2">
<h2><span class="header-section-number">8.44</span> Chemistry of histone modifications</h2>
<p>Lysine methylation</p>
<p>The addition of one, two, or many methyl groups to lysine has little effect on the chemistry of the histone; methylation leaves the charge of the lysine intact and adds a minimal number of atoms so steric interactions are mostly unaffected. However, proteins containing Tudor, chromo or PHD domains, amongst others, can recognise lysine methylation with exquisite sensitivity and differentiate mono, di and tri-methyl lysine, to the extent that, for some lysines (e.g.: H4K20) mono, di and tri-methylation appear to have different meanings. Because of this, lysine methylation tends to be a very informative mark and dominates the known histone modification functions.</p>
<p>Glutamine serotonylation</p>
<p>Recently it has been shown, that the addition of a serotonin group to the position 5 glutamine of H3, happens in serotonergic cells such as neurons. This is part of the differentiation of the serotonergic cells. This post-translational modification happens in conjunction with the H3K4me3 modification. The serotonylation potentiates the binding of the general transcription factor TFIID to the TATA box.</p>
<p>Arginine methylation</p>
<p>Arginine citrullination
Enzymes called peptidylarginine deiminases (PADs) hydrolyze the imine group of arginines and attach a keto group, so that there is one less positive charge on the amino acid residue. This process has been involved in the activation of gene expression by making the modified histones less tightly bound to DNA and thus making the chromatin more accessible. PADs can also produce the opposite effect by removing or inhibiting mono-methylation of arginine residues on histones and thus antagonizing the positive effect arginine methylation has on transcriptional activity.</p>
<p>Lysine acetylation</p>
<p>Addition of an acetyl group has a major chemical effect on lysine as it neutralises the positive charge. This reduces electrostatic attraction between the histone and the negatively charged DNA backbone, loosening the chromatin structure; highly acetylated histones form more accessible chromatin and tend to be associated with active transcription. Lysine acetylation appears to be less precise in meaning than methylation, in that histone acetyltransferases tend to act on more than one lysine; presumably this reflects the need to alter multiple lysines to have a significant effect on chromatin structure. The modification includes H3K27ac.</p>
<p>Serine/threonine/tyrosine phosphorylation</p>
<p>Addition of a negatively charged phosphate group can lead to major changes in protein structure, leading to the well-characterised role of phosphorylation in controlling protein function. It is not clear what structural implications histone phosphorylation has, but histone phosphorylation has clear functions as a post-translational modification, and binding domains such as BRCT have been characterised.</p>
</div>
<div id="functions-in-transcription" class="section level2">
<h2><span class="header-section-number">8.45</span> Functions in transcription</h2>
<p>Most well-studied histone modifications are involved in control of transcription.</p>
<p>Actively transcribed genes
Two histone modifications are particularly associated with active transcription:</p>
<p>Trimethylation of H3 lysine 4 (H3K4me3)
This trimethylation occurs at the promoter of active genes and is performed by the COMPASS complex. Despite the conservation of this complex and histone modification from yeast to mammals, it is not entirely clear what role this modification plays. However, it is an excellent mark of active promoters and the level of this histone modification at a gene’s promoter is broadly correlated with transcriptional activity of the gene. The formation of this mark is tied to transcription in a rather convoluted manner: early in transcription of a gene, RNA polymerase II undergoes a switch from initiating’ to ‘elongating’, marked by a change in the phosphorylation states of the RNA polymerase II C terminal domain (CTD). The same enzyme that phosphorylates the CTD also phosphorylates the Rad6 complex, which in turn adds a ubiquitin mark to H2B K123 (K120 in mammals). H2BK123Ub occurs throughout transcribed regions, but this mark is required for COMPASS to trimethylate H3K4 at promoters.
Trimethylation of H3 lysine 36 (H3K36me3)
This trimethylation occurs in the body of active genes and is deposited by the methyltransferase Set2. This protein associates with elongating RNA polymerase II, and H3K36Me3 is indicative of actively transcribed genes. H3K36Me3 is recognised by the Rpd3 histone deacetylase complex, which removes acetyl modifications from surrounding histones, increasing chromatin compaction and repressing spurious transcription. Increased chromatin compaction prevents transcription factors from accessing DNA, and reduces the likelihood of new transcription events being initiated within the body of the gene. This process therefore helps ensure that transcription is not interrupted.
Repressed genes
Three histone modifications are particularly associated with repressed genes:</p>
<p>Trimethylation of H3 lysine 27 (H3K27me3)
This histone modification is depositied by the polycomb complex PRC2. It is a clear marker of gene repression, and is likely bound by other proteins to exert a repressive function. Another polycomb complex, PRC1, can bind H3K27me3 and adds the histone modification H2AK119Ub which aids chromatin compaction. Based on this data it appears that PRC1 is recruited through the action of PRC2, however, recent studies show that PRC1 is recruited to the same sites in the absence of PRC2.
Di and tri-methylation of H3 lysine 9 (H3K9me2/3)
H3K9me2/3 is a well-characterised marker for heterochromatin, and is therefore strongly associated with gene repression. The formation of heterochromatin has been best studied in the yeast Schizosaccharomyces pombe, where it is initiated by recruitment of the RNA-induced transcriptional silencing (RITS) complex to double stranded RNAs produced from centromeric repeats. RITS recruits the Clr4 histone methyltransferase which deposits H3K9me2/3. This process is called histone methylation. H3K9Me2/3 serves as a binding site for the recruitment of Swi6 (heterochromatin protein 1 or HP1, another classic heterochromatin marker) which in turn recruits further repressive activities including histone modifiers such as histone deacetylases and histone methyltransferases.
Trimethylation of H4 lysine 20 (H4K20me3)
This modification is tightly associated with heterochromatin, although its functional importance remains unclear. This mark is placed by the Suv4-20h methyltransferase, which is at least in part recruited by heterochromatin protein 1.
Bivalent promoters
Analysis of histone modifications in embryonic stem cells (and other stem cells) revealed many gene promoters carrying both H3K4Me3 and H3K27Me3, in other words these promoters display both activating and repressing marks simultaneously. This peculiar combination of modifications marks genes that are poised for transcription; they are not required in stem cells, but are rapidly required after differentiation into some lineages. Once the cell starts to differentiate, these bivalent promoters are resolved to either active or repressive states depending on the chosen lineage.</p>
<p>Other functions
DNA damage
Marking sites of DNA damage is an important function for histone modifications. It also protects DNA from getting destroyed by ultraviolet radiation of sun.</p>
<p>Phosphorylation of H2AX at serine 139 (γH2AX)
Phosphorylated H2AX (also known as gamma H2AX) is a marker for DNA double strand breaks, and forms part of the response to DNA damage. H2AX is phosphorylated early after detection of DNA double strand break, and forms a domain extending many kilobases either side of the damage. Gamma H2AX acts as a binding site for the protein MDC1, which in turn recruits key DNA repair proteins (this complex topic is well reviewed in) and as such, gamma H2AX forms a vital part of the machinery that ensures genome stability.
Acetylation of H3 lysine 56 (H3K56Ac)
H3K56Acx is required for genome stability. H3K56 is acetylated by the p300/Rtt109 complex, but is rapidly deacetylated around sites of DNA damage. H3K56 acetylation is also required to stabilise stalled replication forks, preventing dangerous replication fork collapses. Although in general mammals make far greater use of histone modifications than microorganisms, a major role of H3K56Ac in DNA replication exists only in fungi, and this has become a target for antibiotic development.
DNA repair
Trimethylation of H3 lysine 36 (H3K36me3)
H3K36me3 has the ability to recruit the MSH2-MSH6 (hMutSα) complex of the DNA mismatch repair pathway. Consistently, regions of the human genome with high levels of H3K36me3 accumulate less somatic mutations due to mismatch repair activity.</p>
<p>Chromosome condensation
Phosphorylation of H3 at serine 10 (phospho-H3S10)
The mitotic kinase aurora B phosphorylates histone H3 at serine 10, triggering a cascade of changes that mediate mitotic chromosome condensation. Condensed chromosomes therefore stain very strongly for this mark, but H3S10 phosphorylation is also present at certain chromosome sites outside mitosis, for example in pericentric heterochromatin of cells during G2. H3S10 phosphorylation has also been linked to DNA damage caused by R-loop formation at highly transcribed sites.
Phosphorylation H2B at serine 10/14 (phospho-H2BS10/14)
Phosphorylation of H2B at serine 10 (yeast) or serine 14 (mammals) is also linked to chromatin condensation, but for the very different purpose of mediating chromosome condensation during apoptosis. This mark is not simply a late acting bystander in apoptosis as yeast carrying mutations of this residue are resistant to hydrogen peroxide-induced apoptotic cell death.
Addiction
Epigenetic modifications of histone tails in specific regions of the brain are of central importance in addictions. Once particular epigenetic alterations occur, they appear to be long lasting “molecular scars” that may account for the persistence of addictions.</p>
<p>Cigarette smokers (about 15% of the US population) are usually addicted to nicotine. After 7 days of nicotine treatment of mice, acetylation of both histone H3 and histone H4 was increased at the FosB promoter in the nucleus accumbens of the brain, causing 61% increase in FosB expression. This would also increase expression of the splice variant Delta FosB. In the nucleus accumbens of the brain, Delta FosB functions as a “sustained molecular switch” and “master control protein” in the development of an addiction.</p>
<p>About 7% of the US population is addicted to alcohol. In rats exposed to alcohol for up to 5 days, there was an increase in histone 3 lysine 9 acetylation in the pronociceptin promoter in the brain amygdala complex. This acetylation is an activating mark for pronociceptin. The nociceptin/nociceptin opioid receptor system is involved in the reinforcing or conditioning effects of alcohol.</p>
<p>Methamphetamine addiction occurs in about 0.2% of the US population. Chronic methamphetamine use causes methylation of the lysine in position 4 of histone 3 located at the promoters of the c-fos and the C-C chemokine receptor 2 (ccr2) genes, activating those genes in the nucleus accumbens (NAc). c-fos is well known to be important in addiction. The ccr2 gene is also important in addiction, since mutational inactivation of this gene impairs addiction.</p>
<p>Histone synthesis
The first step of chromatin structure duplication is the synthesis of histone proteins: H1, H2A, H2B, H3, H4. These proteins are synthesized during S phase of the cell cycle. There are different mechanisms which contribute to the increase of histone synthesis.</p>
<p>Yeast
Yeast carry one or two copies of each histone gene, which are not clustered but rather scattered throughout chromosomes. Histone gene transcription is controlled by multiple gene regulatory proteins such as transcription factors which bind to histone promoter regions. In budding yeast, the candidate gene for activation of histone gene expression is SBF. SBF is a transcription factor that is activated in late G1 phase, when it dissociates from its repressor Whi5. This occurs when Whi5 is phosphorylated by Cdc8 which is a G1/S Cdk. Suppression of histone gene expression outside of S phases is dependent on Hir proteins which form inactive chromatin structure at the locus of histone genes, causing transcriptional activators to be blocked.</p>
<p>Metazoans
In metazoans the increase in the rate of histone synthesis is due to the increase in processing of pre-mRNA to its mature form as well as decrease in mRNA degradation; this results in an increase of active mRNA for translation of histone proteins. The mechanism for mRNA activation has been found to be the removal of a segment of the 3’ end of the mRNA strand, and is dependent on association with stem-loop binding protein (SLBP). SLBP also stabilizes histone mRNAs during S phase by blocking degradation by the 3’hExo nuclease. SLBP levels are controlled by cell-cycle proteins, causing SLBP to accumulate as cells enter S phase and degrade as cells leave S phase. SLBP are marked for degradation by phosphorylation at two threonine residues by cyclin dependent kinases, possibly cyclin A/ cdk2, at the end of S phase. Metazoans also have multiple copies of histone genes clustered on chromosomes which are localized in structures called Cajal bodies as determined by genome-wide chromosome conformation capture analysis (4C-Seq).</p>
<p>Link between cell-cycle control machinery and histone synthesis
Nuclear protein Ataxia-Telangiectasia (NPAT), also known as nuclear protein coactivator of histone transcription, is a transcription factor which activates histone gene transcription on chromosomes 1 and 6 of human cells. NPAT is also a substrate of cyclin E-Cdk2, which is required for the transition between G1 phase and S phase. NPAT activates histone gene expression only after it has been phosphorylated by the G1/S-Cdk cyclin E-Cdk2 in early S phase. This shows an important regulatory link between cell-cycle control and histone synthesis.</p>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="the-structure-and-function-of-dna-and-rna.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="dna-replication.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": false,
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/nikolaussucher/genetics/edit/master/chromosome_structure.rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["_main.pdf"],
"toc": {
"collapse": "section"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
